(()=>{var e={735:()=>{},29:(e,t,n)=>{(t=n(645)(!1)).push([e.id,".swagger-ui .topbar {\n  padding: 0.5rem 0 0 0;\n  margin: 0 0 0 0;\n  background-color: #eef0ef;\n  height: 4rem;\n}\n\n.swagger-ui .topbar-wrapper img,\n.topbar-wrapper span {\n  visibility: collapse;\n}\n.swagger-ui .topbar-wrapper {\n  cursor: pointer;\n  content: url('/img/logo.png');\n  height: 3rem;\n}\n\n#imgp img {\n  height: auto;\n  max-width: 100%;\n  max-height: 100%;\n}\n\n#imgp a {\n  text-decoration: none !important;\n}\n\n.scheme-container {\n  display: none;\n}\n\n.opblock-tag-section {\n  margin-top: 2rem;\n}\n\n.swagger-ui .opblock.opblock-get {\n  border-color: #25543c;\n  background: #f2f8f5;\n}\n\n.swagger-ui .opblock.opblock-get .opblock-summary {\n  border-color: #25543c;\n}\n\n.swagger-ui .opblock.opblock-get .opblock-summary-method {\n  background: #4c735f;\n}\n\n.swagger-ui .btn.execute {\n  background-color: #25543c;\n  border-color: #25543c;\n}\n\n.base-url {\n  display: none;\n}\n\n#footer {\n  margin-top: 4rem;\n  background-color: #eef0ef;\n}\n\n.footer img {\n  max-height: 20px;\n}\n\npre {\n  background-color: #7d8492;\n}\n",""]),e.exports=t},645:e=>{"use strict";e.exports=function(e){var t=[];return t.toString=function(){return this.map((function(t){var n=function(e,t){var n,a,i,o=e[1]||"",s=e[3];if(!s)return o;if(t&&"function"==typeof btoa){var l=(n=s,a=btoa(unescape(encodeURIComponent(JSON.stringify(n)))),i="sourceMappingURL=data:application/json;charset=utf-8;base64,".concat(a),"/*# ".concat(i," */")),r=s.sources.map((function(e){return"/*# sourceURL=".concat(s.sourceRoot||"").concat(e," */")}));return[o].concat(r).concat([l]).join("\n")}return[o].join("\n")}(t,e);return t[2]?"@media ".concat(t[2]," {").concat(n,"}"):n})).join("")},t.i=function(e,n,a){"string"==typeof e&&(e=[[null,e,""]]);var i={};if(a)for(var o=0;o<this.length;o++){var s=this[o][0];null!=s&&(i[s]=!0)}for(var l=0;l<e.length;l++){var r=[].concat(e[l]);a&&i[r[0]]||(n&&(r[2]?r[2]="".concat(n," and ").concat(r[2]):r[2]=n),t.push(r))}},t}},441:e=>{"use strict";e.exports=require("sharp")},354:e=>{"use strict";e.exports=JSON.parse('[{"short_name":"Yeast-GEM","full_name":"Consensus genome-scale metabolic model of Saccharomyces cerevisiae","description":"Consensus genome-scale metabolic model of Saccharomyces cerevisiae. It is the continuation of the legacy project yeastnet","version":"8.6.2","link":"https://github.com/SysBioChalmers/yeast-GEM","chat_link":"https://gitter.im/SysBioChalmers/yeast-GEM","authors":[{"given_name":"Feiran","family_name":"Li","email":"feiranl@chalmers.se","organization":""},{"given_name":"Hongzhong","family_name":"Lu","email":"luho@chalmers.se","organization":""},{"given_name":"Benjamín","family_name":"Sánchez","email":"bejsab@dtu.dk","organization":""}],"condition":"Aerobic, glucose-limited, defined media","date":"2022-09-05","sample":{"organism":"Saccharomyces cerevisiae, S288C","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"ref":["31395883"]},{"short_name":"Human-GEM","full_name":"Generic genome-scale metabolic model of Homo sapiens","description":"Human genome-scale metabolic models are important tools for the study of human health and diseases, by providing a scaffold upon which different types of data can be analyzed. This is the latest version of Human-GEM, which is a genome-scale model of the generic human cell. The objective of Human-GEM is to serve as a community model for enabling integrative and mechanistic studies of human metabolism.","version":"1.14.0","link":"https://github.com/SysBioChalmers/Human-GEM","chat_link":"https://github.com/SysBioChalmers/Human-GEM/discussions","authors":[{"given_name":"Jonathan","family_name":"Robinson","email":"jonrob@chalmers.se","organization":""},{"given_name":"Hao","family_name":"Wang","email":"hao.wang@chalmers.se","organization":""},{"given_name":"Pınar","family_name":"Kocabaş","email":"kocabas@chalmers.se","organization":""}],"condition":"Generic metabolism","date":"2023-02-25","sample":{"organism":"Homo sapiens","organ_system":null,"tissue":null,"cell_type":"Generic cell","cell_line":null},"ref":["32209698"]},{"short_name":"Mouse-GEM","full_name":"Generic genome-scale metabolic model of Mus musculus","description":"Mouse genome-scale metabolic models are important tools for the study of mouse and human health and diseases, by providing a scaffold upon which different omics data can be analyzed. This is the latest version of Mouse-GEM, which is a genome-scale model of the generic mouse cell. The objective of Mouse-GEM is to serve as a community model for enabling integrative and mechanistic studies of mouse and human metabolism.","version":"1.4.0","link":"https://github.com/SysBioChalmers/Mouse-GEM","chat_link":"","authors":[{"given_name":"Hao","family_name":"Wang","email":"hao.wang@chalmers.se","organization":""},{"given_name":"Pınar","family_name":"Kocabaş","email":"kocabas@chalmers.se","organization":""}],"condition":"Generic metabolism","date":"2023-03-10","sample":{"organism":"Mus musculus","organ_system":null,"tissue":null,"cell_type":"Generic cell","cell_line":null},"ref":["34282017"]},{"short_name":"Rat-GEM","full_name":"Generic genome-scale metabolic model of Rattus norvegicus","description":"Rat genome-scale metabolic models are important tools for the study of rat and human health and diseases, by providing a scaffold upon which different omics data can be analyzed. This is the latest version of Rat-GEM, which is a genome-scale model of the generic rat cell. The objective of Rat-GEM is to serve as a community model for enabling integrative and mechanistic studies of rat and human metabolism.","version":"1.4.0","link":"https://github.com/SysBioChalmers/Rat-GEM","chat_link":"","authors":[{"given_name":"Hao","family_name":"Wang","email":"hao.wang@chalmers.se","organization":""},{"given_name":"Pınar","family_name":" Kocabaş","email":"kocabas@chalmers.se","organization":""}],"condition":"Generic metabolism","date":"2023-03-11","sample":{"organism":"Rattus norvegicus","organ_system":null,"tissue":null,"cell_type":"Generic cell","cell_line":null},"ref":["34282017"]},{"short_name":"Zebrafish-GEM","full_name":"Generic genome-scale metabolic model of Danio rerio","description":"Zebrafish genome-scale metabolic models are important tools for the study of model animal zebrafish and human diseases, by providing a scaffold upon which different omics data can be analyzed. This is the latest version of Zebrafish-GEM, which is a genome-scale model of the generic zebrafish cell. The objective of Zebrafish-GEM is to serve as a community model for enabling integrative and mechanistic studies of zebrafish and human metabolism.","version":"1.3.0","link":"https://github.com/SysBioChalmers/Zebrafish-GEM","chat_link":"","authors":[{"given_name":"Hao","family_name":"Wang","email":"hao.wang@chalmers.se","organization":""},{"given_name":"Pınar","family_name":" Kocabaş","email":"kocabas@chalmers.se","organization":""}],"condition":"Generic metabolism","date":"2023-03-11","sample":{"organism":"Danio rerio","organ_system":null,"tissue":null,"cell_type":"Generic cell","cell_line":null},"ref":["34282017"]},{"short_name":"Fruitfly-GEM","full_name":"Generic genome-scale metabolic model of Drosophila melanogaster","description":"Fruit fly genome-scale metabolic models are important tools for the study of fruit fly and human diseases, by providing a scaffold upon which different omics data can be analyzed. This is the latest version of Fruitfly-GEM, which is a genome-scale model of the generic fruit fly cell. The objective of Fruitfly-GEM is to serve as a community model for enabling integrative and mechanistic studies of fruit fly and human metabolism.","version":"1.3.0","link":"https://github.com/SysBioChalmers/Fruitfly-GEM","chat_link":"","authors":[{"given_name":"Hao","family_name":"Wang","email":"hao.wang@chalmers.se","organization":""},{"given_name":"Pınar","family_name":" Kocabaş","email":"kocabas@chalmers.se","organization":""}],"condition":"Generic metabolism","date":"2023-03-11","sample":{"organism":"Drosophila melanogaster","organ_system":null,"tissue":null,"cell_type":"Generic cell","cell_line":null},"ref":["34282017"]},{"short_name":"Worm-GEM","full_name":"Generic genome-scale metabolic model of Caenorhabditis elegans","description":"Worm genome-scale metabolic models are important tools for the study of worm and human diseases, by providing a scaffold upon which different omics data can be analyzed. This is the latest version of Worm-GEM, which is a genome-scale model of the generic worm cell. The objective of Worm-GEM is to serve as a community model for enabling integrative and mechanistic studies of worm and human metabolism.","version":"1.4.0","link":"https://github.com/SysBioChalmers/Worm-GEM","chat_link":"","authors":[{"given_name":"Hao","family_name":"Wang","email":"hao.wang@chalmers.se","organization":""},{"given_name":"Pınar","family_name":" Kocabaş","email":"kocabas@chalmers.se","organization":""}],"condition":"Generic metabolism","date":"2023-03-11","sample":{"organism":"Caenorhabditis elegans","organ_system":null,"tissue":null,"cell_type":"Generic cell","cell_line":null},"ref":["34282017"]}]')}},t={};function n(a){var i=t[a];if(void 0!==i)return i.exports;var o=t[a]={id:a,exports:{}};return e[a](o,o.exports,n),o.exports}n.n=e=>{var t=e&&e.__esModule?()=>e.default:()=>e;return n.d(t,{a:t}),t},n.d=(e,t)=>{for(var a in t)n.o(t,a)&&!n.o(e,a)&&Object.defineProperty(e,a,{enumerable:!0,get:t[a]})},n.o=(e,t)=>Object.prototype.hasOwnProperty.call(e,t),(()=>{"use strict";const e=require("express");var t=n.n(e);require("@babel/runtime/helpers/typeof");const a=require("@babel/runtime/helpers/defineProperty");var i=n.n(a);const o=require("@babel/runtime/helpers/asyncToGenerator");var s=n.n(o);const l=require("@babel/runtime/regenerator");var r=n.n(l);const c=require("@babel/runtime/helpers/objectWithoutProperties");var d=n.n(c);const m=require("@babel/runtime/helpers/slicedToArray");var u=n.n(m);const p=require("neo4j-driver");var f=n.n(p),h=f().driver("bolt://neo4j",f().auth.basic(process.env.NEO4J_USERNAME,process.env.NEO4J_PASSWORD),{disableLosslessIntegers:!0});process.on("exit",s()(r().mark((function e(){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,h.close();case 2:case"end":return e.stop()}}),e)}))));const g=h,y=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l=arguments;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=!(l.length>1&&void 0!==l[1])||l[1],a=g.session(),e.prev=2,e.next=5,a.readTransaction((function(e){return e.run(t)}));case 5:if(s=e.sent,!n||0!==s.records.length){e.next=8;break}throw new Error("404");case 8:if(i=s.records[0].get(0),!n||0!==Object.values(i).flat().filter((function(e){return 0!==e})).length){e.next=11;break}throw new Error("404");case 11:e.next=16;break;case 13:e.prev=13,e.t0=e.catch(2),o=e.t0;case 16:return e.prev=16,e.next=19,a.close();case 19:return e.finish(16);case 20:if(!o){e.next=22;break}throw o;case 22:return e.abrupt("return",i);case 23:case"end":return e.stop()}}),e,null,[[2,13,16,20]])})));return function(t){return e.apply(this,arguments)}}(),b=function(e,t){return[e?":".concat(e):"",t?":V".concat(t):""]};var _=["subsystems","compartmentSVGs","externalDbs"];function v(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function w(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?v(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):v(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}const M=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,m,p,f,h,g,v,M,T;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.id,a=t.model,i=t.version,o=t.full,s=b(a,i),l=u()(s,2),c=l[0],m=l[1],p=!["false",'""',"0","null","undefined","NaN"].includes("".concat(o).toLowerCase()),f='\nCALL apoc.cypher.run("\n  MATCH (cs:CompartmentState)-['.concat(m,"]-(c:Compartment").concat(c," {id: '").concat(n,"'})\n  RETURN cs { id: c.id, .* } as data\n  \n  UNION\n    \n  MATCH (:Compartment").concat(c," {id: '").concat(n,"'})-[").concat(m,"]-(cm:CompartmentalizedMetabolite)-[").concat(m,"]-(r:Reaction)-[").concat(m,"]-(s:Subsystem)-[").concat(m,"]-(ss:SubsystemState)\n  USING JOIN on r\n  RETURN {\n    subsystems: COLLECT(DISTINCT({id: s.id, name: ss.name})) ,\n    reactionsCount: COUNT(DISTINCT(r)) ").concat(p?", reactions: COLLECT(DISTINCT(r))":"",",\n    metabolitesCount: COUNT(DISTINCT(cm)) ").concat(p?", metabolites: COLLECT(DISTINCT(cm))":"","\n  } as data\n\n  UNION\n  \n  MATCH (:Compartment").concat(c," {id: '").concat(n,"'})-[").concat(m,"]-(:CompartmentalizedMetabolite)-[").concat(m,"]-(r:Reaction)-[").concat(m,"]-(g:Gene)\n  USING JOIN on r\n  RETURN { genesCount: COUNT(DISTINCT(g)) ").concat(p?", genes: COLLECT(DISTINCT(g))":""," } as data\n\n  UNION\n\n  MATCH (:Compartment").concat(c," {id: '").concat(n,"'})-[").concat(m,"]-(e:ExternalDb)\n  RETURN { externalDbs: COLLECT(DISTINCT(e {.*})) } as data\n  \n  UNION\n  \n  MATCH (:Compartment").concat(c," {id: '").concat(n,"'})-[").concat(m,"]-(csvg:SvgMap)\n  WITH  COLLECT(DISTINCT(csvg {.*})) as csvgs\n  WITH CASE\n    WHEN size(csvgs) > 0\n    THEN {compartmentId: '").concat(n,"', compartmentSVGs: csvgs}\n    ELSE null\n  END as compartmentSVG\n  RETURN { compartmentSVGs: COLLECT(compartmentSVG) } as data\n\", {}) yield value\nRETURN apoc.map.mergeList(COLLECT(value.data)) as compartment\n"),e.next=7,y(f);case 7:return h=e.sent,g=h.subsystems,v=h.compartmentSVGs,M=h.externalDbs,T=d()(h,_),e.abrupt("return",{info:w(w({},T),{},{subsystemCount:g.length}),compartmentSVGs:[{id:T.id,customName:T.name,svgMaps:v[0]?v[0].compartmentSVGs.sort((function(e,t){return e.id.localeCompare(t.id)})):[]}],externalDbs:M,subsystems:g});case 13:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),T=function(){var e=s()(r().mark((function e(t){var n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=g.session(),e.prev=1,e.next=4,n.readTransaction((function(e){return e.run(t)}));case 4:o=e.sent,a=o.records.map((function(e){return e.get(0)})),e.next=11;break;case 8:e.prev=8,e.t0=e.catch(1),i=e.t0;case 11:return e.prev=11,e.next=14,n.close();case 14:return e.finish(11);case 15:if(!i){e.next=17;break}throw i;case 17:return e.abrupt("return",a);case 18:case"end":return e.stop()}}),e,null,[[1,8,11,15]])})));return function(t){return e.apply(this,arguments)}}(),I=require("@babel/runtime/helpers/toConsumableArray");var C=n.n(I);function N(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function G(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?N(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):N(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var x=function(e){return e.reduce((function(e,t){var n=e[t.dbName]||[];return n=[].concat(C()(n),[{id:t.externalId,url:t.url}]),G(G({},e),{},i()({},t.dbName,n))}),{})},H=function(e){return C()(e.compartments).sort((function(e,t){return e.id.localeCompare(t.id)})).map((function(t){var n=e.compartmentSVGs.find((function(e){return e.compartmentId===t.id})),a=n?n.compartmentSVGs.sort((function(e,t){return e.id.localeCompare(t.id)})):[];return{id:t.id,customName:t.name,svgMaps:a}}))},S=function(e){return C()(e.subsystems).sort((function(e,t){return e.id.localeCompare(t.id)})).map((function(t){var n=e.subsystemSVGs.find((function(e){return e.subsystemId===t.id})),a=n?n.subsystemSVGs.sort((function(e,t){return e.id.localeCompare(t.id)})):[];return{id:t.id,customName:t.name,svgMaps:a}}))},E=n(354);function R(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function z(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?R(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):R(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var O="https://metabolicatlas.org",L=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.id,a=t.model,i=t.version,o=b(a,i),s=u()(o,2),l=s[0],c=s[1],d='\nCALL apoc.cypher.run("\n  MATCH (gs:GeneState)-['.concat(c,"]-(g:Gene").concat(l," {id: '").concat(n,"'})\n  RETURN gs { id: g.id, .* } as data\n  \n  UNION\n  \n  MATCH (:Gene").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(:Reaction)-[").concat(c,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(c,"]-(c:Compartment)-[").concat(c,"]-(cs:CompartmentState)\n  RETURN { compartments: COLLECT(DISTINCT({id: c.id, name: cs.name})) } as data\n  \n  UNION\n  \n  MATCH (:Gene").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(:Reaction)-[").concat(c,"]-(s:Subsystem)-[").concat(c,"]-(ss:SubsystemState)\n  RETURN { subsystems: COLLECT(DISTINCT({id: s.id, name: ss.name})) } as data\n  \n  UNION\n  \n  MATCH (:Gene").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(e:ExternalDb)\n  RETURN { externalDbs: COLLECT(DISTINCT(e {.*})) } as data\n  \n  UNION\n  \n  MATCH (:Gene").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(:Reaction)-[").concat(c,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(c,"]-(c:Compartment)-[").concat(c,"]-(csvg:SvgMap)\n  WITH {compartmentId: c.id, compartmentSVGs: COLLECT(DISTINCT(csvg {.*}))} as compartmentSVG\n  RETURN { compartmentSVGs: COLLECT(compartmentSVG) } as data\n  \n  UNION\n  \n  MATCH (:Gene").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(:Reaction)-[").concat(c,"]-(s:Subsystem)-[").concat(c,']-(ssvg:SvgMap)\n  WITH {subsystemId: s.id, subsystemSVGs: COLLECT(DISTINCT(ssvg {.*}))} as subsystemSVG\n  RETURN { subsystemSVGs: COLLECT(subsystemSVG) } as data\n", {}) yield value\nRETURN apoc.map.mergeList(COLLECT(value.data)) as gene\n'),e.next=5,y(d);case 5:return m=e.sent,e.abrupt("return",z(z({},m),{},{compartmentSVGs:H(m),subsystemSVGs:S(m),externalDbs:x(m.externalDbs)}));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),A=function(){var e=E.find((function(e){return"Human-GEM"===e.short_name}));return[":".concat(e.short_name.split("-").map((function(e){return e[0]+e.slice(1).toLowerCase()})).join("")),":V".concat(e.version.split(".").join("_"))]},B=function(){var e=s()(r().mark((function e(){var t,n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return t=A(),n=u()(t,2),a=n[0],i=n[1],o="\nMATCH (g:Gene".concat(a,")-[").concat(i,"]-(r:Reaction)-[").concat(i,"]-(s:Subsystem)-[").concat(i,"]-(ss:SubsystemState)\nUSING JOIN ON r\nRETURN DISTINCT [g.id, ss.name, s.id]\n"),e.abrupt("return",T(o));case 3:case"end":return e.stop()}}),e)})));return function(){return e.apply(this,arguments)}}(),k=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f,h,g;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.id,a=A(),i=u()(a,2),o=i[0],s=i[1],l="\nMATCH (:Gene".concat(o," {id:'").concat(n,"'})-[").concat(s,"]-(r:Reaction)-[").concat(s,"]-(s:Subsystem)-[").concat(s,']-(ss:SubsystemState)\nWITH DISTINCT s.id as sids, COLLECT(r.id) as rids, ss\nUNWIND sids as sid\nCALL apoc.cypher.run("\n  MATCH (:Subsystem {id: $sid})-[').concat(s,"]-(r:Reaction)-[").concat(s,"]-(cm:CompartmentalizedMetabolite)\n  USING JOIN ON r\n  WITH DISTINCT cm\n  MATCH (m:Metabolite)-[").concat(s,"]-(cm)-[").concat(s,"]-(c:Compartment)-[").concat(s,"]-(cs:CompartmentState)\n  USING JOIN ON c\n  RETURN DISTINCT { id: $sid, compartments: COLLECT(DISTINCT(cs.name)), model_metabolite_count: COUNT(DISTINCT(m)), compartment_metabolite_count: COUNT(cm) } as data\n\n  UNION\n\n  MATCH (:Subsystem {id: $sid})-[").concat(s,"]-(r:Reaction)-[").concat(s,"]-(g:Gene)\n  USING JOIN ON r\n  RETURN DISTINCT { id: $sid, genes: COLLECT(DISTINCT(g.id)) } as data\n\n  UNION\n\n  MATCH (:Subsystem {id: $sid})-[").concat(s,"]-(r:Reaction)\n  RETURN DISTINCT { id: $sid, reaction_count: COUNT(DISTINCT(r)) } as data\n\n  UNION\n\n  MATCH (:Subsystem {id: $sid})-[").concat(s,"]-(r:Reaction)\n  WHERE r.id IN $rids\n  RETURN DISTINCT { id: $sid, reactions_catalysed: COUNT(DISTINCT(r)) } as data\n\n  UNION\n\n  MATCH (:Subsystem {id: $sid})-[").concat(s,']-(ssvg:SvgMap)\n  RETURN { id: $sid,  svgs: COLLECT(ssvg.filename) } as data\n", { sid: sid, rids: rids }) yield value\nRETURN DISTINCT {\n  id: sid,\n  name: ss.name,\n  details: apoc.map.mergeList(apoc.coll.flatten(\n    apoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n  ))\n}\n'),c="\nMATCH (:Gene".concat(o," {id: '").concat(n,"'})-[").concat(s,"]-(:Reaction)-[").concat(s,"]-(cm:CompartmentalizedMetabolite)\nWITH DISTINCT cm\nMATCH (cm)-[").concat(s,"]-(c:Compartment)-[").concat(s,"]-(cs:CompartmentState)\nRETURN DISTINCT({id: c.id, name: cs.name})\n"),e.next=6,Promise.all([T(l),T(c)]);case 6:return d=e.sent,m=u()(d,2),p=m[0],f=m[1],h=p.map((function(e){var t=e.id,n=e.name,a=e.details,i=a.compartments,o=a.genes,s=a.reactions_catalysed,l=a.model_metabolite_count,r=a.compartment_metabolite_count,c=a.reaction_count,d=a.svgs;return{name:n,compartments:i,genes:o,reactions_catalysed:s,map_url:"".concat(O,"/api/v2/svg/Human-GEM/").concat(d[0]),subsystem_url:"".concat(O,"/explore/Human-GEM/gem-browser/subsystem/").concat(t),model_metabolite_count:l,compartment_metabolite_count:r,reaction_count:c,gene_count:o.length}})),g=f.map((function(e){var t=e.id;return{name:e.name,compartment_url:"".concat(O,"/explore/Human-GEM/gem-browser/compartment/").concat(t)}})),e.abrupt("return",{gene_url:"".concat(O,"/explore/Human-GEM/gem-browser/gene/").concat(n),compartments:g,subsystems:h,doc:"A subsystem can contain the same chemical metabolite that comes from different compartments."});case 13:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),D=function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l,c,d;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=b(t,n),i=u()(a,2),o=i[0],s=i[1],l="\nMATCH (g:Gene".concat(o,")-[").concat(s,"]-()\nRETURN {gene_count: COUNT(DISTINCT(g))}\n"),e.next=4,y(l);case 4:return c=e.sent,d=c.gene_count,e.abrupt("return",d);case 7:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}();function W(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function F(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?W(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):W(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var j=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.id,a=t.model,i=t.version,o=b(a,i),s=u()(o,2),l=s[0],c=s[1],d='\nCALL apoc.cypher.run("\n  MATCH (rs:ReactionState)-['.concat(c,"]-(r:Reaction").concat(l," {id: '").concat(n,"'})\n  RETURN rs { id: r.id, .* } as data\n  \n  UNION\n  \n  MATCH (r:Reaction").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(c,"]-(c:Compartment)-[").concat(c,"]-(cs:CompartmentState)\n  RETURN { compartments: COLLECT(DISTINCT(cs {id: c.id, .*})) } as data\n  \n  UNION\n  \n  MATCH (r:Reaction").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(s:Subsystem)-[").concat(c,"]-(ss:SubsystemState)\n  RETURN { subsystems: COLLECT(DISTINCT(ss {id: s.id, .*})) } as data\n  \n  UNION\n  \n  MATCH (r:Reaction").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(g:Gene)-[").concat(c,"]-(gs:GeneState)\n  RETURN { genes: COLLECT(DISTINCT(gs {id: g.id, .*})) } as data\n  \n  UNION\n  \n  MATCH (r:Reaction").concat(l," {id: '").concat(n,"'})-[cmE").concat(c,"]-(cm:CompartmentalizedMetabolite)-[").concat(c,"]-(:Metabolite)-[").concat(c,"]-(ms:MetaboliteState)\n  MATCH (cm)-[").concat(c,"]-(:Compartment)-[").concat(c,"]-(cs:CompartmentState)\n  RETURN { metabolites: COLLECT(DISTINCT(ms {id: cm.id, compartment: cs.name, fullName: COALESCE(ms.name, '') + ' [' + COALESCE(cs.letterCode, '') + ']', stoichiometry: cmE.stoichiometry, outgoing: startnode(cmE)=cm, .*})) } as data\n  \n  UNION\n  \n  MATCH (r:Reaction").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(e:ExternalDb)\n  RETURN { externalDbs: COLLECT(DISTINCT(e {.*})) } as data\n  \n  UNION\n  \n  MATCH (r:Reaction").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(p:PubmedReference)\n  RETURN { pubmedIds: COLLECT(DISTINCT(p {.*})) } as data\n  \n  UNION\n  \n  MATCH (r:Reaction").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(c,"]-(c:Compartment)-[").concat(c,"]-(csvg:SvgMap)\n  WITH {compartmentId: c.id, compartmentSVGs: COLLECT(DISTINCT(csvg {.*}))} as compartmentSVG\n  RETURN { compartmentSVGs: COLLECT(compartmentSVG) } as data\n  \n  UNION\n  \n  MATCH (r:Reaction").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(s:Subsystem)-[").concat(c,']-(ssvg:SvgMap)\n  WITH {subsystemId: s.id, subsystemSVGs: COLLECT(DISTINCT(ssvg {.*}))} as subsystemSVG\n  RETURN { subsystemSVGs: COLLECT(subsystemSVG) } as data\n", {}) yield value\nRETURN apoc.map.mergeList(COLLECT(value.data)) as reaction\n'),e.next=5,y(d);case 5:return m=e.sent,e.abrupt("return",F(F({},m),{},{compartmentSVGs:H(m),subsystemSVGs:S(m),externalDbs:x(m.externalDbs)}));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),P=function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l,c,d;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=b(t,n),i=u()(a,2),o=i[0],s=i[1],l="\nMATCH (r:Reaction".concat(o,")-[").concat(s,"]-()\nRETURN {reaction_count: COUNT(DISTINCT(r))}\n"),e.next=4,y(l);case 4:return c=e.sent,d=c.reaction_count,e.abrupt("return",d);case 7:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}();function U(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function V(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?U(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):U(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}const q=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.id,a=t.model,i=t.version,o=b(a,i),s=u()(o,2),l=s[0],c=s[1],d='\nCALL apoc.cypher.run("\n  MATCH (ss:SubsystemState)-['.concat(c,"]-(s:Subsystem").concat(l," {id: '").concat(n,"'})\n  RETURN ss { id: s.id, .* } as data\n  \n  UNION\n  \n  MATCH (:Subsystem").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(:Reaction)-[").concat(c,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(c,"]-(c:Compartment)\n  WITH DISTINCT c\n  MATCH (c)-[").concat(c,"]-(cs:CompartmentState)\n  RETURN { compartments: COLLECT({id: c.id, name: cs.name}) } as data\n  \n  UNION\n  \n  MATCH (:Subsystem").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(:Reaction)-[").concat(c,"]-(g:Gene)\n  WITH DISTINCT g\n  MATCH (g)-[").concat(c,"]-(gs:GeneState)\n  RETURN { genes: COLLECT({id: g.id, name: gs.name})[..1000] } as data\n  \n  UNION\n  \n  MATCH (:Subsystem").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(e:ExternalDb)\n  RETURN { externalDbs: COLLECT(DISTINCT(e {.*})) } as data\n  \n  UNION\n  \n  MATCH (:Subsystem").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(:Reaction)-[").concat(c,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(c,"]-(:Metabolite)-[").concat(c,"]-(ms:MetaboliteState)\n  RETURN { metabolites: COLLECT(DISTINCT(ms {id: cm.id, name: ms.name}))[..1000] }  as data\n  \n  UNION\n  \n  MATCH (:Subsystem").concat(l," {id: '").concat(n,"'})-[").concat(c,"]-(ssvg:SvgMap)\n  WITH COLLECT(DISTINCT(ssvg {.*})) as ssvgs\n  WITH CASE\n    WHEN size(ssvgs)> 0\n    THEN {subsystemId: '").concat(n,"', subsystemSVGs: ssvgs}\n    ELSE null\n  END as subsystemSVG\n  RETURN { subsystemSVGs: COLLECT(subsystemSVG) } as data\n\", {}) yield value\nRETURN apoc.map.mergeList(COLLECT(value.data)) as subsystem\n"),e.next=5,y(d);case 5:return m=e.sent,e.abrupt("return",V(V({},m),{},{subsystemSVGs:[{id:m.id,customName:m.name,svgMaps:m.subsystemSVGs[0]?m.subsystemSVGs[0].subsystemSVGs.sort((function(e,t){return e.id.localeCompare(t.id)})):[]}],externalDbs:x(m.externalDbs)}));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),$=require("@babel/runtime/helpers/taggedTemplateLiteral");var J,K,Y=n.n($),X=n(735),Z=n.n(X),Q={ratcon:{headers:["rxnRatconID"],db:"Ratcon",dbPrefix:"",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},spontaneous:{headers:["spontaneous"],db:"spontaneous",dbPrefix:"",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},tcdb:{headers:["rxnTCDBID"],db:"TCDB",dbPrefix:"",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},hmr2:{headers:["rxnHMR2ID","metHMR2ID"],db:"HMR 2.0",dbPrefix:"",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},recon3dreact:{headers:["rxnRecon3DID"],db:"Recon3D",dbPrefix:"vmhreaction",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},recon3dmet:{headers:["metRecon3DID"],db:"Recon3D",dbPrefix:"vmhmetabolite",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},hepatonet1:{headers:["rxnHepatoNET1ID","metHepatoNET1ID"],db:"HepatoNET1",dbPrefix:"",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},ehmn:{headers:["rxnEHMNID","metEHMNID"],db:"EHMN",dbPrefix:"",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},lipidmaps:{headers:["metLipidMapsID"],db:"LipidMaps",dbPrefix:"lipidmaps",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},pubchem:{headers:["metPubChemID"],db:"PubChem",dbPrefix:"pubchem",compoundSuffix:".compound",reactionSuffix:"",geneSuffix:""},hmdb:{headers:["metHMDBID"],db:"HMDB",dbPrefix:"hmdb",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},ensembl:{headers:["geneEnsemblID"],db:"Ensembl",dbPrefix:"ensembl",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},proteinatlas:{headers:["geneProteinAtlasID"],db:"Protein Atlas",dbPrefix:"hpa",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},entrez:{headers:["geneEntrezID"],db:"NCBI Gene",dbPrefix:"ncbigene",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},enst:{headers:["geneENSTID"],db:"Ensembl transcript",dbPrefix:"ensembl",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},ensp:{headers:["geneENSPID"],db:"Ensembl protein",dbPrefix:"ensembl",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},kegg:{headers:["rxnKEGGID","metKEGGID"],db:"KEGG",dbPrefix:"kegg",compoundSuffix:".compound",reactionSuffix:".reaction",geneSuffix:""},meta_net_x:{headers:["rxnMetaNetXID","metMetaNetXID"],db:"MetaNetX",dbPrefix:"metanetx",compoundSuffix:".chemical",reactionSuffix:".reaction",geneSuffix:""},rhea:{headers:["rxnRheaID","metRheaID"],db:"Rhea",dbPrefix:"rhea",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},rheamaster:{headers:["rxnRheaMasterID"],db:"RheaMaster",dbPrefix:"rhea",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},model_seed:{headers:[],db:"ModelSEED",dbPrefix:"seed",compoundSuffix:".compound",reactionSuffix:".reaction",geneSuffix:""},bigg:{headers:["rxnBiGGID","metBiGGID"],db:"BiGG",dbPrefix:"bigg",compoundSuffix:".metabolite",reactionSuffix:".reaction",geneSuffix:""},metacyc:{headers:[],db:"MetaCyc",dbPrefix:"metacyc",compoundSuffix:".compound",reactionSuffix:".reaction",geneSuffix:""},sabio_rk:{headers:[],db:"SABIO-RK",dbPrefix:"sabiork",compoundSuffix:".compound",reactionSuffix:".reaction",geneSuffix:""},chebi:{headers:["metChEBIID"],db:"ChEBI",dbPrefix:"CHEBI",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},reactome:{headers:["rxnREACTOMEID"],db:"Reactome",dbPrefix:"reactome",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},ncbigene:{headers:["geneEntrezID"],db:"NCBI Protein",dbPrefix:"ncbiprotein",compoundSuffix:"",reactionSuffix:"",geneSuffix:""},uniprot:{headers:["geneUniProtID"],db:"UniProtKB",dbPrefix:"uniprot",compoundSuffix:"",reactionSuffix:"",geneSuffix:""}},ee=["name","formula","smiles"],te=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,m;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,Z()(J||(J=Y()(["\n    select * from compounds\n    where kegg = ","\n  "])),t.toString());case 2:if((n=e.sent).length){e.next=5;break}throw new Error("Compound with kegg ID ".concat(t," returned ").concat(n.length," results."));case 5:return a=n[0],i=a.name,o=a.formula,s=a.smiles,l=d()(a,ee),c=Object.fromEntries(Object.entries({name:i,formula:o,smiles:s}).filter((function(e){var t=u()(e,2);return t[0],t[1]}))),m=Object.fromEntries(Object.entries(l).filter((function(e){var t=u()(e,2);return t[0],t[1]})).map((function(e){var t=u()(e,2),n=t[0],a=t[1],i=Q[n],o=i.db,s=i.dbPrefix,l=i.compoundSuffix;return[o,a.split(";").map((function(e){return{id:e,url:"https://identifiers.org/".concat(s).concat(l,":").concat(e)}}))]}))),e.abrupt("return",{info:c,crossReferences:m});case 10:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),ne=function(){var e=s()(r().mark((function e(t){var n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.formula,a=t.crossReferences,i="A MISSING PLACEHOLDER",e.next=4,Z()(K||(K=Y()(["\n    select smiles from compounds\n    where smiles is not null\n    and (\n      kegg = ","\n      or meta_net_x = ","\n      or chebi = ","\n      or bigg = ","\n      or formula = ","\n    )\n    limit 1\n  "])),a.KEGG||i,a.MetaNetX||i,a.ChEBI||i,a.BiGG||i,n||i);case 4:if((o=e.sent).length){e.next=7;break}return e.abrupt("return",null);case 7:return e.abrupt("return",o[0].smiles);case 8:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();function ae(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function ie(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?ae(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):ae(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var oe=function(){var e=s()(r().mark((function e(t){var n,a,o,s,l,c,d,m,p,f;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.id,a=t.model,o=t.version,s=b(a,o),l=u()(s,2),c=l[0],d=l[1],m="\nCALL apoc.cypher.run('\n  MATCH (ms:MetaboliteState)-[".concat(d,"]-(:Metabolite)-[").concat(d,"]-(cm:CompartmentalizedMetabolite").concat(c,' {id: "').concat(n,'"})\n  RETURN ms { id: cm.id, .* } as data\n  \n  UNION\n  \n  MATCH (:CompartmentalizedMetabolite').concat(c,' {id: "').concat(n,'"})-[').concat(d,"]-(c:Compartment)-[").concat(d,"]-(cs:CompartmentState)\n  RETURN { compartment: cs { id: c.id, .* } } as data\n  \n  UNION\n  \n  MATCH (:CompartmentalizedMetabolite").concat(c,' {id: "').concat(n,'"})-[').concat(d,"]-(c:Compartment)-[").concat(d,"]-(csvg:SvgMap)\n  WITH {compartmentId: c.id, compartmentSVGs: COLLECT(DISTINCT(csvg {.*}))} as compartmentSVG\n  RETURN { compartmentSVGs: COLLECT(compartmentSVG) } as data\n  \n  UNION\n  \n  MATCH (:CompartmentalizedMetabolite").concat(c,' {id: "').concat(n,'"})-[').concat(d,"]-(:Reaction)-[").concat(d,"]-(s:Subsystem)\n  WITH DISTINCT s\n  MATCH (s)-[").concat(d,"]-(ss:SubsystemState)\n  RETURN { subsystems: COLLECT(DISTINCT({id: s.id, name: ss.name})) } as data\n  \n  UNION\n  \n  MATCH (:CompartmentalizedMetabolite").concat(c,' {id: "').concat(n,'"})-[').concat(d,"]-(e:ExternalDb)\n  RETURN { externalDbs: COLLECT(e {.*}) } as data\n  \n  UNION\n  \n  MATCH (:CompartmentalizedMetabolite").concat(c,' {id: "').concat(n,'"})-[').concat(d,"]-(:Reaction)-[").concat(d,"]-(s:Subsystem)\n  WITH DISTINCT s\n  MATCH (s)-[").concat(d,"]-(ssvg:SvgMap)\n  WITH {subsystemId: s.id, subsystemSVGs: COLLECT(DISTINCT(ssvg {.*}))} as subsystemSVG\n  RETURN { subsystemSVGs: COLLECT(subsystemSVG) } as data\n', {}) yield value\nRETURN apoc.map.mergeList(COLLECT(value.data)) as metabolite\n"),e.next=5,y(m);case 5:return p=e.sent,e.prev=6,e.next=9,ne({formula:p.formula,crossReferences:p.externalDbs.reduce((function(e,t){return ie(ie({},e),{},i()({},t.dbName,t.externalId))}),{})});case 9:(f=e.sent)&&(p.smiles=f),e.next=16;break;case 13:e.prev=13,e.t0=e.catch(6),console.error(e.t0);case 16:return p.compartments=[p.compartment],e.abrupt("return",ie(ie({},p),{},{compartmentSVGs:H(p),subsystemSVGs:S(p),externalDbs:x(p.externalDbs)}));case 18:case"end":return e.stop()}}),e,null,[[6,13]])})));return function(t){return e.apply(this,arguments)}}(),se=function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l,c,d;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=b(t,n),i=u()(a,2),o=i[0],s=i[1],l="\nMATCH (cm:CompartmentalizedMetabolite".concat(o,")-[").concat(s,"]-()\nRETURN {metabolite_count: COUNT(DISTINCT(cm))}\n"),e.next=4,y(l);case 4:return c=e.sent,d=c.metabolite_count,e.abrupt("return",d);case 7:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}(),le=function(e){var t=!(arguments.length>1&&void 0!==arguments[1])||arguments[1],n=e.replace(/\s\s+/g," ").replace(/^\s|\s$/g,"").replace(/["\\]/g,"");return!0===t&&(n=n.replace(/[~^!\-:/\\^$*+?.()|[\]{}]/g,"\\$&")),n},re=function(e,t){return C()(new Set(e)).filter((function(e){return new Set(t).has(e)}))},ce=n(354).map((function(e){return{label:e.short_name.replace("-GEM","Gem"),name:e.short_name}})),de=["CompartmentalizedMetabolite","Metabolite","Gene","Reaction","Subsystem","Compartment"],me=["ExternalDb","PubmedReference"],ue=function(e,t){var n=e.mid||e.id;return n in t?t[n].score:0};function pe(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function fe(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?pe(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):pe(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var he=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f,h;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.component,a=t.term,i=t.model,o=t.version,s=t.limit,l='\nCALL db.index.fulltext.queryNodes("fulltext", "'.concat(a," ").concat(a,'~")\nYIELD node, score\nWITH node, score, LABELS(node) as labelList\nOPTIONAL MATCH (node:').concat(i,":").concat(n,")\nOPTIONAL MATCH (node)-[").concat(o,"]-(parentNode:").concat(i,":").concat(n,")\nWHERE node:").concat(i," OR parentNode:").concat(i,"\nWITH DISTINCT(\n\tCASE\n                WHEN EXISTS(node.id) AND NOT apoc.coll.intersection(").concat(JSON.stringify(me),', LABELS(node))\n                THEN { id: node.id, labels: labelList, score: score }\n\t\tELSE { id: parentNode.id, labels: LABELS(parentNode), score: score }\n\tEND\n) as r\nWHERE\n\tany(r IN r.labels WHERE r="').concat(i,'")\n\tAND any(r IN r.labels WHERE r="').concat(n,'")\nRETURN r\n'),s&&(l+="\nLIMIT ".concat(s,"\n")),e.next=5,T(l);case 5:for(c=e.sent,d={},m=0,p=Object.entries(c);m<p.length;m++)f=u()(p[m],2),(h=f[1]).id in d||(d[h.id]=h);return e.abrupt("return",d);case 9:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),ge=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.metaboliteIds,i=t.model,o=t.version,s=t.limit,n){e.next=3;break}return e.abrupt("return",null);case 3:return l=JSON.stringify(n.map((function(e){return{mid:null,cmid:e}}))).replace(/"([^"]+)":/g,"$1:"),c="\nWITH ".concat(JSON.stringify(a)," as mids\nUNWIND\n  CASE\n      WHEN mids = [] THEN [null]\n      ELSE mids\n  END AS mid\nOPTIONAL MATCH (:Metabolite:").concat(i," {id:mid})-[").concat(o,"]-(cm:CompartmentalizedMetabolite)\nWITH cm.id as cmid1, mid\nWITH ").concat(l," as cmids2, cmid1, mid\nWITH cmids2+collect({mid: mid, cmid: cmid1}) as cmids\n\nUNWIND cmids as cmid\n\nCALL apoc.cypher.run('\n  MATCH (ms:MetaboliteState)-[").concat(o,"]-(:Metabolite)-[").concat(o,"]-(:CompartmentalizedMetabolite:").concat(i," {id: $cmid.cmid})\n  RETURN { id: $cmid.cmid, name: ms.name, mid: $cmid.mid } as data\n  \n  UNION\n  \n  MATCH (:CompartmentalizedMetabolite:").concat(i," {id: $cmid.cmid})-[").concat(o,"]-(c:Compartment)-[").concat(o,']-(cs:CompartmentState)\n  RETURN { id: $cmid.cmid, compartment: cs.name } as data\n\', {cmid:cmid}) yield value\nWITH apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as metabolites\nRETURN metabolites\n'),s&&(c+="\nLIMIT ".concat(s,"\n")),e.abrupt("return",T(c));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),ye=function(){var e=s()(r().mark((function e(t){var n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.model,i=t.version,n){e.next=3;break}return e.abrupt("return",null);case 3:return o="\nWITH ".concat(JSON.stringify(n),' as gids\nUNWIND gids as gid\nCALL apoc.cypher.run("\n  MATCH (gs:GeneState)-[').concat(i,"]-(:Gene:").concat(a,' {id: $gid})\n  RETURN { id: $gid, name: gs.name } as data\n", {gid:gid}) yield value\nRETURN apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as gene\n'),e.abrupt("return",T(o));case 5:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),be=function(){var e=s()(r().mark((function e(t){var n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.model,i=t.version,n){e.next=3;break}return e.abrupt("return",null);case 3:return o="\nWITH ".concat(JSON.stringify(n),' as rids\nUNWIND rids as rid\nCALL apoc.cypher.run("\n  MATCH (r:Reaction:').concat(a," {id: $rid})-[cmE").concat(i,"]-(cm:CompartmentalizedMetabolite)-[").concat(i,"]-(:Metabolite)-[").concat(i,"]-(ms:MetaboliteState)\n  MATCH (cm)-[").concat(i,"]-(c:Compartment)-[").concat(i,"]-(cs:CompartmentState)\n  USING JOIN on c\n  RETURN { id: $rid, metabolites: COLLECT(DISTINCT({name: ms.name, fullName: COALESCE(ms.name, '') + ' [' + COALESCE(cs.letterCode, '') + ']', stoichiometry: cmE.stoichiometry, outgoing: startnode(cmE)=cm})) } as data\n\n  UNION\n  \n  MATCH (:Reaction:").concat(a," {id: $rid})-[").concat(i,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(i,"]-(c:Compartment)-[").concat(i,']-(cs:CompartmentState)\n  USING JOIN on c\n  RETURN { id: $rid, compartment: COLLECT(DISTINCT({ id: c.id, name: cs.name })) } as data\n", {rid:rid}) yield value\nRETURN apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as reaction\n'),e.abrupt("return",T(o));case 5:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),_e=function(){var e=s()(r().mark((function e(t){var n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.model,i=t.version,n){e.next=3;break}return e.abrupt("return",null);case 3:return o="\nWITH ".concat(JSON.stringify(n),' as sids\nUNWIND sids as sid\nCALL apoc.cypher.run("\n  MATCH (ss:SubsystemState)-[').concat(i,"]-(:Subsystem:").concat(a,' {id: $sid})\n  RETURN { id: $sid, name: ss.name } as data\n", {sid:sid}) yield value\nRETURN apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as subsystem\n'),e.abrupt("return",T(o));case 5:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),ve=function(){var e=s()(r().mark((function e(t){var n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.model,i=t.version,n){e.next=3;break}return e.abrupt("return",null);case 3:return o="\nWITH ".concat(JSON.stringify(n),' as cids\nUNWIND cids as cid\nCALL apoc.cypher.run("\n  MATCH (cs:CompartmentState)-[').concat(i,"]-(:Compartment:").concat(a,' {id: $cid})\n  RETURN cs { id: $cid, .* } as data\n", {cid:cid}) yield value\nRETURN apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as compartment\n'),e.abrupt("return",T(o));case 5:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),we=function(){var e=s()(r().mark((function e(t){var n,a,o,s,l,c;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.searchTerm,a=t.model,o=t.version,s=t.limit,0!==(l=ce.filter((function(e){return e.label==a}))).length){e.next=4;break}throw new Error("Invalid model: ".concat(a));case 4:return e.next=6,Me({searchTerm:n,model:a,version:o,limit:s||10});case 6:return c=e.sent,e.abrupt("return",i()({},l[0].name,fe(fe({},c),{},{name:l[0].name})));case 8:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),Me=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f,h,g,y,b,_,v,w,M,T,I,N,G,x,H,S,E,R,z,O;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.searchTerm,a=t.model,i=t.version,o=t.limit,t.includeCounts,s=i?":V".concat(i):"",0!==(l=le(n,!0)).length){e.next=5;break}throw new Error("Empty searchTerm!");case 5:return c=de.map((function(e){return he({component:e,term:l,model:a,version:s,limit:o})})),e.next=8,Promise.all(c);case 8:for(d=e.sent,m=Object.assign.apply(Object,[{}].concat(C()(d))),p={},f=0,h=Object.entries(m);f<h.length;f++)g=u()(h[f],2),y=g[0],b=g[1],_=re(de,b.labels),p[_]||(p[_]=[]),p[_].push(y);return e.next=14,Promise.all([ge({ids:p.CompartmentalizedMetabolite||[],metaboliteIds:p.Metabolite||[],model:a,version:s,limit:o}),ye({ids:p.Gene,model:a,version:s}),be({ids:p.Reaction,model:a,version:s}),_e({ids:p.Subsystem,model:a,version:s,includeCounts:!0}),ve({ids:p.Compartment,model:a,version:s,includeCounts:!0})]);case 14:for(v=e.sent,w=u()(v,5),M=w[0],T=w[1],I=w[2],N=w[3],G=w[4],x={metabolites:M,genes:T,reactions:I,subsystems:N,compartments:G},H={},S=0,E=Object.entries(x);S<E.length;S++)R=u()(E[S],2),z=R[0],O=R[1],H[z]=O?O.map((function(e){return fe(fe({},e),{},{score:ue(e,m)})})):[];return e.abrupt("return",{metabolite:H.metabolites,gene:H.genes,reaction:H.reactions,subsystem:H.subsystems,compartment:H.compartments});case 25:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();function Te(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function Ie(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?Te(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):Te(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var Ce=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.metaboliteIds,i=t.model,o=t.version,s=t.limit,0!==n.length||0!==a.length){e.next=3;break}return e.abrupt("return",null);case 3:return l=JSON.stringify(n.map((function(e){return{mid:null,cmid:e}}))).replace(/"([^"]+)":/g,"$1:"),c="\nWITH ".concat(JSON.stringify(a)," as mids\nUNWIND\n  CASE\n      WHEN mids = [] THEN [null]\n      ELSE mids\n  END AS mid\nOPTIONAL MATCH (:Metabolite:").concat(i," {id:mid})-[").concat(o,"]-(cm:CompartmentalizedMetabolite)\nWITH cm.id as cmid1, mid\nWITH ").concat(l," as cmids2, cmid1, mid\nWITH cmids2+collect({mid: mid, cmid: cmid1}) as cmids\n\nUNWIND cmids as cmid\n\nCALL apoc.cypher.run('\n  MATCH (ms:MetaboliteState)-[").concat(o,"]-(:Metabolite)-[").concat(o,"]-(:CompartmentalizedMetabolite:").concat(i," {id: $cmid.cmid})\n  RETURN ms { id: $cmid.cmid, mid: $cmid.mid, .* } as data\n  \n  UNION\n  \n  MATCH (:CompartmentalizedMetabolite:").concat(i," {id: $cmid.cmid})-[").concat(o,"]-(c:Compartment)-[").concat(o,"]-(cs:CompartmentState)\n  RETURN { id: $cmid.cmid, compartment: cs { id: c.id, .* } } as data\n  \n  UNION\n  \n  MATCH (:CompartmentalizedMetabolite:").concat(i," {id: $cmid.cmid})-[").concat(o,"]-(:Reaction)-[").concat(o,"]-(s:Subsystem)\n  WITH DISTINCT s\n  MATCH (s)-[").concat(o,']-(ss:SubsystemState)\n  RETURN { id: $cmid.cmid, subsystem: COLLECT({id: s.id, name: ss.name}) } as data\n\', {cmid:cmid}) yield value\nWITH apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as metabolites\nRETURN metabolites\n'),s&&(c+="\nLIMIT ".concat(s,"\n")),e.abrupt("return",T(c));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),Ne=function(){var e=s()(r().mark((function e(t){var n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.model,i=t.version,n){e.next=3;break}return e.abrupt("return",null);case 3:return o="\nWITH ".concat(JSON.stringify(n),' as gids\nUNWIND gids as gid\nCALL apoc.cypher.run("\n  MATCH (:Gene:').concat(a," {id: $gid})-[").concat(i,"]-(r:Reaction)\n  WITH DISTINCT r\n  MATCH (r)-[").concat(i,"]-(s:Subsystem)\n  WITH DISTINCT s\n  MATCH (s)-[").concat(i,"]-(ss:SubsystemState)\n  RETURN { id: $gid, subsystem: COLLECT({ id: s.id, name: ss.name }) } as data\n  \n  UNION\n  \n  MATCH (:Gene:").concat(a," {id: $gid})-[").concat(i,"]-(:Reaction)-[").concat(i,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(i,"]-(c:Compartment)-[").concat(i,"]-(cs:CompartmentState)\n  USING JOIN on c\n  RETURN { id: $gid, compartment: COLLECT(DISTINCT({ id: c.id, name: cs.name })) } as data\n  \n  UNION\n\n  MATCH (gs:GeneState)-[").concat(i,"]-(:Gene:").concat(a,' {id: $gid})\n  RETURN { id: $gid, name: gs.name } as data\n", {gid:gid}) yield value\nRETURN apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as gene\n'),e.abrupt("return",T(o));case 5:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),Ge=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.model,i=t.version,o=t.includeMetabolites,n){e.next=3;break}return e.abrupt("return",null);case 3:return s="\nWITH ".concat(JSON.stringify(n),' as rids\nUNWIND rids as rid\nCALL apoc.cypher.run("\n  MATCH (rs:ReactionState)-[').concat(i,"]-(:Reaction:").concat(a," {id: $rid})\n  RETURN rs { id: $rid, .* } as data\n"),o&&(s+="\n  UNION\n  \n  MATCH (r:Reaction:".concat(a," {id: $rid})-[cmE").concat(i,"]-(cm:CompartmentalizedMetabolite)-[").concat(i,"]-(:Metabolite)-[").concat(i,"]-(ms:MetaboliteState)\n  MATCH (cm)-[").concat(i,"]-(c:Compartment)-[").concat(i,"]-(cs:CompartmentState)\n  USING JOIN on c\n  RETURN { id: $rid, metabolites: COLLECT(DISTINCT(ms {id: cm.id, compartment: cs.name, fullName: COALESCE(ms.name, '') + ' [' + COALESCE(cs.letterCode, '') + ']', stoichiometry: cmE.stoichiometry, outgoing: startnode(cmE)=cm, .*})) } as data\n")),s+="\n  UNION\n  \n  MATCH (:Reaction:".concat(a," {id: $rid})-[").concat(i,"]-(s:Subsystem)-[").concat(i,"]-(ss:SubsystemState)\n  USING JOIN on s\n  RETURN { id: $rid, subsystem: COLLECT(DISTINCT({ id: s.id, name: ss.name })) } as data\n  \n  UNION\n  \n  MATCH (:Reaction:").concat(a," {id: $rid})-[").concat(i,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(i,"]-(c:Compartment)-[").concat(i,']-(cs:CompartmentState)\n  USING JOIN on c\n  RETURN { id: $rid, compartment: COLLECT(DISTINCT({ id: c.id, name: cs.name })) } as data\n", {rid:rid}) yield value\nRETURN apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as reaction\n'),e.abrupt("return",T(s));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),xe=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.model,i=t.version,o=t.includeCounts,n){e.next=3;break}return e.abrupt("return",null);case 3:return s="\nWITH ".concat(JSON.stringify(n),' as sids\nUNWIND sids as sid\nCALL apoc.cypher.run("\n  MATCH (ss:SubsystemState)-[').concat(i,"]-(:Subsystem:").concat(a," {id: $sid})\n  RETURN { id: $sid, name: ss.name } as data\n"),o&&(s+=" \n  UNION\n  \n  MATCH (:Subsystem:".concat(a," {id: $sid})-[").concat(i,"]-(r:Reaction)\n  RETURN { id: $sid, reactionCount: COUNT(DISTINCT(r)) } as data\n  \n  UNION\n  \n  MATCH (:Subsystem:").concat(a," {id: $sid})-[").concat(i,"]-(r:Reaction)\n  WITH DISTINCT r\n  MATCH (r)-[").concat(i,"]-(cm:CompartmentalizedMetabolite)\n  RETURN { id: $sid, compartmentalizedMetaboliteCount: COUNT(DISTINCT cm) } as data\n  \n  UNION\n  \n  MATCH (:Subsystem:").concat(a," {id: $sid})-[").concat(i,"]-(r:Reaction)\n  WITH DISTINCT r\n  MATCH (r)-[").concat(i,"]-(g:Gene)\n  RETURN { id: $sid, geneCount: COUNT(DISTINCT g) } as data\n")),s+=" \n  UNION\n  \n  MATCH (:Subsystem:".concat(a," {id: $sid})-[").concat(i,"]-(:Reaction)-[").concat(i,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(i,"]-(c:Compartment)-[").concat(i,']-(cs:CompartmentState)\n  USING JOIN on c\n  RETURN { id: $sid, compartment: COLLECT(DISTINCT({ id: c.id, name: cs.name })) } as data\n", {sid:sid}) yield value\nRETURN apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as subsystem\n'),e.abrupt("return",T(s));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),He=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.ids,a=t.model,i=t.version,o=t.includeCounts,n){e.next=3;break}return e.abrupt("return",null);case 3:return s="\nWITH ".concat(JSON.stringify(n),' as cids\nUNWIND cids as cid\nCALL apoc.cypher.run("\n  MATCH (cs:CompartmentState)-[').concat(i,"]-(:Compartment:").concat(a," {id: $cid})\n  RETURN cs { id: $cid, .* } as data\n"),o&&(s+=" \n  UNION\n  \n  MATCH (:Compartment:".concat(a," {id: $cid})-[").concat(i,"]-(:CompartmentalizedMetabolite)-[").concat(i,"]-(r:Reaction)\n  RETURN { id: $cid, reactionCount: COUNT(DISTINCT(r)) } as data\n  \n  UNION\n  \n  MATCH (:Compartment:").concat(a," {id: $cid})-[").concat(i,"]-(cm:CompartmentalizedMetabolite)\n  RETURN { id: $cid, compartmentalizedMetaboliteCount: COUNT(DISTINCT cm) } as data\n  \n  UNION\n  \n  MATCH (:Compartment:").concat(a," {id: $cid})-[").concat(i,"]-(:CompartmentalizedMetabolite)-[").concat(i,"]-(r:Reaction)\n  WITH DISTINCT r\n  MATCH (r)-[").concat(i,"]-(g:Gene)\n  RETURN { id: $cid, geneCount: COUNT(DISTINCT g) } as data\n  \n  UNION\n  \n  MATCH (:Compartment:").concat(a," {id: $cid})-[").concat(i,"]-(:CompartmentalizedMetabolite)-[").concat(i,"]-(r:Reaction)\n  WITH DISTINCT r\n  MATCH (r)-[").concat(i,"]-(s:Subsystem)\n  RETURN { id: $cid, subsystemCount: COUNT(DISTINCT s) } as data\n")),s+=' \n", {cid:cid}) yield value\nRETURN apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as compartment\n',e.abrupt("return",T(s));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),Se=function(){var e=s()(r().mark((function e(t){var n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.searchTerm,a=t.version,i=t.limit,e.next=3,Promise.all(ce.map((function(e){return Ee({searchTerm:n,version:a,model:e.label,limit:i,includeCounts:!0})})));case 3:return o=e.sent,e.abrupt("return",ce.reduce((function(e,t,n){return e[t.name]=Ie(Ie({},o[n]),{},{name:t.name}),e}),{}));case 5:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),Ee=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f,h,g,y,b,_,v,w,M,I,C,N,G,x,H,S,E,R,z,O,L,A,B,k;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.searchTerm,a=t.model,i=t.version,o=t.limit,t.includeCounts,s=i?":V".concat(i):"",l=le(n,!0),c='\nCALL db.index.fulltext.queryNodes("fulltext", "'.concat(l," ").concat(l,'~")\nYIELD node, score\nWITH node, score, LABELS(node) as labelList\nOPTIONAL MATCH (node)-[').concat(s,"]-(parentNode:").concat(a,")\nWHERE node:").concat(a," OR parentNode:").concat(a,"\nWITH DISTINCT(\n\tCASE\n                WHEN EXISTS(node.id) AND NOT apoc.coll.intersection(").concat(JSON.stringify(me),', LABELS(node))\n                THEN { id: node.id, labels: labelList, score: score }\n\t\tELSE { id: parentNode.id, labels: LABELS(parentNode), score: score }\n\tEND\n) as r \nWHERE any(r IN r.labels WHERE r="').concat(a,'")\nRETURN r\n'),o&&(c+="\nLIMIT ".concat(o,"\n")),e.next=7,T(c);case 7:for(d=e.sent,m={},p=0,f=Object.entries(d);p<f.length;p++)(h=u()(f[p],2))[0],(g=h[1]).id in m||(m[g.id]=g);for(y={},b=0,_=Object.entries(m);b<_.length;b++)v=u()(_[b],2),w=v[0],M=v[1],I=re(de,M.labels),y[I]||(y[I]=[]),y[I].push(w);return e.next=14,Promise.all([Ce({ids:y.CompartmentalizedMetabolite||[],metaboliteIds:y.Metabolite||[],model:a,version:s,limit:o}),Ne({ids:y.Gene,model:a,version:s}),Ge({ids:y.Reaction,model:a,version:s,includeMetabolites:!!o}),xe({ids:y.Subsystem,model:a,version:s,includeCounts:!0}),He({ids:y.Compartment,model:a,version:s,includeCounts:!0})]);case 14:for(C=e.sent,N=u()(C,5),G=N[0],x=N[1],H=N[2],S=N[3],E=N[4],R={metabolites:G,genes:x,reactions:H,subsystems:S,compartments:E},z={},O=0,L=Object.entries(R);O<L.length;O++)A=u()(L[O],2),B=A[0],k=A[1],z[B]=k?k.reduce((function(e,t){return Object.keys(t).length&&e.push(Ie(Ie({},t),{},{score:ue(t,m)})),e}),[]):[];return e.abrupt("return",{metabolite:z.metabolites,gene:z.genes,reaction:z.reactions,subsystem:z.subsystems,compartment:z.compartments});case 25:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),Re=function(){var e=s()(r().mark((function e(t){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",t.model?we(t):Se(t));case 1:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),ze="Reaction",Oe="Gene",Le="Metabolite",Ae="Subsystem",Be="Compartment",ke=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:n=t.nodeType,a=t.id,i=t.model,o=t.version,s=t.limit,l=t.isForAllCompartments,c=b(i,o),d=u()(c,2),m=d[0],p=d[1],e.t0=n,e.next=e.t0===ze?5:e.t0===Oe?7:e.t0===Le?9:e.t0===Ae?11:e.t0===Be?13:15;break;case 5:return f="\nMATCH (r1:Reaction".concat(m," {id: '").concat(a,"'})-[cms1").concat(p,"]-(:CompartmentalizedMetabolite)-[").concat(p,"]-(m:Metabolite)\nWITH r1, count(cms1) as ccms1, collect(distinct(m)) as ms\nUNWIND ms as m\nMATCH (m)-[").concat(p,"]-(:CompartmentalizedMetabolite)-[cms2").concat(p,"]-(r:Reaction)\nWITH r1, r, count(cms2) as ccms2, ccms1\nWHERE ccms1 = ccms2\nMATCH (r)-[cms3").concat(p,"]-(:CompartmentalizedMetabolite)\nWITH ccms1, count(cms3) as ccms3, r1, r\nWHERE ccms1 = ccms3 and r1.id <> r.id"),e.abrupt("break",16);case 7:return f="\nMATCH (:Gene".concat(m," {id: '").concat(a,"'})-[").concat(p,"]-(r:Reaction)"),e.abrupt("break",16);case 9:return f=l?"\nMATCH (:CompartmentalizedMetabolite".concat(m," {id: '").concat(a,"'})-[").concat(p,"]-(m:Metabolite)\nWITH m\nMATCH (m)-[").concat(p,"]-(:CompartmentalizedMetabolite").concat(m,")-[").concat(p,"]-(r:Reaction)"):"\nMATCH (:CompartmentalizedMetabolite".concat(m," {id: '").concat(a,"'})-[").concat(p,"]-(r:Reaction)"),e.abrupt("break",16);case 11:return f="\nMATCH (:Subsystem".concat(m," {id: '").concat(a,"'})-[").concat(p,"]-(r:Reaction)"),e.abrupt("break",16);case 13:return f="\nMATCH (:Compartment".concat(m," {id: '").concat(a,"'})-[").concat(p,"]-(:CompartmentalizedMetabolite)-[").concat(p,"]-(r:Reaction)"),e.abrupt("break",16);case 15:throw new Error("Unrecognized node type: ".concat(n));case 16:return f+=s?"\nWITH collect (DISTINCT r)[..".concat(s,"] as reaction\n"):"\nWITH collect (DISTINCT r) as reaction\n",f+='\nCALL apoc.cypher.mapParallel2("\n  WITH (_) as reaction\n  MATCH (rs:ReactionState)-['.concat(p,"]-(:Reaction").concat(m," {id: reaction.id})\n  WITH rs, reaction\n\n  OPTIONAL MATCH (:Reaction").concat(m," {id: reaction.id})-[").concat(p,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm, rs, reaction\n  MATCH (cm)-[").concat(p,"]-(c:Compartment)-[").concat(p,"]-(cs:CompartmentState)\n  USING JOIN on c\n  WITH rs, reaction, COLLECT(DISTINCT(cs {id: c.id, .*})) as compres\n\n  OPTIONAL MATCH (:Reaction").concat(m," {id: reaction.id})-[").concat(p,"]-(s:Subsystem)\n  WITH DISTINCT s, rs, reaction, compres\n  MATCH(s)-[").concat(p,"]-(ss:SubsystemState)\n  WITH rs, reaction, compres, COLLECT(DISTINCT(ss {id: s.id, .*})) as subres\n\n  OPTIONAL MATCH (:Reaction").concat(m," {id: reaction.id})-[").concat(p,"]-(g:Gene)\n  WITH DISTINCT (g), rs, reaction, compres, subres\n  OPTIONAL MATCH (g)-[").concat(p,"]-(gs:GeneState)\n  WITH rs, reaction, compres, subres, COLLECT(DISTINCT(gs {id: g.id, .*})) as genres\n\n  OPTIONAL MATCH (:Reaction").concat(m," {id: reaction.id})-[cmE").concat(p,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm, cmE, rs, reaction, compres, subres, genres\n  OPTIONAL MATCH (cm)-[").concat(p,"]-(c:Compartment)-[").concat(p,"]-(cs:CompartmentState)\n  USING JOIN on c\n  WITH cm, cmE, c, cs, rs, reaction, compres, subres, genres\n  OPTIONAL MATCH (cm)-[").concat(p,"]-(:Metabolite)-[").concat(p,"]-(ms:MetaboliteState)\n  WITH rs, reaction, compres, subres, genres, COLLECT(DISTINCT(ms {id: cm.id, fullName: COALESCE(ms.name, '') + ' [' + COALESCE(cs.letterCode, '') + ']',  compartmentId: c.id, stoichiometry: cmE.stoichiometry, outgoing: startnode(cmE)=cm, .*})) as metres\n\n  RETURN  rs { id: reaction.id, .*,  compartments: compres, subsystems: subres, genes:genres, metabolites: metres } as data\n\",{}, reaction, 50, 60) YIELD value\nRETURN apoc.map.mergeList(apoc.coll.flatten(apoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), \"id\"), [value.data.id])\n)) as reactions\nORDER BY reactions.id\n"),e.abrupt("return",T(f));case 19:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),De=function(e){var t=e.id,n=e.model,a=e.version,i=e.limit;return ke({id:t,model:n,version:a,nodeType:ze,limit:i})},We=function(e){var t=e.id,n=e.model,a=e.version,i=e.limit;return ke({id:t,model:n,version:a,nodeType:Oe,limit:i})},Fe=function(e){var t=e.id,n=e.model,a=e.version,i=e.limit,o=e.isForAllCompartments;return ke({id:t,model:n,version:a,nodeType:Le,limit:i,isForAllCompartments:o})},je=function(e){var t=e.id,n=e.model,a=e.version,i=e.limit;return ke({id:t,model:n,version:a,nodeType:Ae,limit:i})},Pe=function(e){var t=e.id,n=e.model,a=e.version,i=e.limit;return ke({id:t,model:n,version:a,nodeType:Be,limit:i})};const Ue=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.id,a=t.model,i=t.version,o=b(a,i),s=u()(o,2),l=s[0],c=s[1],d="\nMATCH (cm:CompartmentalizedMetabolite".concat(l,")-[").concat(c,"]-(m:Metabolite)-[").concat(c,"]-(rcm:CompartmentalizedMetabolite)-[").concat(c,"]-(c:Compartment)-[").concat(c,']-(cs:CompartmentState)\nWHERE cm.id="').concat(n,'"\nMATCH (m)-[').concat(c,"]-(ms:MetaboliteState)\nRETURN {\n  id: rcm.id,\n  fullName: COALESCE(ms.name, '') + ' [' + COALESCE(cs.letterCode, '') + ']',\n  compartment: cs {\n    id: c.id, \n    .*\n  }\n} as metabolites\nORDER BY metabolites.id"),e.abrupt("return",T(d));case 4:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();var Ve=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f,h,g,y,_;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.model,a=t.version,i=t.componentTypes,o=void 0===i?{compartment:!0,compartmentalizedMetabolite:!0,gene:!0,reaction:!0,subsystem:!0}:i,0!==Object.values(o).filter((function(e){return!0===e})).length){e.next=3;break}throw new Error("At least 1 component type is needed");case 3:return s=b(n,a),l=u()(s,2),c=l[0],d=l[1],m=o.compartment,p=o.compartmentalizedMetabolite,f=o.gene,h=o.reaction,g=o.subsystem,y="",f&&(y+="\nMATCH (:GeneState)-[".concat(d,"]-(g:Gene").concat(c,')\nWITH g.id as gid, rand() as r\nORDER BY r LIMIT 2\nCALL apoc.cypher.run("\n  MATCH (gs:GeneState)-[').concat(d,"]-(:Gene").concat(c," {id: $gid})-[").concat(d,"]-(re:Reaction)\n  RETURN { id: $gid, name: gs.name, reactionCount: COUNT(DISTINCT(re)) } as data\n  UNION\n  MATCH (:Gene").concat(c," {id: $gid})-[").concat(d,"]-(:Reaction)-[").concat(d,"]-(ss:Subsystem)\n  RETURN { id: $gid, subsystemCount: COUNT(DISTINCT(ss)) } as data\n  UNION\n  MATCH (:Gene").concat(c," {id: $gid})-[").concat(d,"]-(:Reaction)-[").concat(d,"]-(:CompartmentalizedMetabolite)-[").concat(d,']-(c:Compartment)\n  RETURN { id: $gid, compartmentCount: COUNT(DISTINCT(c)) } as data\n", {gid:gid}) yield value\nRETURN { gene: apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) } as xs\n')),p&&(y.length>0&&(y+="\nUNION"),y+="\nMATCH (:Metabolite)-[".concat(d,"]-(cm:CompartmentalizedMetabolite").concat(c,')\nWITH cm.id as cmid, rand() as r\nORDER BY r LIMIT 2\nCALL apoc.cypher.run("\n  MATCH (ms:MetaboliteState)-[').concat(d,"]-(:Metabolite)-[").concat(d,"]-(:CompartmentalizedMetabolite").concat(c," {id: $cmid})\n  RETURN { id: $cmid, name: ms.name, formula: ms.formula } as data\n  UNION\n  MATCH (re:Reaction)-[").concat(d,"]-(:CompartmentalizedMetabolite").concat(c," {id: $cmid})\n  RETURN { id: $cmid, reactionCount: count(distinct(re)) } as data\n  UNION\n  MATCH (cs:CompartmentState)-[").concat(d,"]-(:Compartment)-[").concat(d,"]-(cm:CompartmentalizedMetabolite").concat(c,' {id: $cmid})\n  RETURN { id: $cmid, compartment: cs.name } as data\n", {cmid:cmid}) yield value\nRETURN { metabolite: apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) } as xs\n')),m&&(y.length>0&&(y+="\nUNION"),y+="\nMATCH (:CompartmentState)-[".concat(d,"]-(c:Compartment").concat(c,')\nWITH c.id as cid, rand() as r\nORDER BY r LIMIT 1\nCALL apoc.cypher.run("\n  MATCH (cs:CompartmentState)-[').concat(d,"]-(:Compartment").concat(c," {id: $cid})-[").concat(d,"]-(:CompartmentalizedMetabolite)-[").concat(d,"]-(re:Reaction)\n  RETURN { id: $cid, name: cs.name, reactionCount: count(distinct(re)) } as data\n  UNION\n  MATCH (:Compartment").concat(c," {id: $cid})-[").concat(d,"]-(cm:CompartmentalizedMetabolite)\n  RETURN { compartmentalizedMetaboliteCount: count(distinct cm) } as data\n  UNION\n  MATCH (:Compartment").concat(c," {id: $cid})-[").concat(d,"]-(:CompartmentalizedMetabolite)-[").concat(d,"]-(:Reaction)-[").concat(d,"]-(g:Gene)\n  RETURN { geneCount: count(distinct g) } as data\n  UNION\n  MATCH (:Compartment").concat(c," {id: $cid})-[").concat(d,"]-(:CompartmentalizedMetabolite)-[").concat(d,"]-(:Reaction)-[").concat(d,"]-(s:Subsystem)-[").concat(d,']-(sss:SubsystemState)\n  USING JOIN on s\n  RETURN { majorSubsystems: COLLECT(DISTINCT(sss.name))[..15] } as data\n", {cid:cid}) yield value\nRETURN { compartment: apoc.map.mergeList(COLLECT(value.data)) } as xs\n')),h&&(y.length>0&&(y+="\nUNION"),y+="\nMATCH (rs:ReactionState)-[".concat(d,"]-(re:Reaction").concat(c,")\nWITH re, rs, rand() as r\nORDER BY r LIMIT 2\nMATCH (re)-[").concat(d,"*2]-(c:Compartment)\nWITH re, rs, collect(distinct c) as comps\nOPTIONAL MATCH (re)<-[").concat(d,"]-(:CompartmentalizedMetabolite)-[").concat(d,"*2]-(ms:MetaboliteState)\nWITH re, rs, comps, collect(ms.name) as reactants\nOPTIONAL MATCH (re)-[").concat(d,"]->(:CompartmentalizedMetabolite)-[").concat(d,"*2]-(ms:MetaboliteState)\nWITH re, rs, comps, reactants, collect(ms.name) as products\nOPTIONAL MATCH (re)-[").concat(d,"]-(g:Gene)\nWITH re, rs, reactants, products, comps, count(g) as geneCount\nOPTIONAL MATCH (re)-[").concat(d,"]-(s:Subsystem)\nRETURN { reaction: { id: re.id, name: rs.name, reactants: reactants, products: products, geneCount: geneCount, compartmentCount: size(comps), subsystemCount: count(s) }, reversible: rs.reversible } as xs\n")),g&&(y.length>0&&(y+="\nUNION"),y+="\nMATCH (:SubsystemState)-[".concat(d,"]-(s:Subsystem").concat(c,')\nWITH s.id as sid, rand() as r\nORDER BY r LIMIT 2\nCALL apoc.cypher.run("\n  MATCH (ss:SubsystemState)-[').concat(d,"]-(:Subsystem").concat(c," {id: $sid})-[").concat(d,"]-(re:Reaction)\n  RETURN { id: $sid, name: ss.name, reactionCount: count(distinct(re)) } as data\n  UNION\n  MATCH (:Subsystem").concat(c," {id: $sid})-[").concat(d,"]-(:Reaction)-[").concat(d,"]-(cm:CompartmentalizedMetabolite)\n  RETURN { id: $sid, compartmentalizedMetaboliteCount: count(distinct cm) } as data\n  UNION\n  MATCH (:Subsystem").concat(c," {id: $sid})-[").concat(d,"]-(:Reaction)-[").concat(d,"]-(g:Gene)\n  RETURN { id: $sid, geneCount: count(distinct g) } as data\n  UNION\n  MATCH (:Subsystem").concat(c," {id: $sid})-[").concat(d,"]-(:Reaction)-[").concat(d,"]-(cm:CompartmentalizedMetabolite)\n  WITH DISTINCT cm\n  MATCH (cm)-[").concat(d,']-(c:Compartment)\n  RETURN { id: $sid, compartmentCount: count(distinct c) } as data\n", {sid:sid}) yield value\nRETURN { subsystem: apoc.map.mergeList(apoc.coll.flatten(\n\tapoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) } as xs\n')),e.next=13,T(y);case 13:return _=e.sent,e.abrupt("return",_.reduce((function(e,t){var n=u()(Object.entries(t)[0],2),a=n[0],i=n[1];if("compartment"===a)e[a]=i;else{var o="".concat(a,"s");e[o]||(e[o]=[]),e[o]=[].concat(C()(e[o]),[i])}return e}),{}));case 15:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();const qe=Ve,$e=require("worker_threads"),Je=function(e){return new Promise((function(t,n){var a=new $e.Worker("\n    const { parentPort } = require('worker_threads');\n    const populateWithLayout = require('./src/utils/3d-network.js');\n    parentPort.once('message',\n      data => parentPort.postMessage(populateWithLayout(data)));\n  ",{eval:!0});a.on("message",(function(e){return t(e)})),a.on("error",n),a.on("exit",(function(e){0!==e&&n(new Error("stopped with  ".concat(e," exit code")))})),a.postMessage(e)}))};function Ke(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function Ye(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?Ke(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):Ke(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var Xe=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f,h,g,_;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.id,a=t.model,i=t.version,o=b(a,i),s=u()(o,2),l=s[0],c=s[1],d="\nMATCH (comp".concat(l,' {id: "').concat(n,"\"})\nWHERE comp:Gene OR comp:CompartmentalizedMetabolite\n\nWITH CASE WHEN comp:Gene THEN 'gene' ELSE 'metabolite' END AS compType, comp\n\nCALL apoc.when(\n  comp:Gene,\n  'MATCH (g:Gene").concat(l," {id: comp.id})-[").concat(c,"]-(gs:GeneState)'\n  +\t'RETURN {id: g.id, name: gs.name, type: compType} as component',\n  'MATCH (cm:CompartmentalizedMetabolite").concat(l," {id: comp.id})-[").concat(c,"]-(:Metabolite)-[").concat(c,"]-(ms:MetaboliteState)'\n  + ' RETURN {id: cm.id, name: ms.name, type: compType} as component',\n  {comp:comp, compType:compType})\nYIELD value as v\n\nWITH v.component as component\nMATCH (").concat(l," {id: component.id})-[").concat(c,']-(r:Reaction)\nWITH component, collect(r) as reactions\n\nCALL apoc.cypher.mapParallel2("\n WITH (_) as reaction\n MATCH (reaction)-[').concat(c,"]-(cm:CompartmentalizedMetabolite)\n WITH DISTINCT cm as compm, reaction\n MATCH (compm)-[").concat(c,"]-(c:Compartment)-[").concat(c,"]-(cs:CompartmentState)\n WITH COLLECT(DISTINCT({ id: c.id, name: cs.name })) as compartments, reaction\n OPTIONAL MATCH (reaction)-[").concat(c,"]-(s:Subsystem)\n WITH DISTINCT s as subs, reaction, compartments\n OPTIONAL MATCH(subs)-[").concat(c,"]-(ss:SubsystemState)\n WITH reaction,compartments, COLLECT(ss.name) as subsystem\n OPTIONAL MATCH (reaction)-[").concat(c,"]-(g:Gene)\n WITH DISTINCT (g) as gen, reaction, compartments, subsystem\n OPTIONAL MATCH (gen)-[").concat(c,"]-(gs:GeneState)\n WITH reaction, compartments, subsystem, COLLECT(DISTINCT(gs {id: gen.id, .*})) as genes\n MATCH (reaction)-[cmE").concat(c,"]-(cm:CompartmentalizedMetabolite)\n WITH DISTINCT cm, cmE, reaction, compartments, subsystem, genes\n MATCH (c:Compartment)-[").concat(c,"]-(cm)-[").concat(c,"]-(:Metabolite)-[").concat(c,"]-(ms:MetaboliteState)\n WITH DISTINCT cm, cmE, reaction, compartments, subsystem, genes, c, ms\n MATCH (reaction)-[").concat(c,']-(rs:ReactionState)\n RETURN {id: reaction.id, reversible: rs.reversible, subsystem: subsystem, compartments: compartments, genes: genes, metabolites: COLLECT(DISTINCT({id: cm.id, name: ms.name,  compartmentId: c.id, outgoing: startnode(cmE)=cm})) } as data", {}, reactions, 50) YIELD value\n\nWITH apoc.map.mergeList(apoc.coll.flatten(\n  apoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) as reaction, component\nRETURN { component: component, reactions: COLLECT(reaction)}\n'),e.prev=3,e.next=6,y(d);case 6:m=e.sent,e.next=13;break;case 9:throw e.prev=9,e.t0=e.catch(3),new Error("Invalid id: ".concat(n));case 13:return p=[],f=[],h=new Set,g=function(e,t){var n="".concat(e,"-").concat(t),a="".concat(t,"-").concat(e);h.has(n)||h.has(a)||(p.push({s:e,t}),h.add(n))},m.reactions.forEach((function(e){var t=e.genes,a=e.metabolites;a.forEach((function(e){h.has(e.id)||(f.push({g:"m",id:e.id,n:e.name||e.name}),h.add(e.id),n!==e.id&&g(n,e.id))})),t.forEach((function(e){h.has(e.id)||(f.push({g:"e",id:e.id,n:e.name||e.id}),h.add(e.id),n!==e.id&&g(n,e.id)),a.forEach((function(t){return g(e.id,t.id)}))}))})),e.next=20,Je({nodes:f,links:p,dim:2,mainNodeID:n,reCenter:!0});case 20:return _=e.sent,e.abrupt("return",{result:m,network:_});case 22:case"end":return e.stop()}}),e,null,[[3,9]])})));return function(t){return e.apply(this,arguments)}}(),Ze=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,u,p,f,h;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.id,a=t.model,i=t.version,o=t.expanded,e.next=3,Xe({id:n,model:a,version:i});case 3:return s=e.sent,l=s.network,c=s.result,d=new Set,m={},e.next=10,Promise.all(o.filter((function(e){return e.trim()})).map((function(e){return Xe({id:e,model:a,version:i})})));case 10:for(u=e.sent,p=function(e){var t=u[e],n=o[e];m[n]=t.result.component.name,t.network.links.forEach((function(e){return function(e,t){var n="".concat(e,"-").concat(t),a="".concat(t,"-").concat(e);d.has(n)||d.has(a)||(l.links.push({s:e,t}),d.add(n))}(e.s,e.t)})),t.network.nodes.forEach((function(e){l.nodes.map((function(e){return e.id})).includes(e.id)||l.nodes.push({g:e.g,id:e.id,n:e.n})})),c.reactions=[].concat(C()(c.reactions),C()(t.result.reactions))},f=0;f<u.length;f++)p(f);return e.next=15,Je(Ye(Ye({},l),{},{dim:2,mainNodeID:o[o.length-1],reCenter:!0}));case 15:return h=e.sent,e.abrupt("return",{result:c,network:h,expandedNodes:m});case 17:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();const Qe=require("fs");var et=n.n(Qe);const tt=require("readline");var nt=n.n(tt);function at(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function it(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?at(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):at(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}function ot(e){var t,n,a,i=2;for("undefined"!=typeof Symbol&&(n=Symbol.asyncIterator,a=Symbol.iterator);i--;){if(n&&null!=(t=e[n]))return t.call(e);if(a&&null!=(t=e[a]))return new st(t.call(e));n="@@asyncIterator",a="@@iterator"}throw new TypeError("Object is not async iterable")}function st(e){function t(e){if(Object(e)!==e)return Promise.reject(new TypeError(e+" is not an object."));var t=e.done;return Promise.resolve(e.value).then((function(e){return{value:e,done:t}}))}return st=function(e){this.s=e,this.n=e.next},st.prototype={s:null,n:null,next:function(){return t(this.n.apply(this.s,arguments))},return:function(e){var n=this.s.return;return void 0===n?Promise.resolve({value:e,done:!0}):t(n.apply(this.s,arguments))},throw:function(e){var n=this.s.return;return void 0===n?Promise.reject(e):t(n.apply(this.s,arguments))}},new st(e)}var lt=function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l,c,d,m,u;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:a=nt().createInterface({input:et().createReadStream("/project/svg/".concat(n,"/").concat(t.filename)),output:process.stdout,terminal:!1}),i=new Set,o=!1,s=!1,e.prev=4,c=ot(a);case 6:return e.next=8,c.next();case 8:if(!(o=!(d=e.sent).done)){e.next=14;break}(m=d.value).includes('class="rea"')&&(u=m.match(/id="(.+)"\s/))&&i.add(u[1]);case 11:o=!1,e.next=6;break;case 14:e.next=20;break;case 16:e.prev=16,e.t0=e.catch(4),s=!0,l=e.t0;case 20:if(e.prev=20,e.prev=21,!o||null==c.return){e.next=25;break}return e.next=25,c.return();case 25:if(e.prev=25,!s){e.next=28;break}throw l;case 28:return e.finish(25);case 29:return e.finish(20);case 30:return e.abrupt("return",it(it({},t),{},{mapReactionIdSet:C()(i)}));case 31:case"end":return e.stop()}}),e,null,[[4,16,20,30],[21,,25,29]])})));return function(t,n){return e.apply(this,arguments)}}(),rt=function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,Promise.all(t.map(function(){var e=s()(r().mark((function e(t){var a;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,Promise.all(t.svgs.map((function(e){return lt(e,n)})));case 2:return a=e.sent,e.abrupt("return",it(it({},t),{},{svgs:a}));case 4:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}()));case 2:return e.abrupt("return",e.sent);case 3:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}(),ct=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f,h,g,y,_;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.model,a=t.version,i=b(n,a),o=u()(i,2),s=o[0],l=o[1],c="\nMATCH (:CompartmentState)-[".concat(l,"]-(c:Compartment").concat(s,')\nCALL apoc.cypher.run("\n  MATCH (cs:CompartmentState)-[').concat(l,"]-(:Compartment").concat(s," {id: $cid})\n  RETURN cs { id: $cid, .* } as data\n  \n  UNION\n  \n  MATCH (:Compartment").concat(s," {id: $cid})-[").concat(l,"]-(:CompartmentalizedMetabolite)-[").concat(l,"]-(r:Reaction)\n  RETURN { id: $cid, reactionList: COLLECT(DISTINCT(r.id)) } as data\n  \n  UNION\n  \n  MATCH (csvg:SvgMap)-[").concat(l,"]-(:Compartment").concat(s,' {id: $cid})\n  RETURN { id: $cid, svgs: COLLECT(DISTINCT(csvg {.*})) } as data\n", {cid:c.id}) yield value\nRETURN { compartment: apoc.map.mergeList(apoc.coll.flatten(\n  apoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) } as data ORDER BY data.compartment.name\n\nUNION\n\nMATCH (:SubsystemState)-[').concat(l,"]-(s:Subsystem").concat(s,')\nCALL apoc.cypher.run("\n  MATCH (ss:SubsystemState)-[').concat(l,"]-(s:Subsystem {id: $sid})\n  RETURN ss { id: $sid, .* } as data\n  \n  UNION\n  \n  MATCH (:Subsystem").concat(s," {id: $sid})-[").concat(l,"]-(r:Reaction)\n  RETURN { id: $sid, reactionList: COLLECT(DISTINCT(r.id)) } as data\n  \n  UNION\n  \n  MATCH (:Subsystem").concat(s," {id: $sid})-[").concat(l,']-(ssvg:SvgMap)\n  RETURN { id: $sid, svgs: COLLECT(DISTINCT(ssvg {.*})) } as data\n", {sid:s.id}) yield value\nRETURN { subsystem: apoc.map.mergeList(apoc.coll.flatten(\n  apoc.map.values(apoc.map.groupByMulti(COLLECT(value.data), "id"), [value.data.id])\n)) } as data ORDER BY data.subsystem.name\n'),d="\nMATCH (svg:SvgMap".concat(s,")\nWHERE NOT (svg)--()\nRETURN svg { id: svg.id, name: svg.customName, svgs: [svg {.*}]}\n"),e.next=6,Promise.all([T(c),T(d)]);case 6:return m=e.sent,p=u()(m,2),f=p[0],h=p[1],g={compartments:f.filter((function(e){return!!e.compartment})).reduce((function(e,t){return[].concat(C()(e),[t.compartment])}),[]),subsystems:f.filter((function(e){return!!e.subsystem})).reduce((function(e,t){return[].concat(C()(e),[t.subsystem])}),[]),customs:h},e.next=13,rt(g.compartments,"".concat(n.slice(0,-3),"-GEM"));case 13:return y=e.sent,e.next=16,rt(g.subsystems,"".concat(n.slice(0,-3),"-GEM"));case 16:return _=e.sent,e.abrupt("return",it(it({},g),{},{compartments:y,subsystems:_}));case 18:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),dt=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.model,a=t.version,i=t.searchTerm,o=b(n,a),s=u()(o,2),l=s[0],c=s[1],d='\nCALL db.index.fulltext.queryNodes("fulltext", "'.concat(i,'~")\nYIELD node\nOPTIONAL MATCH (node)-[').concat(c,"]-(parentNode").concat(l,")\nWHERE node").concat(l," OR parentNode").concat(l,"\nRETURN [n in COLLECT(DISTINCT(\n\tCASE\n\t\tWHEN EXISTS(node.id) THEN node.id\n\t\tELSE parentNode.id\n\tEND\n))[..100] WHERE n IS NOT NULL]"),e.abrupt("return",y(d));case 4:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();const mt=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f,h,g,_;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.model,a=t.version,i=t.type,o=t.id,s=b(n,a),l=u()(s,2),c=l[0],d=l[1],m="compartment"===i?'(:Compartment {id: "'.concat(o,'"})-[').concat(d,"]-(:CompartmentalizedMetabolite)-[").concat(d,"]-"):"",p="compartment"===i?"-[".concat(d,']-(:Compartment {id: "').concat(o,'"})'):"",f="subsystem"===i?'(s:Subsystem {id: "'.concat(o,'"})-[').concat(d,"]-"):"",h="".concat(m).concat(f),g="\nCALL apoc.cypher.run('\n  MATCH ".concat(h,"(r:Reaction").concat(c,")-[").concat(d,']-(:ReactionState)\n  USING JOIN on r\n  RETURN {\n    nodes: COLLECT(DISTINCT { g: "r", id: r.id, n: r.id })\n  } as data\n  \n  UNION\n  \n  MATCH ').concat(h,"(r:Reaction").concat(c,")-[").concat(d,"]-(g:Gene)\n  USING JOIN on r\n  MATCH (g)-[").concat(d,']-(gs:GeneState)\n  RETURN {\n    nodes: COLLECT(DISTINCT { g: "e", id: g.id, n: gs.name }),\n    links: COLLECT(DISTINCT { s: g.id, t: r.id })\n  } as data\n  \n  UNION\n  \n  MATCH ').concat(f,"(r:Reaction").concat(c,")-[cmE").concat(d,"]-(cm:CompartmentalizedMetabolite)").concat(p,"\n  USING JOIN on ").concat(""===f?"cm":"r","\n  MATCH (cm)-[").concat(d,"]-(:Metabolite)-[").concat(d,']-(ms:MetaboliteState)\n  RETURN {\n    nodes: COLLECT(DISTINCT { g: "m", id: cm.id, n: ms.name }),\n    links: COLLECT(DISTINCT(\n      CASE\n\tWHEN startnode(cmE)=cm THEN { s: cm.id, t: r.id }\n\tELSE { s: r.id, t: cm.id }\n      END\n    ))\n  } as data\n\', {}) yield value\nRETURN {\n  nodes: apoc.coll.flatten(COLLECT(value.data.nodes)),\n  links: apoc.coll.flatten(COLLECT(value.data.links))\n}\n'),e.next=9,y(g);case 9:return _=e.sent,e.abrupt("return",Je(_));case 11:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();function ut(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function pt(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?ut(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):ut(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var ft=["Reaction","CompartmentalizedMetabolite"],ht=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.type,a=t.models,i=u()(a,2),o=i[0],s=i[1],l="\nMATCH (a:".concat(n,":").concat(o.model,")-[:V").concat(o.version,"]-()\nRETURN { ").concat(o.model,": COUNT(DISTINCT(a)) }\n"),c="\nMATCH (b:".concat(n,":").concat(s.model,")-[:V").concat(s.version,"]-()\nRETURN { ").concat(s.model,": COUNT(DISTINCT(b)) }\n"),d="\nMATCH (a:".concat(n,":").concat(o.model,")-[:V").concat(o.version,"]-(:ExternalDb)-[:V").concat(s.version,"]-(b:").concat(n,":").concat(s.model,")\nRETURN { ").concat(o.model,": COUNT(DISTINCT(a)), ").concat(s.model,": COUNT(DISTINCT(b)) }\n"),m=[y(l,!1),y(c,!1),y(d,!1)],e.abrupt("return",Promise.all(m));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),gt=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.type,a=t.models,i=u()(a,3),o=i[0],s=i[1],l=i[2],c="\nMATCH (a:".concat(n,":").concat(o.model,")-[:V").concat(o.version,"]-(e:ExternalDb)-[:V").concat(s.version,"]-(b:").concat(n,":").concat(s.model,")\nUSING JOIN on e\nMATCH (a)-[:V").concat(o.version,"]-(e)-[:V").concat(l.version,"]-(c:").concat(n,":").concat(l.model,")\nRETURN { ").concat(o.model,": COUNT(DISTINCT(a)), ").concat(s.model,": COUNT(DISTINCT(b)), ").concat(l.model,": COUNT(DISTINCT(c)) }\n"),d=[ht({type:n,models:[o,s]}),ht({type:n,models:[o,l]}),ht({type:n,models:[s,l]}),y(c,!1)],e.next=6,Promise.all(d);case 6:return m=e.sent,p=m.flat().filter((function(e,t,n){return n.findIndex((function(t){return JSON.stringify(t)===JSON.stringify(e)}))===t})),e.abrupt("return",p.sort((function(e,t){return Object.keys(e).length-Object.keys(t).length})));case 9:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),yt=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.type,a=t.models,i=u()(a,4),o=i[0],s=i[1],l=i[2],c=i[3],d="\nMATCH (a:".concat(n,":").concat(o.model,")-[:V").concat(o.version,"]-(e:ExternalDb)-[:V").concat(s.version,"]-(b:").concat(n,":").concat(s.model,")\nMATCH (a)-[:V").concat(o.version,"]-(e)-[:V").concat(l.version,"]-(c:").concat(n,":").concat(l.model,")\nMATCH (a)-[:V").concat(o.version,"]-(e)-[:V").concat(c.version,"]-(d:").concat(n,":").concat(c.model,")\nMATCH (b)-[:V").concat(s.version,"]-(e)-[:V").concat(l.version,"]-(c)\nMATCH (b)-[:V").concat(s.version,"]-(e)-[:V").concat(c.version,"]-(d)\nMATCH (c)-[:V").concat(l.version,"]-(e)-[:V").concat(c.version,"]-(d)\nRETURN { ").concat(o.model,": COUNT(DISTINCT(a)), ").concat(s.model,": COUNT(DISTINCT(b)), ").concat(l.model,": COUNT(DISTINCT(c)), ").concat(c.model,": COUNT(DISTINCT(d)) }\n"),m=[gt({type:n,models:[o,s,l]}),gt({type:n,models:[o,s,c]}),gt({type:n,models:[o,l,c]}),gt({type:n,models:[s,l,c]}),y(d,!1)],e.next=6,Promise.all(m);case 6:return p=e.sent,f=p.flat().filter((function(e,t,n){return n.findIndex((function(t){return JSON.stringify(t)===JSON.stringify(e)}))===t})),e.abrupt("return",f.sort((function(e,t){return Object.keys(e).length-Object.keys(t).length})));case 9:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),bt=function(){var e=s()(r().mark((function e(t){var n,a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.models,a=t.type,i="CompartmentalizedMetabolite"===a?3:4,!(n.length<2||n.length>i)){e.next=4;break}throw new Error("At least 2 and at most ".concat(i," models need to be provided for ").concat(a,"."));case 4:if(o={type:a,models:n},2!==n.length){e.next=9;break}return e.next=8,ht(o);case 8:case 12:return e.abrupt("return",e.sent);case 9:if(3!==n.length){e.next=13;break}return e.next=12,gt(o);case 13:return e.abrupt("return",yt(o));case 14:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),_t=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,u,p,f;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.model,a=t.models,i=t.type,o=[n].concat(C()(a)),s=o[0],l=o[1],c=o[2],d="\nMATCH (a:".concat(i,":").concat(s.model,")-[:V").concat(s.version,"]-()\nRETURN { own: COUNT(DISTINCT(a)) }\n"),c?(m="\nMATCH (a:".concat(i,":").concat(s.model,")-[:V").concat(s.version,"]-(e:ExternalDb)-[:V").concat(l.version,"]-(b:").concat(i,":").concat(l.model,")\nUSING JOIN on e\nMATCH (a)-[:V").concat(s.version,"]-(e)-[:V").concat(c.version,"]-(c:").concat(i,":").concat(c.model,")\nRETURN { common: COUNT(DISTINCT(a)) }\n"),u="\nMATCH (a:".concat(i,":").concat(s.model,")-[:V").concat(s.version,"]-()\nWHERE NOT (a)-[:V").concat(s.version,"]-(:ExternalDb)-[:V").concat(l.version,"]-(:").concat(i,":").concat(l.model,")\nAND NOT (a)-[:V").concat(s.version,"]-(:ExternalDb)-[:V").concat(c.version,"]-(:").concat(i,":").concat(c.model,")\nRETURN { unique: COLLECT(DISTINCT(a.id)) }\n")):(m="\nMATCH (a:".concat(i,":").concat(s.model,")-[:V").concat(s.version,"]-(:ExternalDb)-[:V").concat(l.version,"]-(b:").concat(i,":").concat(l.model,")\nRETURN { common: COUNT(DISTINCT(a)) }\n"),u="\nMATCH (a:".concat(i,":").concat(s.model,")-[:V").concat(s.version,"]-()\nWHERE NOT (a)-[:V").concat(s.version,"]-(:ExternalDb)-[:V").concat(l.version,"]-(:").concat(i,":").concat(l.model,")\nRETURN { unique: COLLECT(DISTINCT(a.id)) }\n")),p=[y(d,!1),y(m,!1),y(u,!1)],e.next=7,Promise.all(p);case 7:return f=e.sent,e.abrupt("return",f.reduce((function(e,t){return pt(pt({},e),t)}),{}));case 9:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),vt=function(){var e=s()(r().mark((function e(t){var n,a;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.models,e.next=3,Promise.all(ft.map(function(){var e=s()(r().mark((function e(t){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.t0=i(),e.t1={},e.t2=t,e.next=5,bt({models:n,type:t});case 5:return e.t3=e.sent,e.abrupt("return",(0,e.t0)(e.t1,e.t2,e.t3));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}()));case 3:return a=e.sent,e.abrupt("return",a.reduce((function(e,t){return pt(pt({},e),t)}),{}));case 5:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}(),wt=function(){var e=s()(r().mark((function e(t){var n,a,o,l;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.model,a=t.models,o=ft.map(function(){var e=s()(r().mark((function e(t){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.t0=i(),e.t1={},e.t2=t,e.next=5,_t({model:n,models:a,type:t});case 5:return e.t3=e.sent,e.abrupt("return",(0,e.t0)(e.t1,e.t2,e.t3));case 7:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}()),e.next=4,Promise.all(o);case 4:return l=e.sent,e.abrupt("return",{models:{model:n,models:a},details:l.reduce((function(e,t){return pt(pt({},e),t)}),{})});case 6:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();const Mt=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,u;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.dbName,a=t.externalId,i=t.referenceType,o="\nMATCH (db:ExternalDb {dbName: '".concat(n,"', externalId: '").concat(a,"'})-[v]-(c)\nRETURN { identifier: properties(db), components: COLLECT({ component: c, version: type(v) }) }\n"),"MetabolicAtlas"===n&&(o="\nMATCH (r:Reaction {id: '".concat(a,"'})-[v]-()\nRETURN { identifier: properties(r), components: COLLECT(DISTINCT({component: r, version: type(v)}))}\n\nUNION\n\nMATCH (r:CompartmentalizedMetabolite {id: '").concat(a,"'})-[v]-()\nRETURN { identifier: properties(r), components: COLLECT(DISTINCT({component: r, version: type(v)}))}\n\n")),e.prev=3,e.next=6,y(o);case 6:return s=e.sent,l=s.identifier,c=(c=s.components).map((function(e){var t=e.component,n=e.version,a=t.labels,i=t.properties,o=a.find((function(e){return e.indexOf("Gem")>-1})).replace("Gem","-GEM"),s=a.find((function(e){return-1===e.indexOf("Gem")}));return{id:i.id,model:o,componentType:s,version:n.replace("V","").replace(/_/g,".")}})).sort((function(e,t){return e.model.localeCompare(t.model)})),"MetabolicAtlas"===n&&(l.dbName=n,l.externalId=l.id,l.url="https://identifiers.org/metatlas:".concat(l.id)),e.abrupt("return",{components:c,identifier:l});case 14:if(e.prev=14,e.t0=e.catch(3),["compound","reaction","gene"].includes(i)){e.next=19;break}throw e.t0;case 19:if(d=Object.values(Q).find((function(e){return e.db===n}))){e.next=22;break}throw e.t0;case 22:if(m=d.dbPrefix,void 0!==(u=d["".concat(i,"Suffix")])){e.next=26;break}throw e.t0;case 26:return e.abrupt("return",{identifier:{dbName:n,externalId:a,url:"https://identifiers.org/".concat(m).concat(u,":").concat(a)},components:[]});case 27:case"end":return e.stop()}}),e,null,[[3,14]])})));return function(t){return e.apply(this,arguments)}}();var Tt=["(",")","'",'"',"`",":","[","]","/","\\",",","{","}"];function It(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function Ct(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?It(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):It(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var Nt=t().Router();function Gt(e){for(var t=0,n=Object.values(e);t<n.length;t++)xt(n[t])}function xt(e){if(Array.isArray(e))throw new Error("Arrays not expected as inputs");Tt.forEach((function(t){if(e&&e.includes(t))throw new Error("Malicious char detected")}))}var Ht=function(){var e=s()(r().mark((function e(t,n,a){var i,o,s,l,c,d,m,u,p,f,h,g;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return i=t.params.id,o=t.query,s=o.model,l=o.version,c=o.limit,d=o.full,m=o.searchTerm,u=o.componentTypes,p=o.isForAllCompartments,f=o.expanded,e.prev=2,Gt(h={id:i,version:l,model:s,limit:c,full:d,searchTerm:m,expanded:f}),u&&(h.componentTypes=JSON.parse(u)),p&&(h.isForAllCompartments=JSON.parse(p)),e.next=9,a(h);case 9:g=e.sent,n.json(g),e.next=20;break;case 13:if(e.prev=13,e.t0=e.catch(2),!e.t0.message.startsWith("Invalid id")){e.next=17;break}return e.abrupt("return",n.status(404).send(e.t0.message));case 17:if("404"!==e.t0.message){e.next=19;break}return e.abrupt("return",n.sendStatus(404));case 19:n.status(400).send(e.t0.message);case 20:case"end":return e.stop()}}),e,null,[[2,13]])})));return function(t,n,a){return e.apply(this,arguments)}}();Nt.get("/compartments/:id",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,M));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/compartments/:id/related-reactions",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,Pe));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/genes/:id",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,L));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/genes/:id/related-reactions",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,We));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/hpa/genes",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,B));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/hpa/gene/:id",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,k));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/metabolites/:id",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,oe));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/metabolites/:id/related-reactions",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,Fe));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/metabolites/:id/related-metabolites",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,Ue));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/reactions/:id",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,j));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/reactions/:id/related-reactions",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,De));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/subsystems/:id",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,q));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/subsystems/:id/related-reactions",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,je));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/random-components",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,qe));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/interaction-partners/:id",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,Xe));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/interaction-partners-expansion/:id",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:Ht(t,n,Ze);case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/maps/listing",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,ct));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/maps/search",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,dt));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/search",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.abrupt("return",Ht(t,n,Re));case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/3d-network",function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l,c,d;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.query,i=a.model,o=a.version,s=a.type,l=a.id,e.prev=1,c={model:i,version:o},s&&l&&(c=Ct(Ct({},c),{},{type:s,id:l})),Gt(c),e.next=7,mt(c);case 7:d=e.sent,n.json(d),e.next=14;break;case 11:e.prev=11,e.t0=e.catch(1),n.status(400).send(e.t0.message);case 14:case"end":return e.stop()}}),e,null,[[1,11]])})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/compare",function(){var e=s()(r().mark((function e(t,n){var a,i,o;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(a=t.query.models,e.prev=1,(i=JSON.parse(a)).forEach((function(e){return Gt(e)})),!(!a||i.length<2||i.length>4)){e.next=6;break}throw new Error("At least 2 and at most 4 models need to be provided.");case 6:return e.next=8,vt({models:i});case 8:o=e.sent,n.json(o),e.next=15;break;case 12:e.prev=12,e.t0=e.catch(1),n.status(400).send(e.t0.message);case 15:case"end":return e.stop()}}),e,null,[[1,12]])})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/comparison-details",function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l,c;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(a=t.query,i=a.model,o=a.models,s=JSON.parse(i),l=JSON.parse(o),e.prev=3,!(!o||l.length<1||l.length>2)){e.next=6;break}throw new Error("At least 1 and at most 2 models need to be provided.");case 6:return e.next=8,wt({model:s,models:l});case 8:c=e.sent,n.json(c),e.next=15;break;case 12:e.prev=12,e.t0=e.catch(3),n.status(400).send(e.t0.message);case 15:case"end":return e.stop()}}),e,null,[[3,12]])})));return function(t,n){return e.apply(this,arguments)}}()),Nt.get("/identifier/:dbName/:externalId",function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params,i=a.dbName,o=a.externalId,s=t.query.referenceType,e.prev=2,xt(i),xt(o),xt(s),e.next=8,Mt({dbName:i,externalId:o,referenceType:s});case 8:l=e.sent,n.json(l),e.next=17;break;case 12:if(e.prev=12,e.t0=e.catch(2),"404"!==e.t0.message){e.next=16;break}return e.abrupt("return",n.sendStatus(404));case 16:n.status(400).send(e.t0.message);case 17:case"end":return e.stop()}}),e,null,[[2,12]])})));return function(t,n){return e.apply(this,arguments)}}());const St=Nt,Et=JSON.parse('[{"id":"13","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associated protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Colorectal cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2708,"metabolite_count":2719,"gene_count":1750,"files":[{"path":"models/homo_sapiens/init_cancer_models/colorectal_cancer/colorectal_cancer_-__1.xml-4f845256f8aea27090a88a3caa3ea10d.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"100","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Soft tissue 1","cell_type":"Chondrocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3704,"metabolite_count":3230,"gene_count":928,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/soft_tissue_1/soft tissue 1 - chondrocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"101","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Soft tissue 1","cell_type":"Fibroblasts","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4504,"metabolite_count":3847,"gene_count":1427,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/soft_tissue_1/soft tissue 1 - fibroblasts.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"102","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Soft tissue 1","cell_type":"Peripheral nerve","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3385,"metabolite_count":3361,"gene_count":1135,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/soft_tissue_1/soft tissue 1 - peripheral nerve.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"103","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Soft tissue 2","cell_type":"Adipocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3541,"metabolite_count":3529,"gene_count":1214,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/soft_tissue_2/soft tissue 2 - adipocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"104","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Soft tissue 2","cell_type":"Chondrocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":2496,"metabolite_count":2741,"gene_count":635,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/soft_tissue_2/soft tissue 2 - chondrocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"1","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Breast","cell_type":"Myoepithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4772,"metabolite_count":4046,"gene_count":1776,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/breast/breast_-_myoepithelial_cells_2.xml-9a7735edecc2890e0c68120259a97a48.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"10","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Oral mucosa","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5054,"metabolite_count":4196,"gene_count":1857,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/oral_mucosa/oral mucosa - squamous epithelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"105","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Soft tissue 2","cell_type":"Fibroblasts","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3973,"metabolite_count":3794,"gene_count":1390,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/soft_tissue_2/soft tissue 2 - fibroblasts.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"106","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Soft tissue 2","cell_type":"Peripheral nerve","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3610,"metabolite_count":3552,"gene_count":1189,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/soft_tissue_2/soft tissue 2 - peripheral nerve.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"107","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Spleen","cell_type":"Cells in red pulp","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4407,"metabolite_count":4010,"gene_count":1609,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/spleen/spleen - cells in red pulp.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"108","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Spleen","cell_type":"Cells in white pulp","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4001,"metabolite_count":3765,"gene_count":1387,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/spleen/spleen - cells in white pulp.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"109","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Stomach 1","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5325,"metabolite_count":4493,"gene_count":2348,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/stomach_1/stomach 1 - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"11","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Liver and pancreas","tissue":"Liver","cell_type":"Hepatocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5141,"metabolite_count":4406,"gene_count":1878,"files":[{"path":"models/homo_sapiens/init_normal_models/liver_and_pancreas/liver/liver_-_hepatocytes_2.xml-8ef3fcb49599e1a11671144441721e38.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"110","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Stomach 2","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5366,"metabolite_count":4509,"gene_count":2331,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/stomach_2/stomach 2 - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"111","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Testis","cell_type":"Cells in seminiferous ducts","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5490,"metabolite_count":4360,"gene_count":2198,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/testis/testis - cells in seminiferous ducts.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"112","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Testis","cell_type":"Leydig cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5239,"metabolite_count":4457,"gene_count":2263,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/testis/testis - Leydig cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"113","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Thyroid gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5182,"metabolite_count":4383,"gene_count":2162,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/thyroid_gland/thyroid gland - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"114","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Tonsil","cell_type":"Germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4574,"metabolite_count":4036,"gene_count":1744,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/tonsil/tonsil - germinal center cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"115","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Tonsil","cell_type":"Non-germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4648,"metabolite_count":4087,"gene_count":1776,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/tonsil/tonsil - non-germinal center cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"116","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Tonsil","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5316,"metabolite_count":4337,"gene_count":1986,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/tonsil/tonsil - squamous epithelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"117","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Urinary bladder","cell_type":"Urothelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5218,"metabolite_count":4414,"gene_count":2192,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/urinary_bladder/urinary bladder - urothelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"118","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Uterus 1","cell_type":"Cells in endometrial stroma","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3964,"metabolite_count":3656,"gene_count":1301,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/uterus_1/uterus 1 - cells in endometrial stroma.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"119","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Uterus 1","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5179,"metabolite_count":4348,"gene_count":2003,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/uterus_1/uterus 1 - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"12","gemodelset":{"name":"Bacteria models","description":null,"reference":[]},"sample":{"organism":"Prochlorococcus marinus","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"MED4","description":"Prochlorococcus marinus is a marine cyanobacterium and is most abundant autotrophic species on Earth, contributing a majority of primary production in vast regions of the ocean gyres. This strain, MED4, is a representative of the high-light adapted \'ecotype\' eHLI. The model iJC568 was reconstructed using RAVEN by protein homology and by manual curation. The model consists of 568 genes, 794 reactions, 680 metabolites in 6 sub cellular locations and was validated by growth on minimal media, known metabolic capabilities, photosynthetic performance, and thermodynamic balance. The BOF includes 121 compounds parsed between \'crude\' biomass fractions with corresponding calculated GAM costs.","condition":"Evaluating metabolic capabilities","reaction_count":796,"metabolite_count":680,"gene_count":568,"files":[{"path":"models/bacteria_models/prochlorococcus_marinus/iJC568_Raven.zip","format":"SBML"}],"ref":[],"maintained":false,"year":null},{"id":"120","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Uterus 2","cell_type":"Cells in endometrial stroma","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3489,"metabolite_count":3458,"gene_count":1238,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/uterus_2/uterus 2 - cells in endometrial stroma.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"121","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Uterus 2","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4958,"metabolite_count":4289,"gene_count":2007,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/uterus_2/uterus 2 - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"122","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Vagina","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4577,"metabolite_count":4039,"gene_count":1765,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/vagina/vagina - squamous epithelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"123","gemodelset":{"name":"curated models","description":null,"reference":[]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":null,"cell_type":"Adipocyte","cell_line":null},"tag":"iAdipocytes1809","description":"We evaluated the presence/absence of proteins encoded by 14,077 genes in adipocytes obtained from different tissue samples using immunohistochemistry. By combining this with previously published adipocyte-specific proteome data, we identified proteins associated with 7,340 genes in human adipocytes. This information was used to reconstruct a comprehensive and functional genome-scale metabolic model of adipocyte metabolism. The resulting metabolic model, iAdipocytes1809, enables mechanistic insights into adipocyte metabolism on a genome-wide level, and can serve as a scaffold for integration of omics data to understand the genotype–phenotype relationship in obese subjects. By integrating human transcriptome and fluxome data, we found an increase in the metabolic activity around androsterone, ganglioside GM2 and degradation products of heparan sulfate and keratan sulfate, and a decrease in mitochondrial metabolic activities in obese subjects compared with lean subjects. Our study hereby shows a path to identify new therapeutic targets for treating obesity through combination of high throughput patient data and metabolic modeling.","condition":"Obesity","reaction_count":6100,"metabolite_count":4550,"gene_count":1809,"files":[{"path":"models/homo_sapiens/curated_models/adipocyte/iAdipocytes1809.xml-ea2fa20615a73d37571ab86eaffd2889.zip","format":"SBML"}],"ref":["23511207"],"maintained":false,"year":"2013"},{"id":"124","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Colon","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4482,"metabolite_count":4142,"gene_count":1390,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/colon/Colon-Colon-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/colon/Colon-Colon.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"125","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Diaphragm","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":3884,"metabolite_count":3580,"gene_count":1036,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/diaphragm/Diaphragm-Diaphragm-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/diaphragm/Diaphragm-Diaphragm.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"126","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Duodenum","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4170,"metabolite_count":3900,"gene_count":1250,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/duodenum/Duodenum-Duodenum-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/duodenum/Duodenum-Duodenum.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"127","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Embryonic tissue","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":3927,"metabolite_count":3705,"gene_count":1199,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/embryonic_tissue/Embryonic_tissue-Embryonic_tissue-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/embryonic_tissue/Embryonic_tissue-Embryonic_tissue.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"128","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Eye","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4075,"metabolite_count":3724,"gene_count":1213,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/eye/Eye-Eye-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/eye/Eye-Eye.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"129","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Heart","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":3978,"metabolite_count":3697,"gene_count":1141,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/heart/Heart-Heart-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/heart/Heart-Heart.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"130","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Ileum","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4525,"metabolite_count":4099,"gene_count":1354,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/ileum/Ileum-Ileum-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/ileum/Ileum-Ileum.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"131","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Jejunum","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4502,"metabolite_count":4070,"gene_count":1361,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/jejunum/Jejunum-Jejunum-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/jejunum/Jejunum-Jejunum.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"132","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Kidney cortex","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4509,"metabolite_count":4101,"gene_count":1397,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/kidney_cortex/Kidney_cortex-Kidney_cortex-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/kidney_cortex/Kidney_cortex-Kidney_cortex.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"133","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Vagina","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4274,"metabolite_count":3986,"gene_count":1709,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/vagina/vagina_-_squamous_epithelial_cells_2.xml-3daeb794125f2d3f9ffad3eedd1ea39d.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"186","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Brain medulla","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4141,"metabolite_count":3815,"gene_count":1278,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/brain_medulla/Brain_medulla-Brain_medulla-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/brain_medulla/Brain_medulla-Brain_medulla.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"187","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Brown fat","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4226,"metabolite_count":3889,"gene_count":1263,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/brown_fat/Brown_fat-Brown_fat-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/brown_fat/Brown_fat-Brown_fat.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"188","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Cerebellum","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4259,"metabolite_count":3788,"gene_count":1321,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/cerebellum/Cerebellum-Cerebellum-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/cerebellum/Cerebellum-Cerebellum.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"189","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Kidney medulla","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4185,"metabolite_count":3810,"gene_count":1237,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/kidney_medulla/Kidney_medulla-Kidney_medulla-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/kidney_medulla/Kidney_medulla-Kidney_medulla.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"19","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Endometrial cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2665,"metabolite_count":2676,"gene_count":1713,"files":[{"path":"models/homo_sapiens/init_cancer_models/endometrial_cancer/endometrial_cancer_-__1.xml-62b0b549e9fbbd7304d327e7ac57c9d7.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"190","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Liver","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4356,"metabolite_count":4020,"gene_count":1354,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/liver/Liver-Liver-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/liver/Liver-Liver.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"191","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Lung","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4033,"metabolite_count":3730,"gene_count":1207,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/lung/Lung-Lung-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/lung/Lung-Lung.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"192","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Midbrain","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":3893,"metabolite_count":3652,"gene_count":1142,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/midbrain/Midbrain-Midbrain-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/midbrain/Midbrain-Midbrain.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"239","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Stomach, upper","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5084,"metabolite_count":4479,"gene_count":2260,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/stomach_upper/stomach,_upper_-_glandular_cells_2.xml-cd4e3e5ff3a465640843f093d8a97ebb.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"193","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Muscle","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":3518,"metabolite_count":3527,"gene_count":971,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/muscle/Muscle-Muscle-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/muscle/Muscle-Muscle.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"194","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Olfactory bulb","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4277,"metabolite_count":3769,"gene_count":1239,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/olfactory_bulb/Olfactory_bulb-Olfactory_bulb-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/olfactory_bulb/Olfactory_bulb-Olfactory_bulb.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"195","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Ovary","cell_type":"Follicle cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2680,"metabolite_count":2821,"gene_count":1467,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/ovary/ovary_-_follicle_cells_1.xml-4130887959d0fea416bc81dbc02e4d10.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"196","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Salivary gland","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":3947,"metabolite_count":3727,"gene_count":1075,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/salivary_gland/Salivary_gland-Salivary_gland-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/salivary_gland/Salivary_gland-Salivary_gland.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"197","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Spleen","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4133,"metabolite_count":3820,"gene_count":1207,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/spleen/Spleen-Spleen-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/spleen/Spleen-Spleen.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"198","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Stomach","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":3756,"metabolite_count":3571,"gene_count":1030,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/stomach/Stomach-Stomach-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/stomach/Stomach-Stomach.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"134","gemodelset":{"name":"curated models","description":null,"reference":[]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":null,"cell_type":"Hepatocyte","cell_line":null},"tag":"iHepatocytes2322","description":"Several liver disorders result from perturbations in the metabolism of hepatocytes, and their underlying mechanisms can be outlined through the use of genome-scale metabolic models (GEMs). Here we reconstruct a consensus GEM for hepatocytes, which we call iHepatocytes2322, that extends previous models by including an extensive description of lipid metabolism. We build iHepatocytes2322 using Human Metabolic Reaction 2.0 database and proteomics data in Human Protein Atlas, which experimentally validates the incorporated reactions. The reconstruction process enables improved annotation of the proteomics data using the network centric view of iHepatocytes2322. We then use iHepatocytes2322 to analyse transcriptomics data obtained from patients with non-alcoholic fatty liver disease. We show that blood concentrations of chondroitin and heparan sulphates are suitable for diagnosing non-alcoholic steatohepatitis and for the staging of non-alcoholic fatty liver disease. Furthermore, we observe serine deficiency in patients with NASH and identify PSPH, SHMT1 and BCAT1 as potential therapeutic targets for the treatment of non-alcoholic steatohepatitis.","condition":"Non-alcoholic fatty liver","reaction_count":7930,"metabolite_count":5686,"gene_count":2322,"files":[{"path":"models/homo_sapiens/curated_models/hepatocyte/iHepatocytes2322.xml-952933555997556ab7a4c357da3cab9c.zip","format":"SBML"},{"path":"models/homo_sapiens/curated_models/hepatocyte/iHepatocytes2322_Cobra-8a10cd973f6e1c2ed2c58633e70a1b19.zip","format":"COBRA-friendly SBML"}],"ref":["24419221"],"maintained":false,"year":"2014"},{"id":"135","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Uterus, post-menopause","cell_type":"Cells in endometrial stroma","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":1815,"metabolite_count":2046,"gene_count":1283,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/uterus_post-menopause/uterus,_post-menopause_-_cells_in_endometrial_stroma_1.xml-9136ea18f344c87553be79efc2dd0cbc.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"136","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Uterus, post-menopause","cell_type":"Cells in endometrial stroma","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3431,"metabolite_count":3467,"gene_count":1172,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/uterus_post-menopause/uterus,_post-menopause_-_cells_in_endometrial_stroma_2.xml-aabad73b249b7d52970bed8c52278e3d.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"137","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Fallopian tube","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2435,"metabolite_count":2579,"gene_count":1716,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/fallopian_tube/fallopian_tube_-_glandular_cells_1.xml-843a5293caa4a10fd010d7b7277c66bf.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"138","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Fallopian tube","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4779,"metabolite_count":4335,"gene_count":2072,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/fallopian_tube/fallopian_tube_-_glandular_cells_2.xml-e590ff1cb84ac239800ae9b54e046ecf.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"139","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Ovary","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4015,"metabolite_count":3737,"gene_count":1119,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/ovary/Ovary-Ovary-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/ovary/Ovary-Ovary.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"14","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Breast cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2686,"metabolite_count":2727,"gene_count":1746,"files":[{"path":"models/homo_sapiens/init_cancer_models/breast_cancer/breast_cancer_-__1.xml-45bb75ad90431b92749015190078ff5c.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"140","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Pancreas","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":3927,"metabolite_count":3692,"gene_count":1134,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/pancreas/Pancreas-Pancreas-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/pancreas/Pancreas-Pancreas.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"141","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Liver and pancreas","tissue":"Liver","cell_type":"Bile duct cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2186,"metabolite_count":2389,"gene_count":1373,"files":[{"path":"models/homo_sapiens/init_normal_models/liver_and_pancreas/liver/liver_-_bile_duct_cells_1.xml-3adcda28ab3c8733d9a66df09e8c6c95.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"142","gemodelset":{"name":"curated models","description":null,"reference":[]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":null,"cell_type":"Myocyte","cell_line":null},"tag":"iMyocyte2419","description":"Skeletal myocytes are metabolically active and susceptible to insulin resistance, thus implicated in type 2 diabetes (T2D). This complex disease involves systemic metabolic changes and their elucidation at the systems level requires genome-wide data and biological networks. Genome-scale metabolic models (GEMs) provide a network-context to integrate high-throughput data. We generated myocyte-specific RNA-seq data and investigated their correlation with proteome data. These data were then used to reconstruct a comprehensive myocyte GEM. Next, we performed a meta-analysis of six studies comparing muscle transcription in T2D versus healthy subjects. Transcriptional changes were mapped on the myocyte GEM, revealing extensive transcriptional regulation in T2D, particularly around pyruvate oxidation, branched-chain amino acid catabolism, and tetrahydrofolate metabolism, connected through the down-regulated dihydrolipoamide dehydrogenase. Strikingly, the gene signature underlying this metabolic regulation successfully classifies the disease state of individual samples, suggesting that regulation of these pathways is a ubiquitous feature of myocytes in response to T2D.","condition":"Diabetes Mellitus","reaction_count":5590,"metabolite_count":4448,"gene_count":2419,"files":[{"path":"models/homo_sapiens/curated_models/myocyte/iMyocyte2419-4934cef2be56b22214947b6e2d1b110f.zip","format":"SBML"}],"ref":["25937284"],"maintained":false,"year":"2015"},{"id":"143","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Adipose","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5844,"metabolite_count":4549,"gene_count":2454,"files":[{"path":"models/homo_sapiens/tissue-specific_models/adipose/adipose.xml-c0e57e25277efe805b3d325013c90598.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"144","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Adrenal gland","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5711,"metabolite_count":4478,"gene_count":2523,"files":[{"path":"models/homo_sapiens/tissue-specific_models/adrenal_gland/adrenal_gland.xml-8315ec6f09ee96977f05faff3558b01e.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"145","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Appendix","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5814,"metabolite_count":4570,"gene_count":2534,"files":[{"path":"models/homo_sapiens/tissue-specific_models/appendix/appendix.xml-5727509a8a9bad08026a6b2da59d1de9.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"146","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Bone marrow","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5490,"metabolite_count":4313,"gene_count":2240,"files":[{"path":"models/homo_sapiens/tissue-specific_models/bone_marrow/bone_marrow.xml-c046a526de5aa1e780f4d3191c87b322.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"147","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Brain","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5788,"metabolite_count":4493,"gene_count":2572,"files":[{"path":"models/homo_sapiens/tissue-specific_models/brain/brain.xml-0cc137f6eff1e44d85b0813c28a482f7.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"148","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Colon","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5854,"metabolite_count":4597,"gene_count":2580,"files":[{"path":"models/homo_sapiens/tissue-specific_models/colon/colon.xml-8dcf0b7214cfc03b45f8a8f4a6efb480.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"149","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Duodenum","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5899,"metabolite_count":4599,"gene_count":2592,"files":[{"path":"models/homo_sapiens/tissue-specific_models/duodenum/duodenum.xml-9cb571c0ccfa41f782d79b4156249854.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"15","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Breast cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":4300,"metabolite_count":3956,"gene_count":1771,"files":[{"path":"models/homo_sapiens/init_cancer_models/breast_cancer/breast_cancer_-__3.xml-4297bbc8636e6b22818d180b44898384.zip","format":"SBML"}],"ref":["24550497"],"maintained":false,"year":"2014"},{"id":"150","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Endometrium","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5699,"metabolite_count":4477,"gene_count":2517,"files":[{"path":"models/homo_sapiens/tissue-specific_models/endometrium/endometrium.xml-07234f6a9cd39fed6a1c7e439a80b135.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"151","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Esophagus","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5803,"metabolite_count":4569,"gene_count":2545,"files":[{"path":"models/homo_sapiens/tissue-specific_models/esophagus/esophagus.xml-2dc86f3c3ede96c9797f3f400b276790.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"152","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Fallopian tube","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5755,"metabolite_count":4508,"gene_count":2557,"files":[{"path":"models/homo_sapiens/tissue-specific_models/fallopian_tube/fallopian_tube.xml-2f2f1cf488998bf07c02b9343ce8cada.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"153","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Gallbladder","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5871,"metabolite_count":4601,"gene_count":2605,"files":[{"path":"models/homo_sapiens/tissue-specific_models/gallbladder/gallbladder.xml-857e9e31333d09adf905973e0630c356.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"154","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Heart muscle","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5634,"metabolite_count":4441,"gene_count":2417,"files":[{"path":"models/homo_sapiens/tissue-specific_models/heart_muscle/heart_muscle.xml-5f306365f4d549f6dd3ec95546d2c886.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"155","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Kidney","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5979,"metabolite_count":4611,"gene_count":2623,"files":[{"path":"models/homo_sapiens/tissue-specific_models/kidney/kidney.xml-0dbb98a46d71ccab4b133167f944205d.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"156","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5993,"metabolite_count":4576,"gene_count":2456,"files":[{"path":"models/homo_sapiens/tissue-specific_models/liver/liver.xml-f43f741a755562271c6439db4f0ae11c.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"157","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lung","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5874,"metabolite_count":4565,"gene_count":2560,"files":[{"path":"models/homo_sapiens/tissue-specific_models/lung/lung.xml-8a243e6985ee580fe2ab66051da6f72a.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"158","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lymph node","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5760,"metabolite_count":4484,"gene_count":2415,"files":[{"path":"models/homo_sapiens/tissue-specific_models/lymph_node/lymph_node.xml-dba0b945435f11073c339be7fd7aa437.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"159","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Ovary","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5569,"metabolite_count":4378,"gene_count":2419,"files":[{"path":"models/homo_sapiens/tissue-specific_models/ovary/ovary.xml-f5d533ddea4083ac19a4b8266d11abe0.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"16","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Prostate cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2790,"metabolite_count":2803,"gene_count":1560,"files":[{"path":"models/homo_sapiens/init_cancer_models/prostate_cancer/prostate_cancer_-__1.xml-8fcbc2c008f7e0d7e81330a5ab0e5e74.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"160","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Pancreas","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5678,"metabolite_count":4447,"gene_count":2368,"files":[{"path":"models/homo_sapiens/tissue-specific_models/pancreas/pancreas.xml-ac87f24291d6613ff237f355c3fcf621.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"161","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Placenta","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5685,"metabolite_count":4447,"gene_count":2471,"files":[{"path":"models/homo_sapiens/tissue-specific_models/placenta/placenta.xml-bd8389492cb9e9739523b08d540dbe9d.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"162","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Prostate","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5745,"metabolite_count":4513,"gene_count":2577,"files":[{"path":"models/homo_sapiens/tissue-specific_models/prostate/prostate.xml-29a109d1533e267791c7a7465088578d.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"163","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Rectum","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5837,"metabolite_count":4580,"gene_count":2561,"files":[{"path":"models/homo_sapiens/tissue-specific_models/rectum/rectum.xml-5058e115a609da531494dc637b13d773.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"164","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Salivary","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5778,"metabolite_count":4482,"gene_count":2466,"files":[{"path":"models/homo_sapiens/tissue-specific_models/salivary/salivary.xml-3ad9ab3094deb5d931a7b6a3109cf35b.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"165","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skeletal muscle","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5539,"metabolite_count":4335,"gene_count":2204,"files":[{"path":"models/homo_sapiens/tissue-specific_models/skeletal_muscle/skeletal_muscle.xml-6891aeac88e4ba4e5b909815ccc4c998.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"166","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skin","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5791,"metabolite_count":4545,"gene_count":2465,"files":[{"path":"models/homo_sapiens/tissue-specific_models/skin/skin.xml-1a79f09b808b826f7670a7016b452ec9.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"167","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Small intestine","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5936,"metabolite_count":4626,"gene_count":2615,"files":[{"path":"models/homo_sapiens/tissue-specific_models/small_intestine/small_intestine.xml-aa0b5049f799976ca6e986a6734940b8.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"168","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Smooth muscle","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5694,"metabolite_count":4479,"gene_count":2510,"files":[{"path":"models/homo_sapiens/tissue-specific_models/smooth_muscle/smooth_muscle.xml-507f060376652f1d1b8f2b7491f81d04.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"169","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Spleen","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5726,"metabolite_count":4533,"gene_count":2463,"files":[{"path":"models/homo_sapiens/tissue-specific_models/spleen/spleen.xml-c6ade0d47b3f3d695130ab72c7b813ab.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"17","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Ovarian cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2848,"metabolite_count":2819,"gene_count":1620,"files":[{"path":"models/homo_sapiens/init_cancer_models/ovarian_cancer/ovarian_cancer_-__1.xml-90495c21874af7c5a44e41e6fd575c3a.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"170","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Stomach","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5827,"metabolite_count":4569,"gene_count":2585,"files":[{"path":"models/homo_sapiens/tissue-specific_models/stomach/stomach.xml-b1e10a774f0e8d03e52f4993758cc1c6.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"171","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Testis","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5888,"metabolite_count":4535,"gene_count":2690,"files":[{"path":"models/homo_sapiens/tissue-specific_models/testis/testis.xml-e7500d92f6e63eb9771f05827c5339e3.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"172","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Thyroid gland","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5724,"metabolite_count":4501,"gene_count":2466,"files":[{"path":"models/homo_sapiens/tissue-specific_models/thyroid_gland/thyroid_gland.xml-59fed143e1c27cebd2ad79b2787e833d.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"173","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Tonsil","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5782,"metabolite_count":4544,"gene_count":2490,"files":[{"path":"models/homo_sapiens/tissue-specific_models/tonsil/tonsil.xml-8fb7922f53397158f9fc60dda4c7c850.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"174","gemodelset":{"name":"tissue-specific models","description":"Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray–based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.","reference":["25613900"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Urinary","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Druggable and cancer proteome","reaction_count":5764,"metabolite_count":4547,"gene_count":2548,"files":[{"path":"models/homo_sapiens/tissue-specific_models/urinary/urinary.xml-e06895ce505a96459ad794a452922607.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"175","gemodelset":{"name":"Human gut microbiota models","description":"Using metagenomics and discovering over 3 million genes, constituting a \'parts list\' of this ecosystem, key species were identified. Here we reconstructed genome-scale models (GEMs) for three key species, iBth801 (Bacteroides thetaiotamicron), iEre400 (Eubacterium rectale), iMsi385 (Methanobrevibacter smithii), iBif452 (Bifidobacterium adolescentis L2-32) and iFap484 (Faecalibacterium prausnitzii A2-165​) as relevant representatives of the three main phyla in the human gut (Bacteroidetes, Firmicutes and Euryarchaeota). These GEMs allow for improved annotation of metagenomics data, but will more importantly enable simulation of the gut microbiome ecosystem. Hereby it will be possible to gain information about the functional role of different species in this complex ecosystem as well as evaluate the role of different species in silico. Our modeling will hereby become an important bridge between clinical data and rodent experiments, as the modeling will allow for experimental design.","reference":["23982459"]},"sample":{"organism":"Bacteroides thetaiotaomicron","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iBth1201","description":null,"condition":"Microbial community and host interactions","reaction_count":1190,"metabolite_count":1135,"gene_count":801,"files":[{"path":"models/bacteria_models/human_gut_microbiota_models/iBth1201v1.00.xml-07d63f71ac2bca79770515474a9375bc.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2013"},{"id":"176","gemodelset":{"name":"Human gut microbiota models","description":"Using metagenomics and discovering over 3 million genes, constituting a \'parts list\' of this ecosystem, key species were identified. Here we reconstructed genome-scale models (GEMs) for three key species, iBth801 (Bacteroides thetaiotamicron), iEre400 (Eubacterium rectale), iMsi385 (Methanobrevibacter smithii), iBif452 (Bifidobacterium adolescentis L2-32) and iFap484 (Faecalibacterium prausnitzii A2-165​) as relevant representatives of the three main phyla in the human gut (Bacteroidetes, Firmicutes and Euryarchaeota). These GEMs allow for improved annotation of metagenomics data, but will more importantly enable simulation of the gut microbiome ecosystem. Hereby it will be possible to gain information about the functional role of different species in this complex ecosystem as well as evaluate the role of different species in silico. Our modeling will hereby become an important bridge between clinical data and rodent experiments, as the modeling will allow for experimental design.","reference":["23982459"]},"sample":{"organism":"Eubacterium rectale","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iEre400","description":null,"condition":"Microbial community and host interactions","reaction_count":467,"metabolite_count":423,"gene_count":400,"files":[{"path":"models/bacteria_models/human_gut_microbiota_models/iEre400v1.00.xml-cf5cc84f757c9bcb521123572ea778e7.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2013"},{"id":"177","gemodelset":{"name":"Human gut microbiota models","description":"Using metagenomics and discovering over 3 million genes, constituting a \'parts list\' of this ecosystem, key species were identified. Here we reconstructed genome-scale models (GEMs) for three key species, iBth801 (Bacteroides thetaiotamicron), iEre400 (Eubacterium rectale), iMsi385 (Methanobrevibacter smithii), iBif452 (Bifidobacterium adolescentis L2-32) and iFap484 (Faecalibacterium prausnitzii A2-165​) as relevant representatives of the three main phyla in the human gut (Bacteroidetes, Firmicutes and Euryarchaeota). These GEMs allow for improved annotation of metagenomics data, but will more importantly enable simulation of the gut microbiome ecosystem. Hereby it will be possible to gain information about the functional role of different species in this complex ecosystem as well as evaluate the role of different species in silico. Our modeling will hereby become an important bridge between clinical data and rodent experiments, as the modeling will allow for experimental design.","reference":["23982459"]},"sample":{"organism":"Methanobrevibacter smithii","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iMsi385","description":null,"condition":"Microbial community and host interactions","reaction_count":525,"metabolite_count":564,"gene_count":385,"files":[{"path":"models/bacteria_models/human_gut_microbiota_models/iMsi385.xml-d5928d5bb74a1f00ea9db778da9ff2cf.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2013"},{"id":"178","gemodelset":{"name":"Human gut microbiota models","description":"Using metagenomics and discovering over 3 million genes, constituting a \'parts list\' of this ecosystem, key species were identified. Here we reconstructed genome-scale models (GEMs) for three key species, iBth801 (Bacteroides thetaiotamicron), iEre400 (Eubacterium rectale), iMsi385 (Methanobrevibacter smithii), iBif452 (Bifidobacterium adolescentis L2-32) and iFap484 (Faecalibacterium prausnitzii A2-165​) as relevant representatives of the three main phyla in the human gut (Bacteroidetes, Firmicutes and Euryarchaeota). These GEMs allow for improved annotation of metagenomics data, but will more importantly enable simulation of the gut microbiome ecosystem. Hereby it will be possible to gain information about the functional role of different species in this complex ecosystem as well as evaluate the role of different species in silico. Our modeling will hereby become an important bridge between clinical data and rodent experiments, as the modeling will allow for experimental design.","reference":["23982459"]},"sample":{"organism":"Faecalibacterium prausnitzii a2-165","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iFab484","description":null,"condition":"Microbial community and host interactions","reaction_count":713,"metabolite_count":776,"gene_count":0,"files":[{"path":"models/bacteria_models/human_gut_microbiota_models/iFap484.V01.00.xml-4610c35b7ac5ee14e857b8be68497970.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2013"},{"id":"179","gemodelset":{"name":"Human gut microbiota models","description":"Using metagenomics and discovering over 3 million genes, constituting a \'parts list\' of this ecosystem, key species were identified. Here we reconstructed genome-scale models (GEMs) for three key species, iBth801 (Bacteroides thetaiotamicron), iEre400 (Eubacterium rectale), iMsi385 (Methanobrevibacter smithii), iBif452 (Bifidobacterium adolescentis L2-32) and iFap484 (Faecalibacterium prausnitzii A2-165​) as relevant representatives of the three main phyla in the human gut (Bacteroidetes, Firmicutes and Euryarchaeota). These GEMs allow for improved annotation of metagenomics data, but will more importantly enable simulation of the gut microbiome ecosystem. Hereby it will be possible to gain information about the functional role of different species in this complex ecosystem as well as evaluate the role of different species in silico. Our modeling will hereby become an important bridge between clinical data and rodent experiments, as the modeling will allow for experimental design.","reference":["23982459"]},"sample":{"organism":"Bifidobacterium adolescentis l2-32","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iBif452","description":null,"condition":"Microbial community and host interactions","reaction_count":699,"metabolite_count":768,"gene_count":0,"files":[{"path":"models/bacteria_models/human_gut_microbiota_models/iBif452.V01.00.xml-2d8dd6c1f7b8e7b6209569f0e3e51f10.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2013"},{"id":"18","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cervical cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2836,"metabolite_count":2832,"gene_count":1611,"files":[{"path":"models/homo_sapiens/init_cancer_models/cervical_cancer/cervical_cancer_-__1.xml-dc6fee37822d9fb4fa9f54ff6b70773d.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"180","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Adipose","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":6098,"metabolite_count":4473,"gene_count":2006,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/adipose/iMouseAdipose-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/adipose/iMouseAdipose.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"181","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Colon","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":5859,"metabolite_count":4569,"gene_count":2280,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/colon/iMouseColon-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/colon/iMouseColon.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"182","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Liver","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":7879,"metabolite_count":5655,"gene_count":2237,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/liver/iMouseLiver-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/liver/iMouseLiver.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"183","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Small intestine","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":6235,"metabolite_count":4698,"gene_count":2337,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/small_intestine/iMouseSmallintestine-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/small_intestine/iMouseSmallintestine.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"184","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Adrenal gland","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4386,"metabolite_count":3981,"gene_count":1354,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/adrenal_gland/Adrenal_gland-Adrenal_gland-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/adrenal_gland/Adrenal_gland-Adrenal_gland.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"185","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Brain cortex","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4079,"metabolite_count":3723,"gene_count":1166,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/brain_cortex/Brain_cortex-Brain_cortex-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/brain_cortex/Brain_cortex-Brain_cortex.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"199","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Thymus","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4269,"metabolite_count":3932,"gene_count":1227,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/thymus/Thymus-Thymus-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/thymus/Thymus-Thymus.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"2","gemodelset":{"name":"Bacteria models","description":null,"reference":[]},"sample":{"organism":"Streptomyces coelicolor","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"A3(2)","description":"Streptomyces are filamentous soil bacteria that produce more than half of the known microbial antibiotics. We present the first genome-scale metabolic model of a representative of this group - Streptomyces coelicolor A3(2). The metabolism reconstruction was based on annotated genes, physiological and biochemical information. The stoichiometric model includes 819 biochemical conversions and 152 transport reactions, accounting for a total of 971 reactions. Of the reactions in the network, 700 are unique, while the rest are iso-reactions. The network comprises 500 metabolites. A total of 711 open reading frames (ORFs) were included in the model, which corresponds to 13% of the ORFs with assigned function in the S. coelicolor A3(2) genome. In a comparative analysis with the Streptomyces avermitilis genome, we showed that the metabolic genes are highly conserved between these species and therefore the model is suitable for use with other Streptomycetes. Flux balance analysis was applied for studies of the reconstructed metabolic network and to assess its metabolic capabilities for growth and polyketides production. The model predictions of wild-type and mutants\' growth on different carbon and nitrogen sources agreed with the experimental data in most cases. We estimated the impact of each reaction knockout on the growth of the in silico strain on 62 carbon sources and two nitrogen sources, thereby identifying the core of the essential reactions. We also illustrated how reconstruction of a metabolic network at the genome level can be used to fill gaps in genome annotation.","condition":"Evaluating metabolic capabilities","reaction_count":971,"metabolite_count":500,"gene_count":711,"files":[{"path":"models/bacteria_models/streptomyces_coelicolor/SCOELICOLOR_rxnGene.zip","format":"Reaction-Gene"}],"ref":["15930493"],"maintained":false,"year":"2005"},{"id":"20","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Head and neck cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2931,"metabolite_count":2859,"gene_count":1628,"files":[{"path":"models/homo_sapiens/init_cancer_models/head_and_neck_cancer/head_and_neck_cancer_-__1.xml-1ae89cf4aac7bb5f5343f14cf0538398.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"200","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"Uterus","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":3998,"metabolite_count":3749,"gene_count":1129,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/uterus/Uterus-Uterus-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/uterus/Uterus-Uterus.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"201","gemodelset":{"name":"mouse metabolism models","description":"The gut microbiota has been proposed as an environmental factor that promotes the progression of metabolic diseases. Here we investigated the global metabolic differences between the duodenum, jejunum, ileum, colon, liver and two fat tissues obtained from conventionally raised (CONV-R) and germ-free (GF) mice using gene expression data and tissue-specific GEnome-scale metabolic Models (GEMs). We created a generic Mouse Metabolic Reaction GEM (MMR), reconstructed 28 mouse tissue-specific GEMs based on proteomics data and manually curated GEMs for small intestine, colon, liver and fat tissues. We used these functional models for revealing the global metabolic differences between CONV-R and GF mice. Based on gene expression data, we found that the gut microbiota effect the host amino acid (AA) metabolism, which lead to modifications in glutathione metabolism. To validate our predictions we measured the level of AAs and N-acetylated AAs in the hepatic portal vein of CONV-R and GF mice. Finally, we simulated the metabolic differences between the small intestine of the CONV-R and GF accounting for the content of the diet and relative gene expression differences using the here presented method, Relative Metabolic Differences (RMetD). Our analyses revealed that the gut microbiota influences host amino acid and glutathione metabolism in mice.","reference":["26475342"]},"sample":{"organism":"Mus musculus","organ_system":null,"tissue":"White fat","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Global metabolic differences between CONV-R and GF mice","reaction_count":4307,"metabolite_count":3925,"gene_count":1237,"files":[{"path":"models/mus_musculus/mouse_metabolism_models/white_fat/White_fat-White_fat-Cobra.zip","format":"COBRA-friendly SBML"},{"path":"models/mus_musculus/mouse_metabolism_models/white_fat/White_fat-White_fat.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2015"},{"id":"203","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"Ashbya gossypii","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"Ashbya gossypii","description":"Ashybya gossypii, also known as Eremothecium gossypii is a filamentous fungi used widey for the production of Ribofavin and Vitamin B2. The model of this organism iRL766 is constructed using RAVEN, COBRA toolbox with the functionalities homology based and Hidden Markov Model (HMM) analysis and finaly curated manually. The model is validated by biomass growth, riboflavin production and substrate utilization predictions. The metabolic network constructed comprises 1596 unique metabolic reactions, 1349 metabolites in four sub-cellular compartments; extracellular, cytoplasmic, mitochondrial, and peroxisomal.","condition":"Physiological validations","reaction_count":1637,"metabolite_count":1348,"gene_count":766,"files":[{"path":"models/fungi_models/ashbya_gossypii/iRL766.zip","format":"SBML"}],"ref":["24374726"],"maintained":false,"year":"2014"},{"id":"204","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"Aspergillus niger","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"Aspergillus niger","description":"The release of the genome sequences of two strains of Aspergillus niger has allowed systems-level investigations of this important microbial cell factory. To this end, tools for doing data integration of multi-ome data are necessary, and especially interesting in the context of metabolism. On the basis of an A. niger bibliome survey, we present the largest model reconstruction of a metabolic network reported for a fungal species. The reconstructed gapless metabolic network is based on the reportings of 371 articles and comprises 1190 biochemically unique reactions and 871 ORFs. Inclusion of isoenzymes increases the total number of reactions to 2240.","condition":"Physiological validations","reaction_count":1380,"metabolite_count":0,"gene_count":0,"files":[{"path":"models/fungi_models/aspergillus_niger/ANIGER_RXNS.zip","format":"Reaction-Gene"}],"ref":["18364712"],"maintained":false,"year":"2008"},{"id":"205","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"Aspergillus oryzae","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"Aspergillus oryzae","description":"Since ancient times the filamentous fungus Aspergillus oryzae has been used in the fermentation industry for the production of fermented sauces and the production of industrial enzymes. Recently, the genome sequence of A. oryzae with 12,074 annotated genes was released but the number of hypothetical proteins accounted for more than 50% of the annotated genes. Considering the industrial importance of this fungus, it is therefore valuable to improve the annotation and further integrate genomic information with biochemical and physiological information available for this microorganism and other related fungi. Here we proposed the gene prediction by construction of an A. oryzae Expressed Sequence Tag (EST) library, sequencing and assembly. We enhanced the function assignment by our developed annotation strategy. The resulting better annotation was used to reconstruct the metabolic network leading to a genome scale metabolic model of A. oryzae.","condition":"Gene predictions","reaction_count":2360,"metabolite_count":0,"gene_count":0,"files":[{"path":"models/fungi_models/aspergillus_oryzae/AORYZAE.zip","format":"Excel"}],"ref":["18500999"],"maintained":false,"year":"2008"},{"id":"206","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"Aspergillus nidulans","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"Aspergillus nidulans","description":"Aspergillus nidulans is a member of a diverse group of filamentous fungi, sharing many of the properties of its close relatives with significance in the fields of medicine, agriculture and industry. Furthermore, A. nidulans has been a classical model organism for studies of development biology and gene regulation, and thus it has become one of the best-characterized filamentous fungi. It was the first Aspergillus species to have its genome sequenced, and automated gene prediction tools predicted 9,451 open reading frames (ORFs) in the genome, of which less than 10% were assigned a function.","condition":"Physiological validations and predictions","reaction_count":1324,"metabolite_count":822,"gene_count":717,"files":[{"path":"models/fungi_models/aspergillus_nidulans/ANIDULANS.zip","format":"SBML"}],"ref":["18405346"],"maintained":false,"year":"2008"},{"id":"207","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"P. chrysogenum","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"P. chrysogenum","description":"Using the RAVEN Toolbox the metabolic network of P. chrysogenum metabolic network was reconstructed. The metabolic network comprises 1471 unique metabolic reactions in four sub-cellular compartments; extracellular, cytosolic, mitochondrial, and peroxisomal. 1006 ORFs are associated to the reactions, 89 of which participate in one of 35 protein complexes. In parallel to the automatic reconstruction, an extensive literature study was performed. In total 440 cited articles provide experimental evidence for the majority of the reactions. All model components were extensively annotated to adhere to the MIRIAM standard for biological models. The model was validated with respect to 76 important metabolic functions (the supplementary file simulations.xls is an input file to checkTasks, which was used to performed the validation).","condition":"Physiological validations and predictions","reaction_count":1471,"metabolite_count":0,"gene_count":0,"files":[{"path":"models/fungi_models/p._chrysogenum/iAL1006v1.00.zip","format":"Excel"}],"ref":["23555215"],"maintained":false,"year":"2013"},{"id":"232","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Oral mucosa","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2625,"metabolite_count":2667,"gene_count":1421,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/oral_mucosa/oral_mucosa_-_squamous_epithelial_cells_1.xml-92b5a9a31beb8a09c9b9d93d7fa8e2a3.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"233","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Oral mucosa","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4842,"metabolite_count":4137,"gene_count":1805,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/oral_mucosa/oral_mucosa_-_squamous_epithelial_cells_2.xml-156060e1d082466fe58ea9e965a9dbdc.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"234","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Salivary gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2761,"metabolite_count":2807,"gene_count":1502,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/salivary_gland/salivary_gland_-_glandular_cells_1.xml-3e2bc1295d35d0dea2315ac7fb7a55c4.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"235","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Salivary gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4839,"metabolite_count":4284,"gene_count":1988,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/salivary_gland/salivary_gland_-_glandular_cells_2.xml-15a67cc18e241274b5e0c674f9663798.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"208","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"Pichia pastoris","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"Pichia pastoris","description":"Here we reported the third GEM of P. pastoris named iLC915. The model was used for evaluating the yeasts metabolic capabilities and the simulations were compared with experimental data. Therefore are able to be used for gaining more understanding on metabolic functions of the yeast. Some of these functions were compared against S. cerevisiae. The iLC915 model of P. pastoris was tasted against many tasks including the utilization of carbon sources relevant for protein production, as well as carbon sources that promote peroxysomes formation e.g. methanol, oleic acid and alanine. The effect of the utilization of such carbon sources on cell physiology was also evaluated to predict energetic requirements under each. Furthermore, the effect of the recombinant protein production under several oxygen uptake rates on cells physiology was evaluated and compare with experimental results. Some of former studies were not done with the previous GEM of P. pastoris, hence the current model accomplish most interrogative cycles","condition":"Evaluating metabolic capabilities","reaction_count":1448,"metabolite_count":1361,"gene_count":915,"files":[{"path":"models/fungi_models/pichia_pastoris/iLC915.zip","format":"SBML"}],"ref":["22472172"],"maintained":false,"year":"2012"},{"id":"280","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Male reproductive system","tissue":"Prostate","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4885,"metabolite_count":4285,"gene_count":1981,"files":[{"path":"models/homo_sapiens/init_normal_models/male_reproductive_system/prostate/prostate_-_glandular_cells_2.xml-d92f5e32ba8dbce2bcd7a4e16d2a0afa.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"281","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Male reproductive system","tissue":"Seminal vesicle","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2504,"metabolite_count":2635,"gene_count":1748,"files":[{"path":"models/homo_sapiens/init_normal_models/male_reproductive_system/seminal_vesicle/seminal_vesicle_-_glandular_cells_1.xml-f631f1fcc522b212ca176fc11869617f.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"282","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Male reproductive system","tissue":"Seminal vesicle","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4911,"metabolite_count":4380,"gene_count":2031,"files":[{"path":"models/homo_sapiens/init_normal_models/male_reproductive_system/seminal_vesicle/seminal_vesicle_-_glandular_cells_2.xml-5d891fc0f50718979ad1e5f31d2fe40b.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"209","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"Pichia stipitis","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"Pichia stipitis","description":"Here we reported the first genome-scale metabolic model of P. stipitis named iSS884. The model was used for evaluating the yeast metabolic capabilities and the simulations were compared with experimental data. Therefore is able to be used for gaining more understanding on metabolic functions of this yeast. Some of these functions were compared against S. cerevisiae. The iSS884 model can be used to asses into the P. stipitis ability to produce ethanol from wood sugars promoting low oxygen uptake rates. Such can be useful, for example, to generate process strategies to convert such cheap raw material into ethanol, as was shown.","condition":"Ethanol production-evaluating metabolic capabilities","reaction_count":1376,"metabolite_count":1300,"gene_count":884,"files":[{"path":"models/fungi_models/pichia_stipitis/iSS884 v1.0.zip","format":"SBML"}],"ref":["22472172"],"maintained":false,"year":"2012"},{"id":"21","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Thyroid cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":3037,"metabolite_count":2963,"gene_count":1710,"files":[{"path":"models/homo_sapiens/init_cancer_models/thyroid_cancer/thyroid_cancer_-__1.xml-53aab429a585e535a27f39c241f21916.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"210","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"Saccharomyces cerevisiae","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iTO977","description":"We present iTO977, a comprehensive genome-scale metabolic model that contains more reactions, metabolites and genes than previous models. The model was constructed based on two earlier reconstructions, namely iIN800 and the consensus network, and then improved and expanded using gap-filling methods and by introducing new reactions and pathways based on studies of the literature and databases.","condition":"Physiological validations and predictions","reaction_count":1612,"metabolite_count":1353,"gene_count":977,"files":[{"path":"models/fungi_models/saccharomyces_cerevisiae/iTO977_v1.00_raven.zip","format":"SBML"}],"ref":["23631471"],"maintained":false,"year":"2013"},{"id":"211","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Ovary","cell_type":"Follicle cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3440,"metabolite_count":3520,"gene_count":1096,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/ovary/ovary_-_follicle_cells_2.xml-b213734e25eaa9355834de5efc9c6e94.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"212","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Ovary","cell_type":"Ovarian stroma cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":1995,"metabolite_count":2219,"gene_count":1285,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/ovary/ovary_-_ovarian_stroma_cells_1.xml-16ffa0152880cd9442f819e2a3cead66.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"213","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Ovary","cell_type":"Ovarian stroma cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3531,"metabolite_count":3506,"gene_count":1213,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/ovary/ovary_-_ovarian_stroma_cells_2.xml-37033f4a8bd7afab5c0f42b5cd845ace.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"214","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Liver and pancreas","tissue":"Liver","cell_type":"Hepatocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2976,"metabolite_count":2945,"gene_count":1705,"files":[{"path":"models/homo_sapiens/init_normal_models/liver_and_pancreas/liver/liver_-_hepatocytes_1.xml-deea6f138567478baaeb47074a81fb5f.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"215","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"Saccharomyces cerevisiae","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iIN800","description":"One of the most complex parts of metabolism, lipid metabolism, was poorly described in the original genome scale metabolic models. Therefore the model iIN800 was constructed by expanding the iFF708 model. It included 143 new reactions associated with 65 ORFs involved in lipid metabolism. It also had an improved biomass equation, involving details in the formation of fatty acids and lipids for carbon- and nitrogen limited growth conditions. Predictions of both growth capability and large scale in silico single gene deletions by iIN800 were consistent with experimental data. In addition, 13C-labeling experiments validated the new biomass equations and calculated intracellular fluxes. It has been demonstrated that IN800 can be used as a scaffold to reveal the regulatory importance (using transcriptome data-sets) of lipid metabolism precursors and intermediates that would have been missed in previous models.","condition":"Physiological validations and predictions","reaction_count":1706,"metabolite_count":985,"gene_count":0,"files":[{"path":"models/fungi_models/saccharomyces_cerevisiae/iIN800_sbml.zip","format":"SBML"}],"ref":["18687109"],"maintained":false,"year":"2008"},{"id":"216","gemodelset":{"name":"Fungi models","description":null,"reference":[]},"sample":{"organism":"Saccharomyces cerevisiae","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iFF708","description":"The first genome-scale model was named iFF708. The model consists of 708 metabolic genes associated with 1175 reactions and 733 metabolites. Three cellular compartments, namely cytosol, mitochondria and the extra cellular space were included. The physiological validations and predictions by the model using flux balance analysis (FBA) were shown to have good agreements with many experimental data-sets. Furthermore, large-scale in silico gene deletion analysis of the model showed high accuracy in predicting gene essentiality when comparing to in vivo knock-out phenotypes.","condition":"Physiological validations and predictions","reaction_count":1175,"metabolite_count":733,"gene_count":708,"files":[{"path":"models/fungi_models/saccharomyces_cerevisiae/iFF708.zip","format":"Excel"}],"ref":["12566402"],"maintained":false,"year":"2003"},{"id":"217","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Embryonic kidney","cell_type":null,"cell_line":"HEK293"},"tag":"HEK293","description":null,"condition":"Cancer","reaction_count":5274,"metabolite_count":4220,"gene_count":2287,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/embryonic_kidney/HEK293.xml-de675a2413c948d115a17c8588126876.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"218","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Brain","cell_type":null,"cell_line":"U-251MG"},"tag":"U-251MG","description":null,"condition":"Cancer","reaction_count":5320,"metabolite_count":4209,"gene_count":2194,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/brain/U-251MG.xml-91080a939b86d903928cb7e2c321c2ff.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"219","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Bone","cell_type":null,"cell_line":"U-2OS"},"tag":"U-2OS","description":null,"condition":"Cancer","reaction_count":5452,"metabolite_count":4333,"gene_count":2328,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/bone/U-2OS.xml-5466d9840e6293657d63d4e7153368fc.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"22","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lung cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2709,"metabolite_count":2777,"gene_count":1490,"files":[{"path":"models/homo_sapiens/init_cancer_models/lung_cancer/lung_cancer_-__1.xml-d9de393d1b78fab97cd886f9b66f255d.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"220","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Urinary bladder","cell_type":null,"cell_line":"RT-4"},"tag":"RT-4","description":null,"condition":"Cancer","reaction_count":5584,"metabolite_count":4407,"gene_count":2251,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/urinary_bladder/RT-4.xml-c2cfdb5d7c34cdb1dbfab39a8191b471.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"221","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Prostate","cell_type":null,"cell_line":"PC-3"},"tag":"PC-3","description":null,"condition":"Cancer","reaction_count":5431,"metabolite_count":4263,"gene_count":2297,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/prostate/PC-3.xml-ed8604a79470ef0b998924fb00176934.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"222","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Breast","cell_type":null,"cell_line":"MCF-7"},"tag":"MCF-7","description":null,"condition":"Cancer","reaction_count":5402,"metabolite_count":4318,"gene_count":2243,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/breast/MCF-7.xml-6aab27142337081109339dbeff1f5099.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"223","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver","cell_type":null,"cell_line":"Hep-G2"},"tag":"Hep-G2","description":null,"condition":"Cancer","reaction_count":5646,"metabolite_count":4432,"gene_count":2273,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/liver/Hep-G2.xml-5a91f1955e8a9c8a5c5d2ff4a1737c21.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"224","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cervix","cell_type":null,"cell_line":"HeLa"},"tag":"HeLa","description":null,"condition":"Cancer","reaction_count":5570,"metabolite_count":4397,"gene_count":2267,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/cervix/HeLa.xml-5ccb90e3608520c2b0bc98f08f7c88eb.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"225","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Colon","cell_type":null,"cell_line":"CACO-2"},"tag":"CACO-2","description":null,"condition":"Cancer","reaction_count":5472,"metabolite_count":4326,"gene_count":2271,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/colon/CACO-2.xml-779bd0b52971349bfdd2cdb12706c9a3.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"226","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lung","cell_type":null,"cell_line":"A-549"},"tag":"A-549","description":null,"condition":"Cancer","reaction_count":5574,"metabolite_count":4382,"gene_count":2318,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/lung/A-549.xml-2cef596d55231d6b5934a7e5f9a95c7a.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"227","gemodelset":{"name":"cell-line specific models","description":"Human cancer cell lines are used as important model systems to study molecular mechanisms associated with tumor growth, hereunder how genomic and biological heterogeneity found in primary tumors affect cellular phenotypes. We reconstructed Genome scale metabolic models (GEMs) for eleven cell lines based on RNA-Seq data and validated the functionality of these models with data from metabolite profiling. We used cell line-specific GEMs to analyze the differences in the metabolism of cancer cell lines, and to explore the heterogeneous expression of the metabolic subsystems. Furthermore, we predicted 85 antimetabolites that can inhibit growth of, or even kill, any of the cell lines, while at the same time not being toxic for 83 different healthy human cell types. 60 of these antimetabolites were found to inhibit growth in all cell lines. Finally, we experimentally validated one of the predicted antimetabolites using two cell lines with different phenotypic origins, and found that it is effective in inhibiting the growth of these cell lines. Using immunohistochemistry, we also showed high or moderate expression levels of proteins targeted by the validated antimetabolite. Identified anti-growth factors for inhibition of cell growth may provide leads for the development of efficient cancer treatment strategies.","reference":["25640694"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skin","cell_type":null,"cell_line":"A-431"},"tag":"A-431","description":null,"condition":"Cancer","reaction_count":5297,"metabolite_count":4245,"gene_count":2193,"files":[{"path":"models/homo_sapiens/cell-line_specific_models/skin/A-431.xml-8e6fa764b6a394ff055f1ae3a6505a63.zip","format":"SBML"}],"ref":["25640694"],"maintained":false,"year":"2014"},{"id":"23","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lung cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":3811,"metabolite_count":3652,"gene_count":1472,"files":[{"path":"models/homo_sapiens/init_cancer_models/lung_cancer/lung_cancer_-__3.xml-bf4255bf7e786330b1e66238ae41b966.zip","format":"SBML"}],"ref":["24550497"],"maintained":false,"year":"2014"},{"id":"236","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Esophagus","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2621,"metabolite_count":2618,"gene_count":1463,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/esophagus/esophagus_-_squamous_epithelial_cells_1.xml-131899a31ed3bf53a284462b55b25abd.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"237","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Esophagus","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4954,"metabolite_count":4156,"gene_count":1902,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/esophagus/esophagus_-_squamous_epithelial_cells_2.xml-90410e8e3e0e37f669aa21f8f763c937.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"238","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Stomach, upper","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2600,"metabolite_count":2702,"gene_count":1815,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/stomach_upper/stomach,_upper_-_glandular_cells_1.xml-87e8e277ac7822323e8a5d05ddeef88a.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"24","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skin cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2669,"metabolite_count":2689,"gene_count":1386,"files":[{"path":"models/homo_sapiens/init_cancer_models/skin_cancer/skin_cancer_-__1.xml-c08b74ac78a9d281abe65e7f5fc26250.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"240","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Stomach, lower","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2915,"metabolite_count":2928,"gene_count":1608,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/stomach_lower/stomach,_lower_-_glandular_cells_1.xml-b8ea93ece2c8bfe4eac6115cc985e96e.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"241","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Stomach, lower","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5116,"metabolite_count":4490,"gene_count":2259,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/stomach_lower/stomach,_lower_-_glandular_cells_2.xml-6bf120819a35e832849f629bca2e66e9.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"242","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Duodenum","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2605,"metabolite_count":2687,"gene_count":1777,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/duodenum/duodenum_-_glandular_cells_1.xml-74af5d83af0b92950383520e4d1d7791.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"243","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Duodenum","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5535,"metabolite_count":4496,"gene_count":2265,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/duodenum/duodenum_-_glandular_cells_2.xml-69511c71031b5eaa14bd663d581ad631.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"244","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Small intestine","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2968,"metabolite_count":2933,"gene_count":1666,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/small_intestine/small_intestine_-_glandular_cells_1.xml-3f7425b0cd63e3fe4cf91c6e89d3d654.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"245","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Small intestine","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5484,"metabolite_count":4484,"gene_count":2254,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/small_intestine/small_intestine_-_glandular_cells_2.xml-0af97f59aa3d9db50a5722d49d6106af.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"246","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Appendix","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2856,"metabolite_count":2862,"gene_count":1706,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/appendix/appendix_-_glandular_cells_1.xml-1059e327d3460edb81b57a89818a7071.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"247","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Appendix","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5429,"metabolite_count":4474,"gene_count":2135,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/appendix/appendix_-_glandular_cells_2.xml-dd575327e2d11ef8c88f48f2f7451d03.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"248","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Appendix","cell_type":"Lymphoid tissue","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2149,"metabolite_count":2336,"gene_count":1505,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/appendix/appendix_-_lymphoid_tissue_1.xml-4f4d62e114a83d800d389b9630cbcf42.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"249","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Appendix","cell_type":"Lymphoid tissue","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4201,"metabolite_count":3964,"gene_count":1513,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/appendix/appendix_-_lymphoid_tissue_2.xml-0ef4933352bbec4e873ba610dc669598.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"25","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Testis cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2650,"metabolite_count":2665,"gene_count":1483,"files":[{"path":"models/homo_sapiens/init_cancer_models/testis_cancer/testis_cancer_-__1.xml-28a02a1f968b921bcba1f1aa2f915f0b.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"250","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Colon","cell_type":"Endothelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4817,"metabolite_count":4148,"gene_count":1758,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/colon/colon_-_endothelial_cells_2.xml-a992add60319cc031dc9c944329b3703.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"251","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Colon","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3060,"metabolite_count":2936,"gene_count":1733,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/colon/colon_-_glandular_cells_1.xml-5d6f8550b917741b559a8f095b0d3448.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"252","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Colon","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5454,"metabolite_count":4467,"gene_count":2227,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/colon/colon_-_glandular_cells_2.xml-e1db5e64a9a454ddbadbe44c353c14c6.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"253","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Colon","cell_type":"Peripheral nerve-ganglion","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4772,"metabolite_count":4098,"gene_count":1655,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/colon/colon_-_peripheral_nerve-ganglion_2.xml-a219ba681e06702274e0c207ae7b28a5.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"254","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Rectum","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2515,"metabolite_count":2641,"gene_count":1789,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/rectum/rectum_-_glandular_cells_1.xml-1dd59e09013beab2de66b8aaeec672c1.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"255","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Digestive tract","tissue":"Rectum","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5426,"metabolite_count":4463,"gene_count":2220,"files":[{"path":"models/homo_sapiens/init_normal_models/digestive_tract/rectum/rectum_-_glandular_cells_2.xml-84a4bfad9611a057bc49778c28eec12d.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"256","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Breast","cell_type":"Adipocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3195,"metabolite_count":3256,"gene_count":1094,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/breast/breast_-_adipocytes_2.xml-7e88b7a161e998a00e55dd21277d5c4c.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"257","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Breast","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2718,"metabolite_count":2781,"gene_count":1532,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/breast/breast_-_glandular_cells_1.xml-a897b1910f2df8b0fc433810f99a8370.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"258","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Breast","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5162,"metabolite_count":4237,"gene_count":1971,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/breast/breast_-_glandular_cells_2.xml-3bd988ffb1af78520ad0dfa73f82c65c.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"259","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Cervix, uterine","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2192,"metabolite_count":2343,"gene_count":1557,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/cervix_uterine/cervix,_uterine_-_squamous_epithelial_cells_1.xml-dfe16fb8294a06d2f5ae80efbd24a3b7.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"26","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Urothelial cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2855,"metabolite_count":2871,"gene_count":1532,"files":[{"path":"models/homo_sapiens/init_cancer_models/urothelial_cancer/urothelial_cancer_-__1.xml-6356f39451d84bb0a57c54587c9fc5a5.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"260","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Cervix, uterine","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4259,"metabolite_count":3978,"gene_count":1607,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/cervix_uterine/cervix,_uterine_-_squamous_epithelial_cells_2.xml-6fe71f530182da86b711ff14623b8a54.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"261","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Cervix, uterine","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2270,"metabolite_count":2430,"gene_count":1518,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/cervix_uterine/cervix,_uterine_-_glandular_cells_1.xml-5c2af7ea65959ea641312fb5aa44d885.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"262","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Cervix, uterine","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4415,"metabolite_count":4107,"gene_count":1719,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/cervix_uterine/cervix,_uterine_-_glandular_cells_2.xml-2c94e6c1c8d562cbb7d7c90842dd910b.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"263","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Uterus, pre-menopause","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2460,"metabolite_count":2629,"gene_count":1698,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/uterus_pre-menopause/uterus,_pre-menopause_-_glandular_cells_1.xml-6a9c36da42efa2cd458508a063cbfffa.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"264","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Uterus, pre-menopause","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4876,"metabolite_count":4302,"gene_count":1949,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/uterus_pre-menopause/uterus,_pre-menopause_-_glandular_cells_2.xml-ff7a4b79e1b1790700dc3a3f514cf98c.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"265","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Uterus, pre-menopause","cell_type":"Cells in endometrial stroma","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":1688,"metabolite_count":1904,"gene_count":1159,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/uterus_pre-menopause/uterus,_pre-menopause_-_cells_in_endometrial_stroma_1.xml-0732391e1a2b61b129584fb1590b7d72.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"266","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Uterus, pre-menopause","cell_type":"Cells in endometrial stroma","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3657,"metabolite_count":3604,"gene_count":1303,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/uterus_pre-menopause/uterus,_pre-menopause_-_cells_in_endometrial_stroma_2.xml-8ea10925e6122860428b72b958e44be2.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"267","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Uterus, post-menopause","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2437,"metabolite_count":2559,"gene_count":1715,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/uterus_post-menopause/uterus,_post-menopause_-_glandular_cells_1.xml-a1421d3886808e9b1b85fb1cfba7f6fe.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"268","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Liver and pancreas","tissue":"Liver","cell_type":"Bile duct cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3985,"metabolite_count":3841,"gene_count":1411,"files":[{"path":"models/homo_sapiens/init_normal_models/liver_and_pancreas/liver/liver_-_bile_duct_cells_2.xml-7e476983ea025a0b047445e3f23ddebd.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"269","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Liver and pancreas","tissue":"Gallbladder","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2638,"metabolite_count":2712,"gene_count":1860,"files":[{"path":"models/homo_sapiens/init_normal_models/liver_and_pancreas/gallbladder/gallbladder_-_glandular_cells_1.xml-4c84d6a420d5cc0f75cb0ae63c602ef4.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"27","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Urothelial cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":4210,"metabolite_count":3864,"gene_count":1648,"files":[{"path":"models/homo_sapiens/init_cancer_models/urothelial_cancer/urothelial_cancer_-__3.xml-b391ad03203294d3a420001c843d2fce.zip","format":"SBML"}],"ref":["24550497"],"maintained":false,"year":"2014"},{"id":"270","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Liver and pancreas","tissue":"Gallbladder","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5400,"metabolite_count":4462,"gene_count":2271,"files":[{"path":"models/homo_sapiens/init_normal_models/liver_and_pancreas/gallbladder/gallbladder_-_glandular_cells_2.xml-575308b9bf02138810b1a70a94796ea8.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"271","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Liver and pancreas","tissue":"Pancreas","cell_type":"Exocrine glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2818,"metabolite_count":2826,"gene_count":1683,"files":[{"path":"models/homo_sapiens/init_normal_models/liver_and_pancreas/pancreas/pancreas_-_exocrine_glandular_cells_1.xml-413a1a791d515ae20b2a4d048acd7535.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"272","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Liver and pancreas","tissue":"Pancreas","cell_type":"Exocrine glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4909,"metabolite_count":4387,"gene_count":2101,"files":[{"path":"models/homo_sapiens/init_normal_models/liver_and_pancreas/pancreas/pancreas_-_exocrine_glandular_cells_2.xml-4ee6641960046893f71658cd5f755057.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"273","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Liver and pancreas","tissue":"Pancreas","cell_type":"Islets of langerhans","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2606,"metabolite_count":2709,"gene_count":1650,"files":[{"path":"models/homo_sapiens/init_normal_models/liver_and_pancreas/pancreas/pancreas_-_islets_of_langerhans_1.xml-c6cf7c70edf6deebf083d2f85b14a930.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"274","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Male reproductive system","tissue":"Testis","cell_type":"Cells in seminiferus ducts","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2784,"metabolite_count":2866,"gene_count":1572,"files":[{"path":"models/homo_sapiens/init_normal_models/male_reproductive_system/testis/testis_-_cells_in_seminiferus_ducts_1.xml-66b8c8e4feef4a8f917c7d39cfb6175e.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"275","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Male reproductive system","tissue":"Testis","cell_type":"Cells in seminiferus ducts","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5314,"metabolite_count":4342,"gene_count":2121,"files":[{"path":"models/homo_sapiens/init_normal_models/male_reproductive_system/testis/testis_-_cells_in_seminiferus_ducts_2.xml-ecba8905d0f93f73e0f52ba511fcc147.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"276","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Male reproductive system","tissue":"Testis","cell_type":"Leydig cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2919,"metabolite_count":2908,"gene_count":1766,"files":[{"path":"models/homo_sapiens/init_normal_models/male_reproductive_system/testis/testis_-_leydig_cells_1.xml-94d5e2e6f9012c01dcb487adb74b8492.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"277","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Male reproductive system","tissue":"Epididymis","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2815,"metabolite_count":2856,"gene_count":1685,"files":[{"path":"models/homo_sapiens/init_normal_models/male_reproductive_system/epididymis/epididymis_-_glandular_cells_1.xml-d9bb149d35017cda6ec0c6f18ee26b6c.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"278","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Male reproductive system","tissue":"Epididymis","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5213,"metabolite_count":4343,"gene_count":2139,"files":[{"path":"models/homo_sapiens/init_normal_models/male_reproductive_system/epididymis/epididymis_-_glandular_cells_2.xml-2ab7fed1b020ea044d244f381edea1db.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"279","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Male reproductive system","tissue":"Prostate","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2887,"metabolite_count":2917,"gene_count":1556,"files":[{"path":"models/homo_sapiens/init_normal_models/male_reproductive_system/prostate/prostate_-_glandular_cells_1.xml-714213671783b2790d1488fcf8bd2b7c.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"28","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Renal cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2835,"metabolite_count":2858,"gene_count":1448,"files":[{"path":"models/homo_sapiens/init_cancer_models/renal_cancer/renal_cancer_-__1.xml-d88ed7698da774e1e540b09ded69305b.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"283","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Urinary tract","tissue":"Kidney","cell_type":"Cells in glomeruli","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2398,"metabolite_count":2522,"gene_count":1276,"files":[{"path":"models/homo_sapiens/init_normal_models/urinary_tract/kidney/kidney_-_cells_in_glomeruli_1.xml-82b7acd407c69d2f5f5a43ed272ceab3.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"284","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Urinary tract","tissue":"Kidney","cell_type":"Cells in glomeruli","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3970,"metabolite_count":3853,"gene_count":1482,"files":[{"path":"models/homo_sapiens/init_normal_models/urinary_tract/kidney/kidney_-_cells_in_glomeruli_2.xml-2ca7b62eb8344b2c14995d56498612ed.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"285","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Urinary tract","tissue":"Kidney","cell_type":"Cells in tubules","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2635,"metabolite_count":2758,"gene_count":1841,"files":[{"path":"models/homo_sapiens/init_normal_models/urinary_tract/kidney/kidney_-_cells_in_tubules_1.xml-c4aba33c3b6c4b009b10fbb9d28ab861.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"286","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Urinary tract","tissue":"Kidney","cell_type":"Cells in tubules","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5144,"metabolite_count":4460,"gene_count":2308,"files":[{"path":"models/homo_sapiens/init_normal_models/urinary_tract/kidney/kidney_-_cells_in_tubules_2.xml-827ad587e391dc5d200a4c9f6a3eac96.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"287","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Urinary tract","tissue":"Urinary bladder","cell_type":"Urothelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2491,"metabolite_count":2619,"gene_count":1786,"files":[{"path":"models/homo_sapiens/init_normal_models/urinary_tract/urinary_bladder/urinary_bladder_-_urothelial_cells_1.xml-b42414295b0572746fd3acbebd5e7461.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"288","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Urinary tract","tissue":"Urinary bladder","cell_type":"Urothelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4980,"metabolite_count":4396,"gene_count":2121,"files":[{"path":"models/homo_sapiens/init_normal_models/urinary_tract/urinary_bladder/urinary_bladder_-_urothelial_cells_2.xml-e0c23ff0301e0014a1eeafe438a96b28.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"289","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Placenta","tissue":"Placenta","cell_type":"Trophoblastic cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2488,"metabolite_count":2627,"gene_count":1716,"files":[{"path":"models/homo_sapiens/init_normal_models/placenta/placenta/placenta_-_trophoblastic_cells_1.xml-d20f53a84ffedceeeff9eb4ba2a230e6.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"29","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Renal cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":3914,"metabolite_count":3684,"gene_count":1411,"files":[{"path":"models/homo_sapiens/init_cancer_models/renal_cancer/renal_cancer_-__3.xml-bb999a8ccff9ebd687264b98d821622b.zip","format":"SBML"}],"ref":["24550497"],"maintained":false,"year":"2014"},{"id":"290","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Placenta","tissue":"Placenta","cell_type":"Trophoblastic cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5305,"metabolite_count":4388,"gene_count":2027,"files":[{"path":"models/homo_sapiens/init_normal_models/placenta/placenta/placenta_-_trophoblastic_cells_2.xml-4dacdf75aa95083d68b841079e91e2bd.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"291","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Placenta","tissue":"Placenta","cell_type":"Decidual cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2903,"metabolite_count":2958,"gene_count":1665,"files":[{"path":"models/homo_sapiens/init_normal_models/placenta/placenta/placenta_-_decidual_cells_1.xml-6438d3a2997100d40a4adb5997162e0f.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"292","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Placenta","tissue":"Placenta","cell_type":"Decidual cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4313,"metabolite_count":3983,"gene_count":1525,"files":[{"path":"models/homo_sapiens/init_normal_models/placenta/placenta/placenta_-_decidual_cells_2.xml-eeecd3ed76e20ccb08185d7658e8c9e7.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"293","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 1","cell_type":"Adipocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4095,"metabolite_count":3600,"gene_count":1242,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_1/soft_tissue_1_-_adipocytes_2.xml-70e79b61143ff433897f47a66d6dbccd.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"294","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 1","cell_type":"Chondrocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3391,"metabolite_count":3107,"gene_count":895,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_1/soft_tissue_1_-_chondrocytes_2.xml-9e3c9518ef7f2233640c75d0e167f783.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"295","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 1","cell_type":"Fibroblasts","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4282,"metabolite_count":3826,"gene_count":1405,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_1/soft_tissue_1_-_fibroblasts_2.xml-e67980d15bb973f011db626bb91e040a.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"296","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 1","cell_type":"Peripheral nerve","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2428,"metabolite_count":2539,"gene_count":1698,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_1/soft_tissue_1_-_peripheral_nerve_1.xml-3456dd123e3e5fe5182a56793f27927f.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"297","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 1","cell_type":"Peripheral nerve","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3306,"metabolite_count":3261,"gene_count":1080,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_1/soft_tissue_1_-_peripheral_nerve_2.xml-1cb617b5e9a0aaf8e33cf70e031a6b05.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"298","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 2","cell_type":"Adipocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3550,"metabolite_count":3532,"gene_count":1191,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_2/soft_tissue_2_-_adipocytes_2.xml-e6acab0c972efb3a66c91baafac1e183.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"299","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 2","cell_type":"Chondrocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2490,"metabolite_count":2714,"gene_count":598,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_2/soft_tissue_2_-_chondrocytes_2.xml-10d4f8e0634d48c7ded5382b114de2a6.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"3","gemodelset":{"name":"Bacteria models","description":null,"reference":[]},"sample":{"organism":"Lactococccus lactis","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":null,"description":"Genome-scale flux models are useful tools to represent and analyze microbial metabolism. In this work we reconstructed the metabolic network of the lactic acid bacteria Lactococcus lactis and developed a genome-scale flux model able to simulate and analyze network capabilities and whole-cell function under aerobic and anaerobic continuous cultures. Flux balance analysis (FBA) and minimization of metabolic adjustment (MOMA) were used as modeling frameworks","condition":"Aerobic and anaerobic continuous cultures","reaction_count":864,"metabolite_count":0,"gene_count":0,"files":[{"path":"models/bacteria_models/lactococccus_lactis/LLACTIS_RXN.zip","format":"Reaction-Gene"}],"ref":["15982422"],"maintained":false,"year":"2005"},{"id":"30","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Stomach cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2784,"metabolite_count":2791,"gene_count":1511,"files":[{"path":"models/homo_sapiens/init_cancer_models/stomach_cancer/stomach_cancer_-__1.xml-b1c920e7204bbfadfccc2ed7a665a011.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"300","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 2","cell_type":"Fibroblasts","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3891,"metabolite_count":3793,"gene_count":1367,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_2/soft_tissue_2_-_fibroblasts_2.xml-474630ecea678c31ef07e176640ed722.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"301","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 2","cell_type":"Peripheral nerve","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2507,"metabolite_count":2614,"gene_count":1722,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_2/soft_tissue_2_-_peripheral_nerve_1.xml-1df26c91d0217d376bbcfd89d9014f38.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"302","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Soft tissue 2","cell_type":"Peripheral nerve","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3566,"metabolite_count":3504,"gene_count":1126,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/soft_tissue_2/soft_tissue_2_-_peripheral_nerve_2.xml-cd9d58d5f0434d8a6c145f5b36e95d1d.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"303","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Skin","cell_type":"Fibroblasts","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4415,"metabolite_count":3907,"gene_count":1526,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/skin/skin_-_fibroblasts_2.xml-d0b72c386110fe9121b6155cd6d18e16.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"304","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Skin","cell_type":"Keratinocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4477,"metabolite_count":4116,"gene_count":1800,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/skin/skin_-_keratinocytes_2.xml-074b7687220bf8f12d133204b17af1ff.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"305","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Skin","cell_type":"Langerhans","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2612,"metabolite_count":2723,"gene_count":1432,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/skin/skin_-_langerhans_1.xml-9e7cdead7787adf870eef1dc6730e237.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"306","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Skin","cell_type":"Melanocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2282,"metabolite_count":2557,"gene_count":1754,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/skin/skin_-_melanocytes_1.xml-d0cc29e827bcce48a883869740f22659.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"307","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Skin","cell_type":"Melanocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4635,"metabolite_count":4025,"gene_count":1654,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/skin/skin_-_melanocytes_2.xml-73ee1e9074ab1248e5f762922644c3ac.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"308","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Vulva-anal skin","cell_type":"Epidermal cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2709,"metabolite_count":2778,"gene_count":1523,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/vulva-anal_skin/vulva-anal_skin_-_epidermal_cells_1.xml-5ada79837b4102e9e0af54c5dc1d549f.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"309","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Vulva-anal skin","cell_type":"Epidermal cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4742,"metabolite_count":4245,"gene_count":1979,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/vulva-anal_skin/vulva-anal_skin_-_epidermal_cells_2.xml-adece76ce9ff96d75b76bd5374ab26e4.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"31","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Pancreatic cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2927,"metabolite_count":2879,"gene_count":1613,"files":[{"path":"models/homo_sapiens/init_cancer_models/pancreatic_cancer/pancreatic_cancer_-__1.xml-329cf3df530788d080d4c4f4d4036aed.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"310","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Skeletal muscle","cell_type":"Myocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2709,"metabolite_count":2733,"gene_count":1537,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/skeletal_muscle/skeletal_muscle_-_myocytes_1.xml-1a7f4624a12004396ee779fca3b5af4d.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"311","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Skeletal muscle","cell_type":"Myocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4838,"metabolite_count":4091,"gene_count":1826,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/skeletal_muscle/skeletal_muscle_-_myocytes_2.xml-15d529b56576fdc02da63bf9bdce44ed.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"312","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Smooth muscle","cell_type":"Smooth muscle cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2499,"metabolite_count":2539,"gene_count":1489,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/smooth_muscle/smooth_muscle_-_smooth_muscle_cells_1.xml-bf505339fd5fe45c7c9ef39e5a8e1c71.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"313","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Skin and soft tissues","tissue":"Smooth muscle","cell_type":"Smooth muscle cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4485,"metabolite_count":3856,"gene_count":1586,"files":[{"path":"models/homo_sapiens/init_normal_models/skin_and_soft_tissues/smooth_muscle/smooth_muscle_-_smooth_muscle_cells_2.xml-6631b841e6cf3cc1b68d3bf5f6fd14fd.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"314","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebral cortex","cell_type":"Neuronal cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2679,"metabolite_count":2764,"gene_count":1548,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebral_cortex/cerebral_cortex_-_neuronal_cells_1.xml-338ac869a40a8aaad44040c39e61d4a0.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"315","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebral cortex","cell_type":"Neuronal cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4586,"metabolite_count":4238,"gene_count":1993,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebral_cortex/cerebral_cortex_-_neuronal_cells_2.xml-28977af7ef9468defdafe2dc9e4ec703.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"316","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebral cortex","cell_type":"Endothelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4548,"metabolite_count":3934,"gene_count":1522,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebral_cortex/cerebral_cortex_-_endothelial_cells_2.xml-ef9dda6bf98ba6137d7d9798e5e33e12.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"317","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebral cortex","cell_type":"Glial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2088,"metabolite_count":2269,"gene_count":1452,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebral_cortex/cerebral_cortex_-_glial_cells_1.xml-ff5dba4b5fae054a0690eab3435d1f1f.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"318","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebral cortex","cell_type":"Glial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4164,"metabolite_count":3959,"gene_count":1559,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebral_cortex/cerebral_cortex_-_glial_cells_2.xml-17f6f8703ff7d31560b0ceb8f256c97b.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"319","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebral cortex","cell_type":"Neuropil","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4481,"metabolite_count":4013,"gene_count":1738,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebral_cortex/cerebral_cortex_-_neuropil_2.xml-1820a663f16138f2eb9acf5345be017c.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"32","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":2794,"metabolite_count":2813,"gene_count":1788,"files":[{"path":"models/homo_sapiens/init_cancer_models/liver_cancer/liver_cancer_-__1.xml-9c79f66f78e3a7247a0391649aeb0902.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"320","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Hippocampus","cell_type":"Neuronal cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2405,"metabolite_count":2557,"gene_count":1650,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/hippocampus/hippocampus_-_neuronal_cells_1.xml-28c2390c94d598f5c080cff6db6c373f.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"321","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Hippocampus","cell_type":"Neuronal cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4514,"metabolite_count":4173,"gene_count":1896,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/hippocampus/hippocampus_-_neuronal_cells_2.xml-d1a3deb0c035997844979bc2ef4ddf8a.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"322","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Hippocampus","cell_type":"Glial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2167,"metabolite_count":2314,"gene_count":1441,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/hippocampus/hippocampus_-_glial_cells_1.xml-ab3998405ffcc57d4b0e37c8d0e33ad1.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"323","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Hippocampus","cell_type":"Glial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3989,"metabolite_count":3881,"gene_count":1402,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/hippocampus/hippocampus_-_glial_cells_2.xml-9be6bd58f4aa1ba9eee120ccb0e9d492.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"324","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Lateral ventricle","cell_type":"Neuronal cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2276,"metabolite_count":2478,"gene_count":1655,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/lateral_ventricle/lateral_ventricle_-_neuronal_cells_1.xml-44f2798b254b6bef6740c85ac5b55e8f.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"325","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Lateral ventricle","cell_type":"Neuronal cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4606,"metabolite_count":4261,"gene_count":1882,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/lateral_ventricle/lateral_ventricle_-_neuronal_cells_2.xml-6f38b76194b7d5144036c6be080cce0b.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"326","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Lateral ventricle","cell_type":"Glial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2116,"metabolite_count":2270,"gene_count":1473,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/lateral_ventricle/lateral_ventricle_-_glial_cells_1.xml-38a8ad5d4863a2fd4522b10f3614b1a0.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"327","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Lateral ventricle","cell_type":"Glial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4052,"metabolite_count":3916,"gene_count":1520,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/lateral_ventricle/lateral_ventricle_-_glial_cells_2.xml-a64abd99ab33e01f8965f318167fd4a4.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"328","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebellum","cell_type":"Purkinje cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2273,"metabolite_count":2443,"gene_count":1565,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebellum/cerebellum_-_purkinje_cells_1.xml-e00ac43a212551419a2df49b3652c50e.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"329","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebellum","cell_type":"Cells in granular layer","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2477,"metabolite_count":2605,"gene_count":1300,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebellum/cerebellum_-_cells_in_granular_layer_1.xml-92bb0f4a03125c868d91412339af41ab.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"33","gemodelset":{"name":"INIT cancer models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cancers. The first version models were generated for 16 different human cancers. In a study of clear cell renal carcinomas the INIT algorithm was used to generated revised models for 5 different cancers from Human Protein Atlas v11 data. For each cell type there is also provide a link to the Human Protein Atlas that provides further description of the human cell type and associted protein expression.","reference":["22615553","24550497"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver cancer","cell_type":null,"cell_line":null},"tag":null,"description":null,"condition":"Cancer","reaction_count":4388,"metabolite_count":4020,"gene_count":1715,"files":[{"path":"models/homo_sapiens/init_cancer_models/liver_cancer/liver_cancer_-__3.xml-07bff95988866a0e32afc4a0f072ebc9.zip","format":"SBML"}],"ref":["24550497"],"maintained":false,"year":"2014"},{"id":"330","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebellum","cell_type":"Cells in granular layer","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4287,"metabolite_count":3968,"gene_count":1633,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebellum/cerebellum_-_cells_in_granular_layer_2.xml-1cc6c9761c3b0b291fe7d342da515ddc.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"331","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebellum","cell_type":"Cells in molecular layer","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2136,"metabolite_count":2307,"gene_count":1480,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebellum/cerebellum_-_cells_in_molecular_layer_1.xml-b4641bfb519a162f9a9a59b6b5935cda.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"332","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Central nervous system","tissue":"Cerebellum","cell_type":"Cells in molecular layer","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4112,"metabolite_count":3919,"gene_count":1575,"files":[{"path":"models/homo_sapiens/init_normal_models/central_nervous_system/cerebellum/cerebellum_-_cells_in_molecular_layer_2.xml-9a0e5fb4d50aa5dc73a86e557aca0775.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"333","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Bone marrow","cell_type":"Hematopoietic cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2841,"metabolite_count":2878,"gene_count":1552,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/bone_marrow/bone_marrow_-_hematopoietic_cells_1.xml-9dd25a20ced362373aba460a32413211.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"334","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Bone marrow","cell_type":"Hematopoietic cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4550,"metabolite_count":4269,"gene_count":1848,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/bone_marrow/bone_marrow_-_hematopoietic_cells_2.xml-a8729ee06086fc9e438e4ba8ca1772f7.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"335","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Lymph node","cell_type":"Germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2139,"metabolite_count":2302,"gene_count":1512,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/lymph_node/lymph_node_-_germinal_center_cells_1.xml-09cae0786dc0f98aba948dd8b746569c.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"336","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Lymph node","cell_type":"Germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4078,"metabolite_count":3853,"gene_count":1536,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/lymph_node/lymph_node_-_germinal_center_cells_2.xml-f597ead6da9246be0f212d04f6816030.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"337","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Lymph node","cell_type":"Non-germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2522,"metabolite_count":2612,"gene_count":1444,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/lymph_node/lymph_node_-_non-germinal_center_cells_1.xml-a72b165638b53a5a6877ceecdc61505c.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"338","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Lymph node","cell_type":"Non-germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4343,"metabolite_count":4016,"gene_count":1764,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/lymph_node/lymph_node_-_non-germinal_center_cells_2.xml-81ea605812d26758c0c00e43018be09e.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"339","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Tonsil","cell_type":"Germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2179,"metabolite_count":2307,"gene_count":1520,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/tonsil/tonsil_-_germinal_center_cells_1.xml-0026cbea26aeb652e65905b78bcc3b1e.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"340","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Tonsil","cell_type":"Germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4263,"metabolite_count":3965,"gene_count":1691,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/tonsil/tonsil_-_germinal_center_cells_2.xml-b1c58080dd33bf8293f7d5dad2f4137a.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"341","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Tonsil","cell_type":"Non-germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2682,"metabolite_count":2773,"gene_count":1496,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/tonsil/tonsil_-_non-germinal_center_cells_1.xml-fb46ad8ae2a7b7571f51f18dc8c1a893.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"342","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Tonsil","cell_type":"Non-germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4389,"metabolite_count":4064,"gene_count":1722,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/tonsil/tonsil_-_non-germinal_center_cells_2.xml-8455f04636d605b6be10456de9452f55.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"343","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Tonsil","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2474,"metabolite_count":2617,"gene_count":1725,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/tonsil/tonsil_-_squamous_epithelial_cells_1.xml-0f0796b165fb5852811352e2cee847a8.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"344","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Tonsil","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5050,"metabolite_count":4245,"gene_count":1926,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/tonsil/tonsil_-_squamous_epithelial_cells_2.xml-b4be21b10e45670160a5974040085c85.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"345","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Spleen","cell_type":"Cells in white pulp","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":1915,"metabolite_count":2108,"gene_count":1357,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/spleen/spleen_-_cells_in_white_pulp_1.xml-1951ef84d14868e0560858596b05c9fb.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"346","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Spleen","cell_type":"Cells in white pulp","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3756,"metabolite_count":3726,"gene_count":1342,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/spleen/spleen_-_cells_in_white_pulp_2.xml-9a046a2cb0981f7370f2c91ac1444036.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"347","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Spleen","cell_type":"Cells in red pulp","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2220,"metabolite_count":2404,"gene_count":1553,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/spleen/spleen_-_cells_in_red_pulp_1.xml-9141b584f9ef8f602efc085a6fca0d29.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"348","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Blood and immune system","tissue":"Spleen","cell_type":"Cells in red pulp","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4124,"metabolite_count":3956,"gene_count":1573,"files":[{"path":"models/homo_sapiens/init_normal_models/blood_and_immune_system/spleen/spleen_-_cells_in_red_pulp_2.xml-9e46e1e1ae64626acc0f37225634e281.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"349","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Respiratory system","tissue":"Nasopharynx","cell_type":"Respiratory epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2556,"metabolite_count":2660,"gene_count":1793,"files":[{"path":"models/homo_sapiens/init_normal_models/respiratory_system/nasopharynx/nasopharynx_-_respiratory_epithelial_cells_1.xml-3e7a75fea2293f66e7924f97d6edfd81.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"35","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver cancer","cell_type":null,"cell_line":null},"tag":"patient 2177","description":null,"condition":"Cancer","reaction_count":4823,"metabolite_count":4339,"gene_count":1823,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/liver_cancer/livercancerpatient2177.xml-64c5ae1b40010d576acd52c489f290b7.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"350","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Respiratory system","tissue":"Nasopharynx","cell_type":"Respiratory epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4884,"metabolite_count":4380,"gene_count":2120,"files":[{"path":"models/homo_sapiens/init_normal_models/respiratory_system/nasopharynx/nasopharynx_-_respiratory_epithelial_cells_2.xml-37ec67c311e117246aefc299a1a3a313.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"351","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Respiratory system","tissue":"Bronchus","cell_type":"Respiratory epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3037,"metabolite_count":3009,"gene_count":1754,"files":[{"path":"models/homo_sapiens/init_normal_models/respiratory_system/bronchus/bronchus_-_respiratory_epithelial_cells_1.xml-ebb18bb3f3c9e705164b4fe17700d555.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"352","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Respiratory system","tissue":"Bronchus","cell_type":"Respiratory epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4887,"metabolite_count":4377,"gene_count":2119,"files":[{"path":"models/homo_sapiens/init_normal_models/respiratory_system/bronchus/bronchus_-_respiratory_epithelial_cells_2.xml-f590cbdaeb1345039b25d0cf331b30a3.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"353","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Respiratory system","tissue":"Lung","cell_type":"Pneumocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2607,"metabolite_count":2693,"gene_count":1435,"files":[{"path":"models/homo_sapiens/init_normal_models/respiratory_system/lung/lung_-_pneumocytes_1.xml-9bc9bc40c77dae4e84f53fa20d78bafb.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"354","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Respiratory system","tissue":"Lung","cell_type":"Pneumocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4498,"metabolite_count":3921,"gene_count":1492,"files":[{"path":"models/homo_sapiens/init_normal_models/respiratory_system/lung/lung_-_pneumocytes_2.xml-3ed499d4ae4075b888f0c52e7f40a9cb.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"355","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Respiratory system","tissue":"Lung","cell_type":"Macrophages","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2595,"metabolite_count":2692,"gene_count":1782,"files":[{"path":"models/homo_sapiens/init_normal_models/respiratory_system/lung/lung_-_macrophages_1.xml-f0a590a9e4245579e463c2b8acb9e8a4.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"356","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Respiratory system","tissue":"Lung","cell_type":"Macrophages","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5184,"metabolite_count":4376,"gene_count":2075,"files":[{"path":"models/homo_sapiens/init_normal_models/respiratory_system/lung/lung_-_macrophages_2.xml-e4423379ea57c426fff128c12c32cdfd.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"357","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Endocrine glands","tissue":"Thyroid gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2971,"metabolite_count":2956,"gene_count":1760,"files":[{"path":"models/homo_sapiens/init_normal_models/endocrine_glands/thyroid_gland/thyroid_gland_-_glandular_cells_1.xml-27646064c7f6dbca37c52e2c88f6ff19.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"358","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Endocrine glands","tissue":"Thyroid gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4915,"metabolite_count":4344,"gene_count":2104,"files":[{"path":"models/homo_sapiens/init_normal_models/endocrine_glands/thyroid_gland/thyroid_gland_-_glandular_cells_2.xml-5be90ce20d101df63aa4548811531a1a.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"359","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Endocrine glands","tissue":"Parathyroid gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2448,"metabolite_count":2602,"gene_count":1708,"files":[{"path":"models/homo_sapiens/init_normal_models/endocrine_glands/parathyroid_gland/parathyroid_gland_-_glandular_cells_1.xml-c3a0c2abe8f941090bb60b190c3d81c1.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"36","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver cancer","cell_type":null,"cell_line":null},"tag":"patient 2280","description":null,"condition":"Cancer","reaction_count":4954,"metabolite_count":4446,"gene_count":2025,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/liver_cancer/livercancerpatient2280.xml-8ebcaf5dc28f978577a360e30e19e37b.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"360","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Endocrine glands","tissue":"Parathyroid gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4723,"metabolite_count":4280,"gene_count":1847,"files":[{"path":"models/homo_sapiens/init_normal_models/endocrine_glands/parathyroid_gland/parathyroid_gland_-_glandular_cells_2.xml-0595b311d778feba8be2e853b00f1fb7.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"361","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Endocrine glands","tissue":"Adrenal gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":3010,"metabolite_count":2974,"gene_count":1680,"files":[{"path":"models/homo_sapiens/init_normal_models/endocrine_glands/adrenal_gland/adrenal_gland_-_glandular_cells_1.xml-d1a7c73499b85c4a85dab531950848a3.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"362","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Endocrine glands","tissue":"Adrenal gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5038,"metabolite_count":4378,"gene_count":2182,"files":[{"path":"models/homo_sapiens/init_normal_models/endocrine_glands/adrenal_gland/adrenal_gland_-_glandular_cells_2.xml-3dec68e7f4d87c46fb63b17cd9d9611a.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"363","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Cardiovascular system","tissue":"Heart muscle","cell_type":"Myocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2634,"metabolite_count":2710,"gene_count":1487,"files":[{"path":"models/homo_sapiens/init_normal_models/cardiovascular_system/heart_muscle/heart_muscle_-_myocytes_1.xml-1cb23a9213902fe0d3947e5c27316725.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"364","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Cardiovascular system","tissue":"Heart muscle","cell_type":"Myocytes","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":5040,"metabolite_count":4268,"gene_count":1964,"files":[{"path":"models/homo_sapiens/init_normal_models/cardiovascular_system/heart_muscle/heart_muscle_-_myocytes_2.xml-3c49df1fef3ecda73fa5cdacf1d7e485.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"366","gemodelset":{"name":"Human Metabolic Reaction","description":"The Human Metabolic Reaction (HMR) is a genome-scale metabolic model for the generic human cell. In its latest version (HMR2), it contains 3,765 genes associated with over 8,000 reactions and over 3,000 unique metabolites. For simulation purposes in MATLAB, the authors recommend to import the SBML file using the RAVEN Toolbox function importModel.","reference":[]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":null,"cell_type":"Generic cell","cell_line":null},"tag":"HMR 1.0","description":null,"condition":"Evaluating metabolic capabilities","reaction_count":8174,"metabolite_count":6006,"gene_count":3674,"files":[{"path":"models/homo_sapiens/human_metabolic_reaction/generic_cell/HMRcollection.xml-8849b3803fdcc7dbd26fea46ef0dcc50.zip","format":"SBML"},{"path":"models/homo_sapiens/human_metabolic_reaction/generic_cell/HMRdatabase1_00.xlsx-200fd7ed2b9e440e7a2fce45f67a12dd.zip","format":"Excel"}],"ref":["23511207"],"maintained":false,"year":"2013"},{"id":"367","gemodelset":{"name":"Human Metabolic Reaction","description":"The Human Metabolic Reaction (HMR) is a genome-scale metabolic model for the generic human cell. In its latest version (HMR2), it contains 3,765 genes associated with over 8,000 reactions and over 3,000 unique metabolites. For simulation purposes in MATLAB, the authors recommend to import the SBML file using the RAVEN Toolbox function importModel.","reference":[]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":null,"cell_type":"Generic cell","cell_line":null},"tag":"HMR 2.0","description":null,"condition":"Evaluating metabolic capabilities","reaction_count":8181,"metabolite_count":6007,"gene_count":3765,"files":[{"path":"models/homo_sapiens/human_metabolic_reaction/generic_cell/HMRcollection2_00.xml-f0de1f951d16f78abf131cece19f8af7.zip","format":"SBML"},{"path":"models/homo_sapiens/human_metabolic_reaction/generic_cell/HMRdatabase2_00.xlsx-07f0f2b5f71a03b0d6efdea752a9cebb.zip","format":"Excel"},{"path":"models/homo_sapiens/human_metabolic_reaction/generic_cell/HMRdatabase2_00_Cobra.xml-b36c4d8bbafd1a3b8bf3b49e8b7cad35.zip","format":"COBRA-friendly SBML"},{"path":"models/homo_sapiens/human_metabolic_reaction/generic_cell/HMRdatabase2_00.mat-6142d4b2c5f4679ddd6c34de85eef405.zip","format":"MATLAB"}],"ref":["24419221"],"maintained":false,"year":"2014"},{"id":"37","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver cancer","cell_type":null,"cell_line":null},"tag":"patient 2556","description":null,"condition":"Cancer","reaction_count":4690,"metabolite_count":4324,"gene_count":1715,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/liver_cancer/livercancerpatient2556.xml-34a4b01be4ef6c39ed4e356f6cae4488.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"38","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver cancer","cell_type":null,"cell_line":null},"tag":"patient 2766","description":null,"condition":"Cancer","reaction_count":4967,"metabolite_count":4418,"gene_count":2009,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/liver_cancer/livercancerpatient2766.xml-496ac7dfe8f83a2e50c828b6b3a6a9d0.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"39","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver cancer","cell_type":null,"cell_line":null},"tag":"patient 3196","description":null,"condition":"Cancer","reaction_count":4764,"metabolite_count":4377,"gene_count":1860,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/liver_cancer/livercancerpatient3196.xml-a2893ed8a7d8aba051b92d0ea6436fb6.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"4","gemodelset":{"name":"Bacteria models","description":null,"reference":[]},"sample":{"organism":"Synechocystis sp. pcc6803","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iSyn669","description":"Synechocystis sp. PCC6803 is a cyanobacterium considered as a candidate photo-biological production platform an attractive cell factory capable of using CO2 and light as carbon and energy source, respectively. Compilation and characterization of Synechocystis genome-scale metabolic model is a pre-requisite towards achieving a proficient photosynthetic cell factory. To this end, we report two genome-scale metabolic models: iSyn669, which includes 882 reactions, associated with 669 genes, and 790 metabolites and iSyn811, an upgraded genome-scale metabolic model of this cyanobacteria consisting of 956 reactions and accounting for 811 genes and 911 metabolites. These models have been thoroughly studied with tools and algorithms such as flux balance analysis, knock-out analysis, gene essentiality, flux coupling analysis, reporter features and industrially-relevant metabolites production (hydrogen, succinate, ethanol). These models will greatly facilitate the development of cyanobacteria as green microbial cell factories.","condition":"Evaluating metabolic capabilities","reaction_count":882,"metabolite_count":790,"gene_count":669,"files":[{"path":"models/bacteria_models/synechocystis_sp._pcc6803/iSyn669.zip","format":"BioOpt"}],"ref":["21083885","21226012"],"maintained":false,"year":"2011"},{"id":"40","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver cancer","cell_type":null,"cell_line":null},"tag":"patient 3477","description":null,"condition":"Cancer","reaction_count":4833,"metabolite_count":4341,"gene_count":1803,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/liver_cancer/livercancerpatient3477.xml-46beb717835fc8d6b3b5a0bbf00d247b.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"41","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Adrenal gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5365,"metabolite_count":4427,"gene_count":2273,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/adrenal_gland/adrenal gland - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"42","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Appendix","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5590,"metabolite_count":4482,"gene_count":2217,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/appendix/appendix - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"43","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Appendix","cell_type":"Lymphoid tissue","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4460,"metabolite_count":4017,"gene_count":1566,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/appendix/appendix - lymphoid tissue.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"44","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Bone marrow","cell_type":"Hematopoietic cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4847,"metabolite_count":4266,"gene_count":1888,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/bone_marrow/bone marrow - hematopoietic cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"45","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Breast","cell_type":"Adipocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3180,"metabolite_count":3238,"gene_count":1106,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/breast/breast - adipocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"46","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Breast","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5348,"metabolite_count":4287,"gene_count":2027,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/breast/breast - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"47","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Breast","cell_type":"Myoepithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4976,"metabolite_count":4081,"gene_count":1807,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/breast/breast - myoepithelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"48","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Bronchus","cell_type":"Respiratory epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5174,"metabolite_count":4423,"gene_count":2191,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/bronchus/bronchus - respiratory epithelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"49","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cerebellum","cell_type":"Cells in granular layer","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4555,"metabolite_count":3975,"gene_count":1684,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/cerebellum/cerebellum - cells in granular layer.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"5","gemodelset":{"name":"Bacteria models","description":null,"reference":[]},"sample":{"organism":"Synechocystis sp. pcc6803","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"iSyn811","description":"Synechocystis sp. PCC6803 is a cyanobacterium considered as a candidate photo-biological production platform an attractive cell factory capable of using CO2 and light as carbon and energy source, respectively. Compilation and characterization of Synechocystis genome-scale metabolic model is a pre-requisite towards achieving a proficient photosynthetic cell factory. To this end, we report two genome-scale metabolic models: iSyn669, which includes 882 reactions, associated with 669 genes, and 790 metabolites and iSyn811, an upgraded genome-scale metabolic model of this cyanobacteria consisting of 956 reactions and accounting for 811 genes and 911 metabolites. These models have been thoroughly studied with tools and algorithms such as flux balance analysis, knock-out analysis, gene essentiality, flux coupling analysis, reporter features and industrially-relevant metabolites production (hydrogen, succinate, ethanol). These models will greatly facilitate the development of cyanobacteria as green microbial cell factories.","condition":"Aerobic and anaerobic continuous cultures","reaction_count":956,"metabolite_count":911,"gene_count":811,"files":[{"path":"models/bacteria_models/synechocystis_sp._pcc6803/iSyn811.zip","format":"BioOpt"}],"ref":["21083885","21226012"],"maintained":false,"year":"2011"},{"id":"50","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cerebellum","cell_type":"Cells in molecular layer","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4350,"metabolite_count":3955,"gene_count":1599,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/cerebellum/cerebellum - cells in molecular layer.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"51","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cerebellum","cell_type":"Purkinje cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4672,"metabolite_count":4203,"gene_count":1911,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/cerebellum/cerebellum - Purkinje cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"52","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cerebral cortex","cell_type":"Endothelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4718,"metabolite_count":3974,"gene_count":1560,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/cerebral_cortex/cerebral cortex - endothelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"53","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cerebral cortex","cell_type":"Glial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4220,"metabolite_count":3978,"gene_count":1595,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/cerebral_cortex/cerebral cortex - glial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"54","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cerebral cortex","cell_type":"Neuronal cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4918,"metabolite_count":4311,"gene_count":2051,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/cerebral_cortex/cerebral cortex - neuronal cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"55","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cerebral cortex","cell_type":"Neuropil","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4565,"metabolite_count":4046,"gene_count":1776,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/cerebral_cortex/cerebral cortex - neuropil.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"56","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cervix, uterine","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4678,"metabolite_count":4138,"gene_count":1769,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/cervix_uterine/cervix, uterine - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"57","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Cervix, uterine","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4503,"metabolite_count":3984,"gene_count":1675,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/cervix_uterine/cervix, uterine - squamous epithelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"58","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Colon","cell_type":"Endothelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5017,"metabolite_count":4186,"gene_count":1798,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/colon/colon - endothelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"59","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Colon","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5646,"metabolite_count":4483,"gene_count":2311,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/colon/colon - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"6","gemodelset":{"name":"Bacteria models","description":null,"reference":[]},"sample":{"organism":"Amycolatopsis balhimycina","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":"Amycolatopsis balhimycina","description":"Infection caused by methicillin-resistant Staphylococcus aureus (MRSA) is an increasing societal problem. Typically treatments of these infections can only be done with the glycopeptides antibiotics vancomycin, synercid, and teiocoplanin. Amycolatopsis balhimycina serves as a model strain for glycopeptide production due to its ability to produce the vancomycin-analogue balhimycin. The balhimycin yield obtained by A. balhimycina is, however, low and there is therefore a need to improve balhimycin production by this organism and this may also lead to a wider use of this organism as a model for glycopeptide production. With the objective to improve the knowledge base of this organism, we therefore performed genome-sequencing of A. balhimycina and used these data to reconstruct a genome-scale metabolic model for the organism. We generated an almost complete A. balhimycina genome sequence comprising 10,562,587 base pairs assembled in 2,153 contigs. The high GC-genome (~69%) included 8,585 open reading frames (ORFs). We used our integrative toolbox called SEQTOR for gene annotation, and used annotated genes to reconstruct a genome-scale metabolic network of A. balhimycina. In the model reconstruction we also used biochemical information retrieved from pathway databases, biochemistry textbooks and recent publications. The resulting network contained 583 ORFs as protein encoding genes (7% of the predicted 8,585 ORFs), 407 EC numbers, 647 metabolites and 1,363 metabolic reactions. During the analysis of the metabolic model, linear, quadratic and evolutionary programming problems using Flux Balance Analysis (FBA), Minimization Of Metabolic Adjustments (MOMA) and OptGene were applied and phenotypic behavior and improved balhimycin production were simulated. The model showed a good agreement between in silico data and experimental data and also identified key reactions associated with increased balhimycin production. The reconstruction of the genome-scale metabolic model of A. balhimycina serves as a basis for physiological characterization. The model allows a rational design of engineering strategies for increasing balhimycin production in A. balhimycina and glycopeptide production in general.","condition":"Evaluating metabolic capabilities","reaction_count":1363,"metabolite_count":647,"gene_count":583,"files":[{"path":"models/bacteria_models/amycolatopsis_balhimycina/final_AB_model_Bioopt_BioMet.zip","format":"BioOpt"}],"ref":["22252737"],"maintained":false,"year":"2010"},{"id":"60","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Colon","cell_type":"Peripheral nerve-ganglion","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5000,"metabolite_count":4151,"gene_count":1701,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/colon/colon - peripheral nerve-ganglion.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"61","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Duodenum","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5744,"metabolite_count":4545,"gene_count":2367,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/duodenum/duodenum - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"62","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Epididymis","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5461,"metabolite_count":4416,"gene_count":2220,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/epididymis/epididymis - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"63","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Esophagus","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5148,"metabolite_count":4188,"gene_count":1968,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/esophagus/esophagus - squamous epithelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"64","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Fallopian tube","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5088,"metabolite_count":4391,"gene_count":2161,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/fallopian_tube/fallopian tube - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"65","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Gallbladder","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5559,"metabolite_count":4469,"gene_count":2334,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/gallbladder/gallbladder - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"66","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Heart muscle","cell_type":"Myocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5275,"metabolite_count":4334,"gene_count":2025,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/heart_muscle/heart muscle - myocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"67","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Hippocampus","cell_type":"Glial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4056,"metabolite_count":3915,"gene_count":1432,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/hippocampus/hippocampus - glial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"68","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Hippocampus","cell_type":"Neuronal cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4635,"metabolite_count":4244,"gene_count":1958,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/hippocampus/hippocampus - neuronal cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"69","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Kidney","cell_type":"Cells in glomeruli","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4124,"metabolite_count":3961,"gene_count":1541,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/kidney/kidney - cells in glomeruli.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"7","gemodelset":{"name":"Bacteria models","description":null,"reference":[]},"sample":{"organism":"Pseudomonas fluorescens","organ_system":null,"tissue":null,"cell_type":null,"cell_line":null},"tag":null,"description":"Alginate biosynthesis in Pseudomonas fluorescens was studied in a genomics perspective, using an alginate over-producing strain carrying a mutation in the anti-sigma factor gene mucA. Cells were cultivated in chemostats under nitrogen limitation on fructose or glycerol as carbon sources, and cell mass, growth rate, sugar uptake, alginate and CO2 production were monitored. In addition a genome scale metabolic model was constructed and samples were collected for transcriptome analyses. The analyses show that polymer production operates in a close to optimal way with respect to stoichiometric utilization of the carbon source and that the cells increase the uptake of carbon source to compensate for the additional needs following from alginate synthesis. The transcriptome studies show that in the presence of the mucA mutation, the alg operon is upregulated together with genes involved in energy generation, genes on both sides of the succinate node of the TCA cycle and genes encoding ribosomal and other translation-related proteins. Strains expressing a functional MucA protein (no alginate production) synthesize cellular biomass in an inefficient way, apparently due to a cycle that involves oxidation of NADPH without ATP production. The results of this study indicate that the most efficient way of using a mucA mutant as a cell factory for alginate production would be to use non-growing conditions and nitrogen deprivation.","condition":"Evaluating metabolic capabilities metabolic capabilities","reaction_count":1191,"metabolite_count":0,"gene_count":0,"files":[{"path":"models/bacteria_models/pseudomonas_fluorescens/Pflur.zip","format":"Excel"}],"ref":[],"maintained":false,"year":null},{"id":"70","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Kidney","cell_type":"Cells in tubules","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5428,"metabolite_count":4498,"gene_count":2415,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/kidney/kidney - cells in tubules.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"71","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lateral ventricle","cell_type":"Glial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4150,"metabolite_count":3998,"gene_count":1529,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/lateral_ventricle/lateral ventricle - glial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"72","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lateral ventricle","cell_type":"Neuronal cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4822,"metabolite_count":4283,"gene_count":1921,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/lateral_ventricle/lateral ventricle - neuronal cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"73","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver","cell_type":"Bile duct cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4101,"metabolite_count":3866,"gene_count":1435,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/liver/liver - bile duct cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"74","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Liver","cell_type":"Hepatocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5413,"metabolite_count":4434,"gene_count":1957,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/liver/liver - hepatocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"75","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lung","cell_type":"Macrophages","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5437,"metabolite_count":4438,"gene_count":2113,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/lung/lung - macrophages.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"76","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Vagina","cell_type":"Squamous epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":2386,"metabolite_count":2518,"gene_count":1270,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/vagina/vagina_-_squamous_epithelial_cells_1.xml-4ebdf8ab1e249a0d88209d61840f12a3.zip","format":"SBML"}],"ref":["22615553"],"maintained":false,"year":"2012"},{"id":"77","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lung","cell_type":"Pneumocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4630,"metabolite_count":3937,"gene_count":1530,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/lung/lung - pneumocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"78","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lymph node","cell_type":"Germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4337,"metabolite_count":3880,"gene_count":1555,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/lymph_node/lymph node - germinal center cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"79","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Lymph node","cell_type":"Non-germinal center cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4565,"metabolite_count":4017,"gene_count":1809,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/lymph_node/lymph node - non-germinal center cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"8","gemodelset":{"name":"INIT normal models","description":"INIT models are automatically generated Genome-Scale Metabolic Models for different human cell types. The first version models were generated for 69 different human cell types whereas the second version models were generated by a revised algorithm, tINIT, where the toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type‐specific GEMs.","reference":["22615553","24646661"]},"sample":{"organism":"Homo sapiens","organ_system":"Breast and female reproductive system","tissue":"Uterus, post-menopause","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Healthy","reaction_count":4704,"metabolite_count":4267,"gene_count":1944,"files":[{"path":"models/homo_sapiens/init_normal_models/breast_and_female_reproductive_system/uterus_post-menopause/uterus,_post-menopause_-_glandular_cells_2.xml-6faf013aa8e17313cccfc5583dc87baf.zip","format":"SBML"}],"ref":["24646661"],"maintained":false,"year":"2014"},{"id":"80","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Ovary","cell_type":"Follicle cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3809,"metabolite_count":3625,"gene_count":1164,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/ovary/ovary - follicle cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"81","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Ovary","cell_type":"Ovarian stroma cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":3667,"metabolite_count":3570,"gene_count":1278,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/ovary/ovary - ovarian stroma cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"82","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Pancreas","cell_type":"Exocrine glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5183,"metabolite_count":4422,"gene_count":2164,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/pancreas/pancreas - exocrine glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"83","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Pancreas","cell_type":"Islets of langerhans","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4756,"metabolite_count":4207,"gene_count":1783,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/pancreas/pancreas - islets of Langerhans.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"84","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Parathyroid gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5048,"metabolite_count":4355,"gene_count":1898,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/parathyroid_gland/parathyroid gland - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"85","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Placenta","cell_type":"Decidual cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4349,"metabolite_count":3997,"gene_count":1588,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/placenta/placenta - decidual cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"86","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Placenta","cell_type":"Trophoblastic cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5481,"metabolite_count":4414,"gene_count":2078,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/placenta/placenta - trophoblastic cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"87","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Prostate","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5166,"metabolite_count":4328,"gene_count":2021,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/prostate/prostate - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"88","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Rectum","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5619,"metabolite_count":4487,"gene_count":2310,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/rectum/rectum - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"89","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Salivary gland","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5101,"metabolite_count":4316,"gene_count":2062,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/salivary_gland/salivary gland - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"9","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Nasopharynx","cell_type":"Respiratory epithelial cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5133,"metabolite_count":4384,"gene_count":2188,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/nasopharynx/nasopharynx - respiratory epithelial cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"90","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Seminal vesicle","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5207,"metabolite_count":4426,"gene_count":2117,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/seminal_vesicle/seminal vesicle - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"91","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skeletal muscle","cell_type":"Myocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4952,"metabolite_count":4136,"gene_count":1864,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/skeletal_muscle/skeletal muscle - myocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"92","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skin 1","cell_type":"Fibroblasts","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4652,"metabolite_count":3945,"gene_count":1574,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/skin_1/skin 1 - fibroblasts.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"93","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skin 1","cell_type":"Keratinocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4729,"metabolite_count":4126,"gene_count":1848,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/skin_1/skin 1 - keratinocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"94","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skin 1","cell_type":"Langerhans","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4195,"metabolite_count":3788,"gene_count":1546,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/skin_1/skin 1 - Langerhans.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"95","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skin 1","cell_type":"Melanocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4827,"metabolite_count":4062,"gene_count":1707,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/skin_1/skin 1 - melanocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"96","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Skin 2","cell_type":"Epidermal cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5015,"metabolite_count":4285,"gene_count":2036,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/skin_2/skin 2 - epidermal cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"97","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Small intestine","cell_type":"Glandular cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":5707,"metabolite_count":4532,"gene_count":2372,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/small_intestine/small intestine - glandular cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"98","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Smooth muscle","cell_type":"Smooth muscle cells","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4714,"metabolite_count":3910,"gene_count":1634,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/smooth_muscle/smooth muscle - smooth muscle cells.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"},{"id":"99","gemodelset":{"name":"personalized genome-scale metabolic models","description":"Genome-scale metabolic models (GEMs) serve as scaffolds for integration of omics data for understanding the genotype-phenotype relationship. Here, we evaluated the presence/absence of proteins encoded by 15,841 genes in 27 hepatocellular carcinoma (HCC) patients using immunohistochemistry and reconstructed personalized GEMs for six HCC patients using the proteomics data, task-driven model reconstruction (tINIT) algorithm and HMR 2.0. The personalized GEMs were employed to identify anticancer drugs using the concept of antimetabolites i.e. drugs that are structural analogs to metabolites. The toxicity of each antimetabolite was predicted by assessing the in silico functionality of 83 healthy cell type-specific GEMs, which were also reconstructed with the tINIT algorithm. We predicted 101 antimetabolites that are effective in preventing tumor growth in all HCC patients and 46 antimetabolites that are specific to individual patients. 22 of the 101 predicted antimetabolites have already been used in different cancer treatment strategies, while the remaining antimetabolites represent new potential drugs. Finally, one of the identified targets was validated experimentally, and it was confirmed to attenuate growth of the HepG2 cell line.","reference":["24646661"]},"sample":{"organism":"Homo sapiens","organ_system":null,"tissue":"Soft tissue 1","cell_type":"Adipocytes","cell_line":null},"tag":null,"description":null,"condition":"Metabolic state of healthy versus diseased cells","reaction_count":4062,"metabolite_count":3567,"gene_count":1260,"files":[{"path":"models/homo_sapiens/personalized_genome-scale_metabolic_models/soft_tissue_1/soft tissue 1 - adipocytes.zip","format":"SBML"}],"ref":[],"maintained":false,"year":"2014"}]');function Rt(e,t){var n=Object.keys(e);if(Object.getOwnPropertySymbols){var a=Object.getOwnPropertySymbols(e);t&&(a=a.filter((function(t){return Object.getOwnPropertyDescriptor(e,t).enumerable}))),n.push.apply(n,a)}return n}function zt(e){for(var t=1;t<arguments.length;t++){var n=null!=arguments[t]?arguments[t]:{};t%2?Rt(Object(n),!0).forEach((function(t){i()(e,t,n[t])})):Object.getOwnPropertyDescriptors?Object.defineProperties(e,Object.getOwnPropertyDescriptors(n)):Rt(Object(n)).forEach((function(t){Object.defineProperty(e,t,Object.getOwnPropertyDescriptor(n,t))}))}return e}var Ot=t().Router(),Lt=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.apiName,a=t.apiVersion,e.next=3,Promise.all([D(n,a),P(n,a),se(n,a)]);case 3:return i=e.sent,o=u()(i,3),s=o[0],l=o[1],c=o[2],e.abrupt("return",zt(zt({},t),{},{gene_count:s,reaction_count:l,metabolite_count:c}));case 9:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();Ot.get("/integrated_models",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.prev=0,a=E.map((function(e){return zt(zt({},e),{},{apiName:e.short_name.split("-").map((function(e){return e[0]+e.slice(1).toLowerCase()})).join(""),apiVersion:e.version.split(".").join("_")})})).sort((function(e,t){return e.short_name.toLowerCase()<t.short_name.toLowerCase()?-1:1})),e.next=4,Promise.all(a.map((function(e){return Lt(e)})));case 4:i=e.sent,n.json(i),e.next=11;break;case 8:e.prev=8,e.t0=e.catch(0),n.status(400).send(e.t0.message);case 11:case"end":return e.stop()}}),e,null,[[0,8]])})));return function(t,n){return e.apply(this,arguments)}}()),Ot.get("/integrated_models/:name",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:a=t.params.name;try{(i=E.filter((function(e){return e.short_name.toLowerCase()===a.toLowerCase()}))[0])?n.json(i):n.sendStatus(404)}catch(e){n.status(400).send(e.message)}case 2:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Ot.get("/models",function(){var e=s()(r().mark((function e(t,n){return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:try{n.json(Et.sort((function(e,t){return e.id-t.id})))}catch(e){n.status(400).send(e.message)}case 1:case"end":return e.stop()}}),e)})));return function(t,n){return e.apply(this,arguments)}}()),Ot.use("/models",t().static("repository"));const At=Ot,Bt=require("@babel/runtime/helpers/toArray");var kt=n.n(Bt);const Dt=require("fs/promises");var Wt=E.map((function(e){return e.short_name})),Ft={},jt=function(){var e=s()(r().mark((function e(t,n,a){var i,o,s,l;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(i="".concat(t,"_").concat(n,"_").concat(a),!Ft[i]){e.next=3;break}return e.abrupt("return",Ft[i]);case 3:if(Wt.includes(t)){e.next=5;break}throw new Error("Invalid model provided: ".concat(t,"."));case 5:if(o=/^([a-zA-Z0-9][^*/><?\"|:]*)$/,n.match(o)){e.next=8;break}throw new Error("Invalid data type provided: ".concat(n,"."));case 8:if(s=/^[\w\-. ]+\.tsv$/i,a.match(s)){e.next=11;break}throw new Error("Invalid filename provided: ".concat(a,"."));case 11:return e.next=13,(0,Dt.readFile)("./dataOverlay/".concat(t,"/").concat(n,"/").concat(a),"utf8");case 13:return l=e.sent,Ft[i]=l,e.abrupt("return",l);case 16:case"end":return e.stop()}}),e)})));return function(t,n,a){return e.apply(this,arguments)}}(),Pt=t().Router();Pt.get("/:model",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(a=t.params.model,e.prev=1,Wt.includes(a)){e.next=4;break}throw new Error("Invalid model provided: ".concat(a,"."));case 4:return e.next=6,(0,Dt.readFile)("./dataOverlay/".concat(a,"/index.json"),"utf8");case 6:i=e.sent,n.json(JSON.parse(i)),e.next=13;break;case 10:e.prev=10,e.t0=e.catch(1),"ENOENT"===e.t0.code?n.json({}):(console.error(e.t0.message),n.sendStatus(404));case 13:case"end":return e.stop()}}),e,null,[[1,10]])})));return function(t,n){return e.apply(this,arguments)}}()),Pt.get("/:model/:dataType/:filename/data-sets",function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l,c,d,m;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params,i=a.model,o=a.dataType,s=a.filename,e.prev=1,e.next=4,jt(i,o,s);case 4:l=e.sent,c=l.split(/\r?\n/)[0].split("\t"),d=kt()(c),m=d.slice(1),n.json(m.sort((function(e,t){return e.localeCompare(t)}))),e.next=13;break;case 9:e.prev=9,e.t0=e.catch(1),console.error(e.t0.message),n.sendStatus(404);case 13:case"end":return e.stop()}}),e,null,[[1,9]])})));return function(t,n){return e.apply(this,arguments)}}()),Pt.get("/:model/:dataType/:filename/data-sets/:dataSetName",function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l,c,d,m,u,p,f,h,g;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params,i=a.model,o=a.dataType,s=a.filename,l=a.dataSetName,e.prev=1,e.next=4,jt(i,o,s);case 4:for(c=e.sent,d=c.split(/\r?\n/),m=d[0].split("\t"),u=m.indexOf(l),kt()(m).slice(1),p={},f=1;f<d.length;f++)h=d[f].split("\t"),g=h[u],p[h[0]]=g;n.json(p),e.next=18;break;case 14:e.prev=14,e.t0=e.catch(1),console.error(e.t0.message),n.sendStatus(404);case 18:case"end":return e.stop()}}),e,null,[[1,14]])})));return function(t,n){return e.apply(this,arguments)}}()),Pt.get("/:dataType/example",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:a=t.params.dataType,e.prev=1,i="",e.t0=a,e.next="genes"===e.t0?6:"reactions"===e.t0?8:10;break;case 6:return i="id\theart\tliver\nENSG00000177666\t0.484\t0.349\nENSG00000175535\t0.564\t0\nENSG00000187021\t0.114\t0",e.abrupt("break",11);case 8:return i="id\tadipose\ttissue_t-cells_9\nMAR03905\t0.484\t0.349\nMAR03907\t0.564\t0\nMAR04097\t0.114\t0",e.abrupt("break",11);case 10:n.sendStatus(404);case 11:n.set("Content-Type","text/tab-separated-values"),n.set("Content-Disposition","attachment; filename=".concat(a,"-example.tsv")),n.send(i),e.next=20;break;case 16:e.prev=16,e.t1=e.catch(1),console.error(e.t1.message),n.sendStatus(404);case 20:case"end":return e.stop()}}),e,null,[[1,16]])})));return function(t,n){return e.apply(this,arguments)}}());const Ut=Pt,Vt=require("express-rate-limit");var qt=n.n(Vt);const $t=require("sanitize-filename");var Jt=n.n($t),Kt=(n(441),function(){var e=s()(r().mark((function e(t,a,i){var o,s,l;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return o=n(441),s="/project/svg/".concat(i,"/").concat(t,".svg"),e.next=4,o(s,{limitInputPixels:!1}).resize(a).flatten({background:{r:255,g:255,b:255}}).png().toBuffer().then((function(e){return e})).catch((function(e){}));case 4:return l=e.sent,e.abrupt("return",l);case 6:case"end":return e.stop()}}),e)})));return function(t,n,a){return e.apply(this,arguments)}}());const Yt=Kt;var Xt=E.map((function(e){return e.short_name})),Zt=t().Router(),Qt=qt()({windowMs:1e3,max:200});function en(e){if(e){var t=parseInt(e)||null;if(!/^\d+$/.test(e)||t<100||t>600)throw new Error("Invalid dimension provided: ".concat(e,". The width and/or height parameters should be an integer between ").concat(100," and ").concat(600,"."))}return t||null}Zt.get("/:svgName",Qt,function(){var e=s()(r().mark((function e(t,n){var a,i,o,s,l,c,d,m;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(a=t.params.svgName,i=t.query,o=i.model,i.version,s=i.width,l=i.height,e.prev=2,Xt.includes(o)){e.next=5;break}throw new Error("Invalid model provided: ".concat(o,"."));case 5:if(c="/project/svg/".concat(Jt()(o),"/").concat(Jt()(a),".svg"),et().existsSync(c)){e.next=8;break}throw new Error("2D map does not exist for ".concat(a," in ").concat(o,"."));case 8:return d={width:en(s),height:en(l),fit:"cover"},e.next=11,Yt(a,d,o);case 11:m=e.sent,n.type("png"),n.send(m),e.next=21;break;case 16:if(e.prev=16,e.t0=e.catch(2),"404"!==e.t0.message){e.next=20;break}return e.abrupt("return",n.sendStatus(404));case 20:n.status(400).send(escape(e.t0.message));case 21:case"end":return e.stop()}}),e,null,[[2,16]])})));return function(t,n){return e.apply(this,arguments)}}());const tn=Zt;var nn,an=function(){var e=s()(r().mark((function e(t){var n;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,Z()(nn||(nn=Y()(["\n    select * from domains\n    where abbreviation = ","\n  "])),t.toString());case 2:if((n=e.sent).length){e.next=5;break}throw new Error("Domain ".concat(t," returned ").concat(n.length," results."));case 5:return e.abrupt("return",{info:n[0],crossReferences:[]});case 6:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();const on=an;var sn;const ln=function(){var e=s()(r().mark((function e(t){var n;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,Z()(sn||(sn=Y()(["\n    select * from ec\n    where ec = ","\n  "])),t.toString());case 2:if((n=e.sent).length){e.next=5;break}throw new Error("EC ".concat(t," returned ").concat(n.length," results."));case 5:return e.abrupt("return",{info:n[0],crossReferences:[]});case 6:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();var rn,cn,dn,mn,un,pn,fn,hn,gn,yn,bn,_n,vn,wn,Mn,Tn,In,Cn,Nn,Gn,xn,Hn,Sn,En;function Rn(e,t){var n="undefined"!=typeof Symbol&&e[Symbol.iterator]||e["@@iterator"];if(!n){if(Array.isArray(e)||(n=function(e,t){if(e){if("string"==typeof e)return zn(e,t);var n=Object.prototype.toString.call(e).slice(8,-1);return"Object"===n&&e.constructor&&(n=e.constructor.name),"Map"===n||"Set"===n?Array.from(e):"Arguments"===n||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(n)?zn(e,t):void 0}}(e))||t&&e&&"number"==typeof e.length){n&&(e=n);var a=0,i=function(){};return{s:i,n:function(){return a>=e.length?{done:!0}:{done:!1,value:e[a++]}},e:function(e){throw e},f:i}}throw new TypeError("Invalid attempt to iterate non-iterable instance.\nIn order to be iterable, non-array objects must have a [Symbol.iterator]() method.")}var o,s=!0,l=!1;return{s:function(){n=n.call(e)},n:function(){var e=n.next();return s=e.done,e},e:function(e){l=!0,o=e},f:function(){try{s||null==n.return||n.return()}finally{if(l)throw o}}}}function zn(e,t){(null==t||t>e.length)&&(t=e.length);for(var n=0,a=new Array(t);n<t;n++)a[n]=e[n];return a}var On=["gene","organism","domain","reaction_id","ec_number","compound"],Ln=["gene","organism","domain","compound","reaction_id"],An=["kcat_values"],Bn=function(e){var t,n=[],a=Rn(On);try{for(a.s();!(t=a.n()).done;){var i=t.value,o=e[i];o&&o.length>0&&("ec_number"===i?n.push(Z()(rn||(rn=Y()(["array[","] <@ ",""])),Z()(cn||(cn=Y()(["",""])),o.toString()),Z()(dn||(dn=Y()(["string_to_array(ec_number, ';')"]))))):Ln.includes(i)?n.push(Z()(mn||(mn=Y()(["lower(",") = ",""])),Z()(i),o.toString().toLowerCase())):n.push(Z()(un||(un=Y()([""," = ",""])),Z()(i),o.toString())))}}catch(e){a.e(e)}finally{a.f()}var s,l=Rn(An);try{for(l.s();!(s=l.n()).done;){var r=s.value,c=e[r];if(c){var d=JSON.parse(c),m=d.min,u=d.max;void 0!==m&&n.push(Z()(pn||(pn=Y()([""," >= ",""])),Z()(r),m)),void 0!==u&&n.push(Z()(fn||(fn=Y()([""," <= ",""])),Z()(r),u))}}}catch(e){l.e(e)}finally{l.f()}return n};const kn=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c,d,m,p,f,h,g,y,b,_,v,w,M,T,I,C;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:if(n=t.filters,a=void 0===n?{}:n,i=t.pagination,o=(i=void 0===i?{}:i).column,s=void 0===o?"gene":o,l=i.isAscending,c=void 0===l?"true":l,d=i.pageSize,m=void 0===d?50:d,p=i.page,f=void 0===p?1:p,h=["gene","organism","domain","reaction_id","ec_number","compound","kcat_values"],g=Bn(a),y="true"===c.toLowerCase()?Z()(hn||(hn=Y()(["asc"]))):Z()(gn||(gn=Y()(["desc"]))),h.includes(s)){e.next=8;break}throw new Error("Can not sort on unknown column ".concat(s));case 8:return b=Ln.includes(s)?Z()(yn||(yn=Y()(["lower(",")"])),Z()(s)):Z()(s),_=!Object.prototype.hasOwnProperty.call(a,"domain")&&(Object.prototype.hasOwnProperty.call(a,"ec_number")?["reaction_id","compound"].includes(s):["gene","organism","domain","compound","reaction_id"].includes(s)),v=Z()(bn||(bn=Y()(["\n    ","\n\n    select "," \n    from enzymes\n    ","\n    order by "," ","\n    offset ","\n\n    ","\n\n    limit ","\n  "])),_?Z()(_n||(_n=Y()(["with sub_query as materialized ("]))):Z()(vn||(vn=Y()([""]))),Z()(h),g.length>0?Z()(wn||(wn=Y()(["where ",""])),g.reduce((function(e,t){return Z()(Mn||(Mn=Y()([""," and ",""])),e,t)}),Z()(Tn||(Tn=Y()(["true"]))))):Z()(In||(In=Y()([""]))),b,y,(f-1)*m,_?Z()(Cn||(Cn=Y()([") select "," from sub_query"])),Z()(h)):Z()(Nn||(Nn=Y()([""]))),m),w=Z()(Gn||(Gn=Y()(["\n     select count(*) from enzymes\n     ","\n   "])),g.length>0?Z()(xn||(xn=Y()(["where ",""])),g.reduce((function(e,t){return Z()(Hn||(Hn=Y()([""," and ",""])),e,t)}),Z()(Sn||(Sn=Y()(["true"]))))):Z()(En||(En=Y()([""])))),e.next=14,Promise.all([v,w]);case 14:return M=e.sent,T=u()(M,2),I=T[0],C=T[1],e.abrupt("return",{enzymes:I,totalCount:C[0].count});case 19:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();var Dn,Wn=["kegg"],Fn=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,Z()(Dn||(Dn=Y()(["\n    select * from genes\n    where kegg = ","\n  "])),t.toString());case 2:if((n=e.sent).length){e.next=5;break}throw new Error("Gene ".concat(t," returned ").concat(n.length," results."));case 5:return a=n[0],i=a.kegg,o=d()(a,Wn),s=Object.fromEntries(Object.entries({kegg:i}).filter((function(e){var t=u()(e,2);return t[0],t[1]}))),l=Object.fromEntries(Object.entries(o).filter((function(e){var t=u()(e,2);return t[0],t[1]})).map((function(e){var t=u()(e,2),n=t[0],a=t[1],i=Q[n],o=i.db,s=i.dbPrefix;return[o,a.split(";").map((function(e){return{id:e,url:"https://identifiers.org/".concat(s,":").concat(e)}}))]}))),e.abrupt("return",{info:s,crossReferences:l});case 10:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();const jn=Fn;var Pn,Un=function(){var e=s()(r().mark((function e(t){var n;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,Z()(Pn||(Pn=Y()(["\n    select * from organisms\n    where kegg = ","\n  "])),t.toString());case 2:if((n=e.sent).length){e.next=5;break}throw new Error("Organism ".concat(t," returned ").concat(n.length," results."));case 5:return e.abrupt("return",{info:n[0],crossReferences:[]});case 6:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();const Vn=Un;var qn,$n=["name","equation"],Jn=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return e.next=2,Z()(qn||(qn=Y()(["\n    select * from reactions\n    where kegg = ","\n  "])),t.toString());case 2:if((n=e.sent).length){e.next=5;break}throw new Error("Reaction with kegg ID ".concat(t," returned ").concat(n.length," results."));case 5:return a=n[0],i=a.name,o=a.equation,s=d()(a,$n),l=Object.fromEntries(Object.entries({name:i,equation:o}).filter((function(e){var t=u()(e,2);return t[0],t[1]}))),c=Object.fromEntries(Object.entries(s).filter((function(e){var t=u()(e,2);return t[0],t[1]})).map((function(e){var t=u()(e,2),n=t[0],a=t[1],i=Q[n],o=i.db,s=i.dbPrefix,l=i.reactionSuffix;return[o,a.split(";").map((function(e){return{id:e,url:"https://identifiers.org/".concat(s).concat(l,":").concat(e)}}))]}))),e.abrupt("return",{info:l,crossReferences:c});case 10:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();const Kn=Jn;var Yn,Xn,Zn,Qn,ea,ta;const na=function(){var e=s()(r().mark((function e(t){var n,a,i,o,s,l,c;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return n=t.toLowerCase(),a=Z()(Yn||(Yn=Y()(["\n    select\n      type,\n      id,\n      match,\n      "," as score\n    from multi_search\n    where match % ","\n    order by score desc\n    limit 10\n  "])),Z()(Xn||(Xn=Y()(["similarity(\n        ",",\n        ","\n      )"])),Z()(Zn||(Zn=Y()(["replace(match, '.', 'a')"]))),Z()(Qn||(Qn=Y()(["replace(",", '.', 'a')"])),n)),n),i=Z()(ea||(ea=Y()(["\n    select 'gene' as type, kegg as id, kegg as match, 1 as score\n    from genes\n    where lower(kegg) = ","\n  "])),n),e.next=5,Z()(ta||(ta=Y()(["set pg_trgm.similarity_threshold = 0.25"])));case 5:return e.next=7,Promise.all([a,i]);case 7:return o=e.sent,s=u()(o,2),l=s[0],c=s[1],e.abrupt("return",[].concat(C()(c),C()(l)).slice(0,10));case 12:case"end":return e.stop()}}),e)})));return function(t){return e.apply(this,arguments)}}();var aa=t().Router();aa.get("/compounds/:id",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params.id,e.prev=1,e.next=4,te(a);case 4:i=e.sent,n.json(i),e.next=12;break;case 8:e.prev=8,e.t0=e.catch(1),console.error(e.t0.message),n.sendStatus(404);case 12:case"end":return e.stop()}}),e,null,[[1,8]])})));return function(t,n){return e.apply(this,arguments)}}()),aa.get("/ecs/:value",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params.value,e.prev=1,e.next=4,ln(a);case 4:i=e.sent,n.json(i),e.next=12;break;case 8:e.prev=8,e.t0=e.catch(1),console.error(e.t0.message),n.sendStatus(404);case 12:case"end":return e.stop()}}),e,null,[[1,8]])})));return function(t,n){return e.apply(this,arguments)}}()),aa.get("/genes/:value",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params.value,e.prev=1,e.next=4,jn(a);case 4:i=e.sent,n.json(i),e.next=12;break;case 8:e.prev=8,e.t0=e.catch(1),console.error(e.t0.message),n.sendStatus(404);case 12:case"end":return e.stop()}}),e,null,[[1,8]])})));return function(t,n){return e.apply(this,arguments)}}()),aa.get("/organisms/:value",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params.value,e.prev=1,e.next=4,Vn(a);case 4:i=e.sent,n.json(i),e.next=12;break;case 8:e.prev=8,e.t0=e.catch(1),console.error(e.t0.message),n.sendStatus(404);case 12:case"end":return e.stop()}}),e,null,[[1,8]])})));return function(t,n){return e.apply(this,arguments)}}()),aa.get("/domains/:value",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params.value,e.prev=1,e.next=4,on(a);case 4:i=e.sent,n.json(i),e.next=12;break;case 8:e.prev=8,e.t0=e.catch(1),console.error(e.t0.message),n.sendStatus(404);case 12:case"end":return e.stop()}}),e,null,[[1,8]])})));return function(t,n){return e.apply(this,arguments)}}()),aa.get("/reactions/:id",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params.id,e.prev=1,e.next=4,Kn(a);case 4:i=e.sent,n.json(i),e.next=12;break;case 8:e.prev=8,e.t0=e.catch(1),console.error(e.t0.message),n.sendStatus(404);case 12:case"end":return e.stop()}}),e,null,[[1,8]])})));return function(t,n){return e.apply(this,arguments)}}()),aa.get("/enzymes",function(){var e=s()(r().mark((function e(t,n){var a,i,o,s;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.query,i=a.filters,o=a.pagination,e.prev=1,e.next=4,kn({filters:i,pagination:o});case 4:s=e.sent,n.json(s),e.next=12;break;case 8:e.prev=8,e.t0=e.catch(1),console.error(e.t0.message),n.status(400).send(e.t0.message);case 12:case"end":return e.stop()}}),e,null,[[1,8]])})));return function(t,n){return e.apply(this,arguments)}}()),aa.get("/search/:term",function(){var e=s()(r().mark((function e(t,n){var a,i;return r().wrap((function(e){for(;;)switch(e.prev=e.next){case 0:return a=t.params.term,e.prev=1,e.next=4,na(a);case 4:i=e.sent,n.json(i),e.next=12;break;case 8:e.prev=8,e.t0=e.catch(1),console.error(e.t0.message),n.sendStatus(400);case 12:case"end":return e.stop()}}),e,null,[[1,8]])})));return function(t,n){return e.apply(this,arguments)}}());const ia=aa,oa=require("swagger-ui-express");var sa=n.n(oa);const la=JSON.parse('{"swagger":"2.0","info":{"description":"Metabolic Atlas can be accessed programmatically using this web API. Here we provide endpoints to get global information on the list of models integrated in Metabolic Atlas and to retrieve bulk or detailed information on those models. The API is still under development and might change without notice.\\n\\nThe results are returned in JSON format.","version":"2.0","title":"Metabolic Atlas API","termsOfService":"https://metabolicatlas.org/about","contact":{"email":"contact@metabolicatlas.org"},"license":{"name":"GPL version 3","url":"https://www.gnu.org/licenses/gpl-3.0.html"}},"basePath":"/api/v2","tags":[{"name":"Integrated models"},{"name":"Identifier"},{"name":"Genes"},{"name":"Metabolites"},{"name":"Reactions"},{"name":"Subsystems"},{"name":"Compartments"},{"name":"Interaction Partners"},{"name":"GotEnzymes"},{"name":"Miscellaneous"}],"paths":{"/repository/integrated_models":{"get":{"tags":["Integrated models"],"summary":"List all integrated GEMs.","description":"This query retrieves a list of all GEMs (genome-scale metabolic models) that Metabolic Atlas integrates. These GEMs are available through tools like GEM Browser, Map Viewer and Interaction Partners. No parameters are required for this query.","operationId":"integratedModelListing","produces":["application/json"],"responses":{"200":{"description":"Successful query"}}}},"/repository/integrated_models/{name}":{"get":{"tags":["Integrated models"],"summary":"Get information for a given GEM.","description":"This query retrieve information for a given GEM (genome-scale metabolic model). The GEMs are available through tools like GEM Browser, Map Viewer and Interaction Partners.","operationId":"integratedModelInfo","produces":["application/json"],"parameters":[{"name":"name","in":"path","required":true,"type":"string","description":"the original GEM name, e.g. **Human-GEM**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/compartments/{id}?model={model}":{"get":{"tags":["Compartments"],"summary":"Get information for a given compartment.","description":"This query retrieves all the metabolites, genes, reactions and subsystems contained by the given compartment.","operationId":"compartmentInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the compartment identifier, e.g. **golgi_apparatus**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"},{"name":"full","in":"query","required":false,"type":"string","description":"if left out, or set to false, only get the number of metabolites, genes, reactions contained by the given compartment.","default":true}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/compartments/{id}/related-reactions?model={model}":{"get":{"tags":["Compartments"],"summary":"Get related reactions for a given compartment.","description":"This query retrieves all the related reactions for a given compartment. There reactions are associated to compartments via the compartmentalized metabolites.","operationId":"compartmentRelatedReactionInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the compartment ID, e.g. **golgi_apparatus**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/genes/{id}?model={model}":{"get":{"tags":["Genes"],"summary":"Get information for a given gene.","description":"This query retrieves all the information for a given gene.","operationId":"geneInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the gene ID, e.g. **ENSG00000196502**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/genes/{id}/related-reactions?model={model}":{"get":{"tags":["Genes"],"summary":"Get related reactions for a given gene.","description":"This query retrieve all the reactions related to the specified gene.","operationId":"geneRelatedReactionInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the gene ID, e.g. **ENSG00000196502**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/metabolites/{id}?model={model}":{"get":{"tags":["Metabolites"],"summary":"Get information for a given metabolite.","description":"This query retrieves all the information about the specified metabolite.","operationId":"metaboliteInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the metabolite ID, e.g. **MAM01587m**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/metabolites/{id}/related-reactions?model={model}":{"get":{"tags":["Metabolites"],"summary":"Get related reactions for a given metabolite.","description":"This query retrieves all the related reactions that involve the specified metabolite.","operationId":"metaboliteRelatedReactionInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the metabolite ID, e.g. **MAM01587m**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/metabolites/{id}/related-metabolites?model={model}":{"get":{"tags":["Metabolites"],"summary":"Get related metabolites for a given metabolite.","description":"This query retrieves all related metabolites involving a given metabolite. Metabolites are compartmentalized, which means that the related metabolites are equivalent metabolites in other compartments.","operationId":"metaboliteRelatedMetaboliteInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the metabolite ID, e.g. **MAM01587m**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/reactions/{id}?model={model}":{"get":{"tags":["Reactions"],"summary":"Get information for a given reaction.","description":"This query retrieves all the information we have on a reaction.","operationId":"reactionInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the reaction ID, e.g. **MAR03000**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/reactions/{id}/related-reactions?model={model}":{"get":{"tags":["Reactions"],"summary":"Get related reactions involving a given reaction.","description":"This query retrieves all related reactions involving a given reaction. A limit of 1000 related reactions is applied. These related reactions are computed based on the sharing of related (equivalent) metabolites.","operationId":"reactionRelatedReactionInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the reaction ID, e.g. **MAR03000**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/subsystems/{id}?model={model}":{"get":{"tags":["Subsystems"],"summary":"Get information for a given subsystem.","description":"This query retrieves all the information for a given subsystem, which includes names and identifiers for metabolites, genes, and compartments.","operationId":"subsystemInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the subsystem ID, e.g. **eicosanoid_metabolism**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/subsystems/{id}/related-reactions?model={model}":{"get":{"tags":["Subsystems"],"summary":"Get related reactions for a given subsystem.","description":"This query retrieves all the reactions for a specified subsystem. The query limits the results to 1000 related reactions.","operationId":"subsystemRelatedReactionInfo","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the subsystem ID, e.g. **eicosanoid_metabolism**"},{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/interaction-partners/{id}?model={model}":{"get":{"tags":["Interaction Partners"],"summary":"Get interaction partners for a given gene or metabolite.","description":"This query retrieves all first order interaction partners for a given gene or metabolite.","operationId":"interactionPartners","produces":["application/json"],"parameters":[{"name":"id","in":"path","required":true,"type":"string","description":"the gene ID, e.g. **ENSG00000196502** or the metabolite ID, e.g. **MAM03933c**"},{"name":"model","in":"path","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/maps/listing?model={model}":{"get":{"tags":["Miscellaneous"],"summary":"List all compartments and subsystems for a GEM.","description":"This query retrieves a list of all compartments and subsystems that are available on Metabolic Atlas for a specified GEM. This includes identifiers for the SVG maps. **Note that the response is large and might cause the browser to hang.**","operationId":"mapListing","produces":["application/json"],"parameters":[{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **YeastGem**"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/random-components?model={model}&componentTypes={componentTypes}":{"get":{"tags":["Miscellaneous"],"summary":"Get random components for a GEM.","description":"This query retrieves detailed information for a random sample of genes, metabolites, compartments, reactions and subsystems.","operationId":"randomComponentInfo","parameters":[{"name":"model","in":"query","required":true,"type":"string","description":"the model name, e.g. **HumanGem**"},{"name":"componentTypes","in":"query","type":"object","default":{"gene":true,"compartmentalizedMetabolite":true},"required":true,"description":"list of component types to return, e.g. **{\\"gene\\":true, \\"compartmentalizedMetabolite\\": true}**"}],"produces":["application/json"],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}},"/gotenzymes/reactions/{reactionId}":{"get":{"tags":["GotEnzymes"],"summary":"Get information and cross references for the given reaction.","operationId":"gotEnzymesReactionInfo","parameters":[{"name":"reactionId","in":"path","required":true,"type":"string","description":"the reaction id, e.g. **R01677**"}],"produces":["application/json"],"responses":{"200":{"description":"Successful query"},"404":{"description":"Not found"}}}},"/gotenzymes/genes/{geneId}":{"get":{"tags":["GotEnzymes"],"summary":"Get information and cross references for the given gene.","operationId":"gotEnzymesGeneInfo","parameters":[{"name":"geneId","in":"path","required":true,"type":"string","description":"the gene id, e.g. **zwf**"}],"produces":["application/json"],"responses":{"200":{"description":"Successful query"},"404":{"description":"Not found"}}}},"/gotenzymes/ecs/{ecNumber}":{"get":{"tags":["GotEnzymes"],"summary":"Get information and cross references for the given ec number.","operationId":"gotEnzymesECInfo","parameters":[{"name":"ecNumber","in":"path","required":true,"type":"string","description":"the ec number, e.g. **2.5.1.19**"}],"produces":["application/json"],"responses":{"200":{"description":"Successful query"},"404":{"description":"Not found"}}}},"/gotenzymes/compounds/{compoundId}":{"get":{"tags":["GotEnzymes"],"summary":"Get information and cross references for the given compound.","operationId":"gotEnzymesCompoundInfo","parameters":[{"name":"compoundId","in":"path","required":true,"type":"string","description":"the compound id, e.g. **C00003**"}],"produces":["application/json"],"responses":{"200":{"description":"Successful query"},"404":{"description":"Not found"}}}},"/gotenzymes/organisms/{organismId}":{"get":{"tags":["GotEnzymes"],"summary":"Get information and cross references for the given organism.","operationId":"gotEnzymesOrganismInfo","parameters":[{"name":"organismId","in":"path","required":true,"type":"string","description":"the organism id, e.g. **hsa**"}],"produces":["application/json"],"responses":{"200":{"description":"Successful query"},"404":{"description":"Not found"}}}},"/gotenzymes/enzymes?":{"get":{"tags":["GotEnzymes"],"summary":"Search for GotEnzymes predicted kcat.","description":"This query retrieves the predicted kcat for reactions, genes, ec numbers, compounds, organisms and domains matching the provided parameters. The results are paginated to handle large results, this can also be configured through parameters provided.","operationId":"gotEnzymes","parameters":[{"name":"filters[reaction_id]","in":"query","type":"string","description":"the reaction id, e.g. **R00835** (or leave empty for more results)","default":"R00835"},{"name":"filters[gene]","in":"query","type":"string","description":"the gene id, e.g. **zwf** (or leave empty for more results)","default":"zwf"},{"name":"filters[ec_number]","in":"query","type":"string","description":"the ec number, e.g. **1.1.1.49** (or leave empty for more results)","default":"1.1.1.49"},{"name":"filters[compound]","in":"query","type":"string","description":"the compound id, e.g. **C00092** (or leave empty for more results)","default":"C00092"},{"name":"filters[organism]","in":"query","type":"string","description":"the organism id, e.g. **cpj** (or leave empty for more results)","default":"cpj"},{"name":"filters[domain]","in":"query","type":"string","description":"the domain id, e.g. **B** (or leave empty for more results)","default":"B"},{"name":"pagination[pageSize]","in":"query","type":"integer","description":"How many results to return by each page, e.g. **50** (50 by default if left empty)","default":"50"},{"name":"pagination[page]","in":"query","type":"integer","description":"Which result page to return, e.g. **1** (1 by default if left empty)","default":"1"},{"name":"pagination[column]","in":"query","type":"string","description":"Which column to sort on, e.g. **gene** (gene by default if left empty)","default":"gene"},{"name":"pagination[isAscending]","in":"query","type":"boolean","description":"If result should be in ascending or descending order (true by default if left empty)","default":"true","enum":["true","false"]}],"produces":["application/json"],"responses":{"200":{"description":"Successful query"},"400":{"description":"Invalid input"}}}},"/identifier/{dbName}/{externalId}":{"get":{"tags":["Identifier"],"summary":"Get model components for a given database identifier","description":"This query retrieves the Metabolic Atlas components that a database identifier is associated with, across all integrated models.","operationId":"identifier","produces":["application/json"],"parameters":[{"name":"dbName","in":"path","required":true,"type":"string","description":"the name of the database, e.g. **Protein Atlas** or **MetabolicAtlas**"},{"name":"externalId","in":"path","required":true,"type":"string","description":"the external ID, e.g. **ENSG00000120915** or **MAR02279** for *Protein Atlas* and *MetabolicAtlas* respectively"}],"responses":{"200":{"description":"Successful query"},"404":{"description":"Invalid input"}}}}}}');var ra=n(29),ca=n.n(ra),da=t().Router(),ma={customSiteTitle:"Metabolic Atlas API",customCss:ca(),customJs:"public/helper.js",customCssUrl:"https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css"};da.use("/favicon*",(function(e,n,a){return t().static("public/favicon".concat(e.params[0]))(e,n,a)})),da.use("",sa().serve),da.get("",sa().setup(la,ma));const ua=da;var pa=t().Router();pa.use(St),pa.use("/repository",At),pa.use("/data-overlay",Ut),pa.use("/svg-thumbnail",tn),pa.use("/gotenzymes",ia),pa.use(ua);const fa=pa;var ha=t()();ha.use("/api/v2",fa),ha.use("/api/v2/public",t().static("public")),ha.use("/api/v2/svg",t().static("svg")),ha.listen(8081)})()})();